
			<!doctype html>
	<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" class="no-javascript">	<head>
		<meta charset="utf-8" />
		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
				<title>Atomic structure of the entire mammalian mitochondrial complex I :  Nature :  Nature Research</title>

				<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.getElementsByTagName('html')[0].className = 'js-enabled';
		//--><!]]></script>
		
							<script type="text/javascript" src="http://nexus.ensighten.com/npg/Bootstrap.js"></script>
		

									
										
																							






																											
	
	
																
					
	
	
	

		  
		
																																					





																		







																


	  
	
			
		
	








								




																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																															


						

											<meta name="keywords" content="Bioenergetics, Structural biology, Energy metabolism" />
															<meta name="description" content="Mitochondrial complex I (also known as NADH:ubiquinone oxidoreductase) contributes to cellular energy production by transferring electrons from NADH to ubiquinone coupled to proton translocation across the membrane. It is the largest protein assembly of the respiratory chain with a total mass of 970 kilodaltons. Here we present a nearly complete atomic structure of ovine (Ovis aries) mitochondrial complex I at 3.9 Å resolution, solved by cryo-electron microscopy with cross-linking and mass-spectrometry mapping experiments. All 14 conserved core subunits and 31 mitochondria-specific supernumerary subunits are resolved within the L-shaped molecule. The hydrophilic matrix arm comprises flavin mononucleotide and 8 iron–sulfur clusters involved in electron transfer, and the membrane arm contains 78 transmembrane helices, mostly contributed by antiporter-like subunits involved in proton translocation. Supernumerary subunits form an interlinked, stabilizing shell around the conserved core. Tightly bound lipids (including cardiolipins) further stabilize interactions between the hydrophobic subunits. Subunits with possible regulatory roles contain additional cofactors, NADPH and two phosphopantetheine molecules, which are shown to be involved in inter-subunit interactions. We observe two different conformations of the complex, which may be related to the conformationally driven coupling mechanism and to the active–deactive transition of the enzyme. Our structure provides insight into the mechanism, assembly, maturation and dysfunction of mitochondrial complex I, and allows detailed molecular analysis of disease-causing mutations." />
															<meta name="robots" content="noarchive" />
															<meta name="access" content="Yes" />
															<meta name="emanating" content="128X138X65X119" />
															<meta name="apple-itunes-app" content="app-id=409367154" />
															<meta name="license.subtype" content="N/A" />
															<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2.5" />
						
											<meta name="DC.publisher" content="Nature Research" />
															<meta name="DC.language" content="en" />
															<meta name="DC.rights" content="© 2016 Macmillan Publishers Limited, part of Springer Nature. All rights reserved." />
															<meta name="DC.title" content="Atomic structure of the entire mammalian mitochondrial complex I" />
															<meta name="DC.identifier" content="doi:10.1038/nature19794" />
															<meta name="DC.date" content="2016-09-05" />
																				<meta name="DC.creator" content="Karol Fiedorczuk" />
																<meta name="DC.creator" content="James A. Letts" />
																<meta name="DC.creator" content="Gianluca Degliesposti" />
																<meta name="DC.creator" content="Karol Kaszuba" />
																<meta name="DC.creator" content="Mark Skehel" />
																<meta name="DC.creator" content="Leonid A. Sazanov" />
										
											<meta name="prism.copyright" content="© 2016 Macmillan Publishers Limited, part of Springer Nature. All rights reserved." />
															<meta name="prism.rightsAgent" content="permissions@nature.com" />
															<meta name="prism.publicationDate" content="2016-09-05" />
															<meta name="prism.volume" content="538" />
															<meta name="prism.issue" content="7625" />
															<meta name="prism.section" content="Letter" />
															<meta name="prism.startingPage" content="406" />
															<meta name="prism.endingPage" content="410" />
															<meta name="prism.publicationName" content="Nature" />
															<meta name="prism.issn" content="0028-0836" />
															<meta name="prism.eIssn" content="1476-4687" />
						
											<meta name="citation_publisher" content="Nature Research" />
															<meta name="citation_title" content="Atomic structure of the entire mammalian mitochondrial complex I" />
															<meta name="citation_date" content="2016-10-20" />
															<meta name="citation_online_date" content="2016-09-05" />
															<meta name="citation_volume" content="538" />
															<meta name="citation_issue" content="7625" />
															<meta name="citation_firstpage" content="406" />
															<meta name="citation_doi" content="doi:10.1038/nature19794" />
															<meta name="citation_journal_title" content="Nature" />
																				<meta name="citation_author" content="Karol Fiedorczuk" />
																<meta name="citation_author" content="James A. Letts" />
																<meta name="citation_author" content="Gianluca Degliesposti" />
																<meta name="citation_author" content="Karol Kaszuba" />
																<meta name="citation_author" content="Mark Skehel" />
																<meta name="citation_author" content="Leonid A. Sazanov" />
										
											<meta name="WT.cg_n" content="Nature" />
															<meta name="WT.cg_s" content="Letter" />
															<meta name="WT.template" content="C" />
															<meta name="WT.mod_name" content="editor_summary;editor_feature;nature_jobs_and_events;most_read_and_related;innocentive" />
															<meta name="WT.action" content="impression;impression;impression;impression;impression" />
															<meta name="WT.source" content="article" />
															<meta name="WT.dl" content="0" />
															<meta name="WT.ndl" content="0" />
															<meta name="WT.site_id" content="562" />
															<meta name="WT.site_id_name" content="University of Colorado - Boulder" />
																
										
					
	<link rel="shortcut icon" href="/favicon.ico" />

	<link rel="search" href="/search/executeSearch" />

	<link rel="prev" title="Previous article" href="/nature/journal/v538/n7625/full/nature19836.html" />
	<link rel="next" title="Next article" href="/nature/journal/v538/n7625/full/nature19785.html" />

	<link rel="search" type="application/opensearchdescription+xml" href="http://www.nature.com/opensearch/opensearch.xml" title="nature.com" />
	<link rel="search" type="application/sru+xml" href="http://www.nature.com/opensearch/request" title="nature.com" />
		<link rel="alternate" type="application/rdf+xml" href="http://data.nature.com/ns/articles/10.1038/nature19794" />
										
										
					
			
																																																
										
																<link href="/view/styles/global.yui.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.layout.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.shared.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.sidebar.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.tabs.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.article.css" type="text/css" rel="stylesheet" />
								
																																															
												
																		<link href="/view/styles/global.main.print.css" type="text/css" rel="stylesheet" media="print" />
								
																																															
										
																<link href="/view/styles/global.header-v2.css" type="text/css" rel="stylesheet" />
								
																<!--[if gt IE 8]><!-->
																																										
										
																<link href="/view/styles/global.mobile.css" type="text/css" rel="stylesheet" />
								
																																															
										
																<link href="/view/styles/global.mobile-header-v2.css" type="text/css" rel="stylesheet" />
								
																<!--<![endif]-->
																																										
										
																<link href="/view/nature/styles/site.css" type="text/css" rel="stylesheet" />
								
																<!--[if gt IE 8]><!-->
																																										
										
																<link href="/view/nature/styles/site.mobile.css" type="text/css" rel="stylesheet" />
								
				<!--<![endif]-->
		
				<!--[if IE 6]>
		<script type="text/javascript" src="/view/scripts/global.ie6.js"></script>
		<![endif]-->
			</head>

				
						
	
			<!--[if lt IE 6]><body class="www-nature-com letter lt-ie6"><![endif]-->
	<!--[if IE 6]><body class="www-nature-com letter ie6"><![endif]-->
	<!--[if IE 7]><body class="www-nature-com letter ie7"><![endif]-->
	<!--[if IE 8]><body class="www-nature-com letter ie8"><![endif]-->
	<!--[if gt IE 8]><body class="www-nature-com letter gt-ie8"><![endif]-->
	<!--[if !IE]>--><body class="www-nature-com letter"><!--<![endif]-->

		
		<div id="constrain">
						<div id="constrain-header" class="cleared  show-eu-cookie-notice ">
								<header><div class="wrapper">
												
										
					<ul class="header-jumplinks hidden">
	<li><a href="#content">Jump to main content</a></li>
	<li><a href="#responsive-menu">Jump to navigation</a></li>
</ul>									<div class="eu-cookie-notice">
			<div class="cookie-wrapper">
				<form method="get" action="">
					<p class="content">
						<span class="icon"></span>We use cookies to improve your experience with our site.
						<span class="tools"><button type="submit" name="cookies" value="accepted">Accept and close</button> | <a href="/info/cookies.html">More info</a>.</span>
					</p>
				</form>
			</div>
		</div>
															
										
													
																					





<div class="constrain cleared" id="foreword">
				<nav>		<h1 class="home" id="top"><a href="/">nature.com<span class="hidden"> homepage</span></a></h1>
		<p class="sitemap">
			<a class="all-publications" href="/siteindex">Publications A-Z index</a>
			<a class="all-subjects" href="/subjects">Browse by subject</a>
		</p>
		<ul class="right">
																		<li class="cart">
																<a href="/ecommerce/cart/list.action" class="cart">Cart</a></li>
																				<li class="login">
																<a href="/foxtrot/svc/login" class="login">Login</a></li>
																				<li class="register">
																<a href="/register" class="register">Register</a></li>
														</ul>
	</nav>

	
			
																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																												
		
				<p class="institution-access access"><span>Access provided to</span><span class="user"> University of Colorado - Boulder</span><span> by</span><span class="inst"> University Libraries</span></p>
	</div>

	<div class="constrain cleared" id="top-ads">
					
												
									
																																							
																					
															
			
						
																				
														
																																																																																																	
																																																																																																													
													
																												
										
									
			
		
							
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
										
		
									
		
	
	
		
										
					
		
										
					
		
										
					
		
										
					
		
																																																																										
					
		
										
			
			
											
				
													<div id="div-gpt-ad-3" class="ad adunit-leaderboard div-gpt-ad leaderboard" data-gpt-unitpath="/285/nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="pos=top;artid=nature19794;type=lt;issue=7625;subjmeta=631,57,1464,45,535,80,642,333,1465;abr=!webtv">
													<noscript>
						
						
												
						
						<a href="http://pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=728x90&t=pos%3Dtop%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810"><img src="http://pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=728x90&t=pos%3Dtop%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810" alt="Advertisement" /></a>

													</noscript>
											</div>
											</div>

<div id="header" class="constrain v2">
		
						<header>
				<h1><a href="/nature">Nature</a></h1>
				<h2>International weekly journal of science</h2>
			</header>
			
								
										
					





			<form method="get" action="/search/executeSearch" id="header-search" class="search-form" role="search">
			<input type="hidden" name="sp-c" value="25"/>
			<input type="hidden" name="sp-s" value=""/>
															 									 						<input type="hidden" name="sp-q-1" value="NATURE,NEWS"/>						 			 <fieldset class="search-and-go"><label class="placeholder-replaces" for="header-keyword"><span>Search</span><input type="search" id="header-keyword" name="sp-q" placeholder="Search" value="" class="txt" maxlength="200"/></label><button type="submit" id="header-search-submit">Go</button></fieldset><fieldset class="search-advanced"><a class="adv-search" href="/search/adv_search?sp-q-1=nature,news">Advanced search</a></fieldset>
</form>
		
		
			<nav class="nav">
			<div id="responsive-menu">
				<div id="js-menu"><a href="#responsive-menu" class="menu-up"><span>Menu</span></a><a href="#js-menu" class="menu-down"><span>Menu</span></a></div>
				<ul id="main-navigation" class="clear">
									<li class="nav-home first"><a  href="/nature/index.html"><span>Home</span></a></li>
									<li class="nav-news"><a  href="/news/index.html"><span>News &amp; Comment</span></a></li>
									<li class="nav-research"><a  href="/nature/research/research_by_subject.html"><span>Research</span></a></li>
									<li class="nav-careers"><a  href="/nature/careers/index.html"><span>Careers &amp; Jobs</span></a></li>
									<li class="nav-current active"><a  href="/nature/journal/v538/n7625/index.html"><span>Current Issue</span></a></li>
									<li class="nav-archive"><a  href="/nature/archive/index.html"><span>Archive</span></a></li>
									<li class="nav-av"><a  href="/nature/audio_video/index.html"><span>Audio &amp; Video</span></a></li>
									<li class="nav-authors"><a  href="/nature/authors/index.html"><span>For Authors</span></a></li>
								</ul>
			</div>
		</nav>
	</div>

<div class="header-bottom">
					
							 								 								 					
				
					<div id="breadcrumbs" class="constrain clear">
				<nav role="navigation">
					
																																						<ul id="sub-navigation">
																			<li class="first parent parent-1"><a href="/nature/journal/v538/n7625/index.html">Current Issue</a></li>
																																										<li class="parent parent-2"><a href="/nature/journal/v538/n7625/index.html#lt">Letters</a></li>
																																										<li class="current"><span>Article</span></li>
										</ul>
				</nav>
			</div>
			
								
										
					
	

									
										
					
	

	</div>

					</div></header>

			</div>

			<div id="constrain-content" class="cleared">
				



	




																																																																																																																																																																																																																													















		












												










			


																		





		
						

						

	
		
						
					











































													


















 




























	






	






















																																																																									
																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																






											
										
							

		

			





		






		



							


								
						
<div id="content" class="col article-template" role="article">
										
										
					

			<article>									
										
								<header>			<div class="top-links cleared">
									<p class="article-type has-japanese">
									<span class="journal-title">Nature</span><span class="divider"> |
					</span>Letter				</p>

																
										
					<div class="comm-tool-box" >
			<ul class="article-com-tools has-japanese">
			<li class="print"><a href="javascript:window.print();" id="print-link" class="track" data-tooltip="Print">Print</a></li>
							<li class="bookmarking">
				<a id="toggle-bookmarking-links" href="javascript:;" class="tip" data-tooltip="Share/bookmark">Share/bookmark</a>
				<div id="bookmarking-links">
												
										
					
<ul class="bookmarking">
			<li class="citeulike"><a href="http://www.citeulike.org/posturl2?username=&amp;url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I&amp;rewrite=yes">Cite<span class="hidden"> </span>U<span class="hidden"> </span>Like</a></li>
		<li class="facebook"><a href="http://www.facebook.com/sharer.php?u=http://dx.doi.org/10.1038/nature19794&amp;t=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I">Facebook</a></li>
				<li class="twitter"><a href="http://twitter.com/home/?status=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I+http://go.nature.com/KKbOMV">Twitter</a></li>
	<li class="delicious"><a href="http://del.icio.us/post?url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I">Delicious</a></li>
	<li class="digg"><a href="http://digg.com/submit?phase=2&amp;url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I">Digg</a></li>
	<li class="google-plus"><a href="https://plus.google.com/share?hl=en&amp;url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I" >Google&#43;</a></li>
	<li class="linkedin"><a href="http://www.linkedin.com/cws/share?url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I">LinkedIn</a></li>
	<li class="reddit"><a href="http://www.reddit.com/s/http://dx.doi.org/10.1038/nature19794">Reddit</a></li>
	<li class="stumbleupon"><a href="http://www.stumbleupon.com/submit?url=http://dx.doi.org/10.1038/nature19794&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I">StumbleUpon</a></li>
</ul>					</div>
			</li>
			</ul>
</div>

								
										
					
		
<div class="language-switch-box">
	<span class="japanese language-switch"><a href="/nature/journal/v538/n7625/fp/nature19794_ja.html?lang=ja">日本語要約</a></span>
</div>	

		
		
		
								
		<ul class="prev-next-article">
							<li class="prev left-arrow">
											<a id="prev-article" class="track" href="/nature/journal/v538/n7625/full/nature19836.html">
							<div class="nav-details">
								<h3>Previous article</h3>
								<p class="article-type">
									<span class="journal-title">Nature</span>
																			<span class="divider"> | </span>
										Letter
																	</p>
								Molecular basis of Lys11-polyubiquitin specificity in the deubiquitinase Cezanne
							</div>
						</a>
									</li>
			
										<li class="next right-arrow">
										<a id="next-article" class="track" href="/nature/journal/v538/n7625/full/nature19785.html">
						<div class="nav-details">
							<h3>Next article</h3>
							<p class="article-type">
								<span class="journal-title">Nature</span>
																	<span class="divider"> | </span>
									Letter
															</p>
							X-ray structure of the human α4β2 nicotinic receptor
						</div>
					</a>
									</li>
					</ul>
									</div>

			
			 
			 
			
						
			
													
							<h1 class="article-heading">Atomic structure of the entire mammalian mitochondrial complex I</h1>
									
																	
										
																				
										
																				
										
												
												
												
												
												
												
								
																




	
	<ul class="authors citation-authors">
																																													
						
			<li class="vcard">

				<a href="#auth-1" class="name"><span class="fn">Karol Fiedorczuk</span></a><sup><a href="#a1">1</a>, <a href="#a2">2</a></sup><sup>, </sup>			</li>

						
								
																				
				
																																																																																																																																																																																																																																																																																																																								
							
																									
						
			<li class="vcard">

				<a href="#auth-2" class="name"><span class="fn">James A. Letts</span></a><sup><a href="#a1">1</a></sup><sup>, </sup>			</li>

						
								
																				
				
																																																																																																																																																																																																																																																																																																																								
							
																									
						
			<li class="vcard">

				<a href="#auth-3" class="name"><span class="fn">Gianluca Degliesposti</span></a><sup><a href="#a3">3</a></sup><sup>, </sup>			</li>

						
								
																				
				
																																																																																																																																																																																																																																																																																																																								
							
																									
						
			<li class="vcard">

				<a href="#auth-4" class="name"><span class="fn">Karol Kaszuba</span></a><sup><a href="#a1">1</a></sup><sup>, </sup>			</li>

						
								
																				
				
																																																																																																																																																																																																																																																																																																																								
							
																									
						
			<li class="vcard">

				<a href="#auth-5" class="name"><span class="fn">Mark Skehel</span></a><sup><a href="#a3">3</a></sup><sup>, </sup>			</li>

						
								
																				
				
																																																																																																																																																																																																																																																																																																																								
							
																																								
						
			<li class="vcard last-author c1">

				<a href="#auth-6" class="name"><span class="fn">Leonid A. Sazanov</span></a><sup><a href="#a1">1</a></sup><sup>, </sup>			</li>

						
								
																								
				
																																																																																																																																																																																																																																																																																																																								
							
			</ul>

	
	
							
		
													
					<ul id="author-links" class="cleared">
									<li class="first"><a href="#affil-auth">Affiliations</a></li>
									<li><a href="#contrib-auth">Contributions</a></li>
									<li><a href="#corres-auth">Corresponding author</a></li>
							</ul>
			
	<dl class="citation">
		<dt>Journal name:</dt>
		<dd class="journal-title">Nature</dd>
						<dt class="volume">Volume:</dt>
		<dd class="volume">
			538<span>,</span></dd>
											<dt>Pages:</dt>
				<dd class="page">406&ndash;410</dd>
							 				<dt class="published">Date published:</dt>
		<dd><time datetime="2016-09-05">(20 October 2016)</time></dd>
						 		<dt>DOI:</dt>
		<dd class="doi">doi:10.1038/nature19794</dd>
			</dl>

	<dl class="citation dates">
																							<dt class="received first">Received</dt>
			<dd><time datetime="2016-08-01">
															01 August 2016
												</time></dd>
											 												<dt class="accepted">Accepted</dt>
			<dd><time datetime="2016-08-26">
															26 August 2016
												</time></dd>
									
				
		    
		    
		    		    			    		    			    		    			    
	    	
			<dt class="published-online">Published online</dt>
			<dd><time datetime="2016-09-05" >
																	
				05 September 2016							</time></dd>
			
								</dl>
	
								
										
						
				
			 
										
										
					
	
														
										
					

 




<aside><div class="article-tools">
	<h2 class="hidden">Article tools</h2>
	<ul class="tools">
		 					
						
		 												<li class="download-pdf"><a href="/nature/journal/v538/n7625/pdf/nature19794.pdf" id="download-pdf">PDF</a>
						<div id="pdf-options" class="hidden">
							<h3 id="toggle-download-links">PDF</h3>
							<div id="download-links">
								<ul class="downloading ">
									<li class="download-option articlepdf"><a href="/nature/journal/v538/n7625/pdf/nature19794.pdf">Download as PDF (18,943 KB)</a>
									</li>
																		<li class="download-option readcube">
										<a href="http://readcube.com/view/10.1038/nature19794" class="readcube-view">View interactive PDF in ReadCube</a>
									</li>
								</ul>
							</div>
						</div>
					</li>
																		<li class="download-citation"><a href="/nature/journal/v538/n7625/ris/nature19794.ris">Citation</a></li>
							
															<li class="order-reprints"><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&amp;publication=Nature&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I&amp;contentID=10.1038%2Fnature19794&amp;volumeNum=538&amp;issueNum=7625&amp;numPages=5&amp;pageNumbers=pp406-410&amp;orderBeanReset=true&amp;publicationDate=2016-09-05&amp;author=Karol+Fiedorczuk%2C+James+A.+Letts%2C+Gianluca+Degliesposti%2C+Karol+Kaszuba%2C+Mark+Skehel%2C+Leonid+A.+Sazanov">Reprints</a></li>
				<li class="rights"><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPG&amp;publication=Nature&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I&amp;contentID=10.1038%2Fnature19794&amp;volumeNum=538&amp;issueNum=7625&amp;numPages=5&amp;pageNumbers=pp406-410&amp;publicationDate=2016-09-05&amp;author=Karol+Fiedorczuk%2C+James+A.+Letts%2C+Gianluca+Degliesposti%2C+Karol+Kaszuba%2C+Mark+Skehel%2C+Leonid+A.+Sazanov">Rights &amp; permissions</a></li>
					
		
					<li class="article-metrics">
				<a href="/nature/journal/v538/n7625/nature19794/metrics">Article metrics</a>
			</li>
			</ul>
</div></aside>

				
										
								
			
		</header>


											
										
					

	
	
							
											 									

		
										
				
				
			
										
																													
			
										
																																		
			
										
																																		
																
																			
																		
																		
																		
																		
																									
																																																					
																																																																		
																																														
																																														
																																							
																																
																		
																		
																		
					
						
																
																																																						
				
				
				

			
																				
						
									<section><div class="section first no-nav no-title first-no-nav">

								<div class="content ">
					
					
					
												
										
					<div class="first-paragraph" id="first-paragraph"><p>Mitochondrial complex I (also known as NADH:ubiquinone oxidoreductase) contributes to cellular energy production by transferring electrons from NADH to ubiquinone coupled to proton translocation across the membrane<sup><a href="#ref1" title="Sazanov, L. A. A giant molecular proton pump: structure and mechanism of respiratory complex I. Nat. Rev. Mol. Cell Biol. 16, 375–388 (2015)" id="ref-link-5">1</a>, <a href="#ref2" title="Moser, C. C., Farid, T. A., Chobot, S. E. &amp; Dutton, P. L. Electron tunneling chains of mitochondria. Biochim. Biophys. Acta 1757, 1096–1109 (2006)" id="ref-link-6">2</a></sup>. It is the largest protein assembly of the respiratory chain with a total mass of 970 kilodaltons<sup><a href="#ref3" title="Letts, J. A. &amp; Sazanov, L. A. Gaining mass: the structure of respiratory complex I-from bacterial towards mitochondrial versions. Curr. Opin. Struct. Biol. 33, 135–145 (2015)" id="ref-link-7">3</a></sup>. Here we present a nearly complete atomic structure of ovine (<i>Ovis aries</i>) mitochondrial complex I at 3.9 Å resolution, solved by cryo-electron microscopy with cross-linking and mass-spectrometry mapping experiments. All 14 conserved core subunits and 31 mitochondria-specific supernumerary subunits are resolved within the L-shaped molecule. The hydrophilic matrix arm comprises flavin mononucleotide and 8 iron–sulfur clusters involved in electron transfer, and the membrane arm contains 78 transmembrane helices, mostly contributed by antiporter-like subunits involved in proton translocation. Supernumerary subunits form an interlinked, stabilizing shell around the conserved core. Tightly bound lipids (including cardiolipins) further stabilize interactions between the hydrophobic subunits. Subunits with possible regulatory roles contain additional cofactors, NADPH and two phosphopantetheine molecules, which are shown to be involved in inter-subunit interactions. We observe two different conformations of the complex, which may be related to the conformationally driven coupling mechanism and to the active–deactive transition of the enzyme. Our structure provides insight into the mechanism, assembly, maturation and dysfunction of mitochondrial complex I, and allows detailed molecular analysis of disease-causing mutations.</p></div>


							
										
														
																																																													
															
					<div class="article-keywords inline-list cleared">
				<h2>Subject terms:</h2>
				<ul>
																														<li><a href="/nature/archive/subject.html?code=1464">Bioenergetics</a><span class="bullet">&bull;</span> </li><li><a href="/nature/archive/subject.html?code=535">Structural biology</a><span class="bullet">&bull;</span> </li><li class="last"><a href="/nature/archive/subject.html?code=1465">Energy metabolism</a></li>				</ul>
			</div>
											
										
														
	
		<div class="figures-at-a-glance">

												
		<h1 class="at-a-glance-heading">At a glance</h1>

																
																																																												
					
																																										
		
		
		<div id="figure-browser-group" class="tab-group primary-tabs">
							<div class="tab-box active first" id="figure-figures">
					<h3 class="tab">Figures</h3>
					<div class="tab-content figure-browser" id="figure-browser-figures">
						<div class="tab-inner">
							<div class="item-index"><span class="item-count"></span>
																	<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html" class="view-all" >View all figures</a>
															</div>

														<ol style="width: 2040px">
																																		
																		
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_F1.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-f1.jpg" alt="Structure of ovine complex I." class="fig carousel-item" data-media-desc="Figure" data-media-id="figure" style="width:145px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Figure 1<span class="break">: </span><span class="figure-desc">Structure of ovine complex I.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Cryo-EM density coloured by subunit, with core subunits in grey (left–right view). <b>b</b>, Structure depicted as a cartoon, with core subunits coloured and labelled, and supernumerary subunits in grey and transparent. Approximate lipid bilayer boundaries are indicated. <b>c</b>, Structure depicted with core subunits in grey and supernumerary subunits coloured and labelled (left–right, IMS–matrix views). Amphipathic helices at the ‘heel’ of the complex, probably attached to the lipid bilayer, are indicated as AH.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F1.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#f1">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																		
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_F2.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-f2.jpg" alt="Arrangement of redox centres and substrate binding sites." class="fig carousel-item" data-media-desc="Figure" data-media-id="figure" style="width:214px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Figure 2<span class="break">: </span><span class="figure-desc">Arrangement of redox centres and substrate binding sites.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Fe–S clusters are shown as spheres with centre-to-centre and edge-to-edge (in brackets) distances indicated in Å, overlaid with transparent grey depictions from <i>T. thermophilus</i>. Both traditional and structure-based (in brackets) nomenclature for clusters is shown. <b>b</b>, NADH-binding site (overlay with <i>T. thermophilus</i> structure in grey, containing NADH). Cryo-EM density for flavin mononucleotide (FMN) is shown in blue. Key residues involved in interactions with FMN and NADH are shown as sticks. <b>c</b>, Quinone-binding site with subunits coloured as <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>. Key β1–β2<sup>49-kDa</sup> loop deviates from bacterial structure (grey) and is more similar to <i>Y. lipolytica</i> (orange, PDB 4WZ7; ref. <a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)">12</a>), clashing with the decyl-ubiquinone (DQ) head group position in <i>T. thermophilus</i> (grey; ref. <a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)">9</a>). <b>d</b>, Environment surrounding the Q cavity (brown surface, entrance point indicated by an arrow), with some of the functionally important residues shown as sticks and labelled with non-ND1 subunit names in brackets. The quinone from the aligned <i>T. thermophilus</i> structure is shown in grey (DQ), demonstrating that the distal part of the cavity is blocked in the ovine enzyme.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F2.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#f2">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																		
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_F3.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-f3.jpg" alt="Additional cofactors identified in the structure." class="fig carousel-item" data-media-desc="Figure" data-media-id="figure" style="width:105px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Figure 3<span class="break">: </span><span class="figure-desc">Additional cofactors identified in the structure.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Overview of the model, coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1c</a>, with cofactors shown as sticks. CDL, cardiolipin; PC, phosphatidylcholine; PE, phosphatidylethanolamine; PPT, phosphopantetheine. <b>b</b>, NADPH in the 39-kDa subunit. Interacting residues are shown. <b>c</b>, Zn<sup>2+</sup> ion in the 13-kDa subunit, with coordinating residues. <b>d</b>, Phosphopantetheine in SDAP-α. <b>e</b>, Phosphopantetheine in SDAP-β. <b>f</b>. Lipids phosphatidylethanolamine, phosphatidylcholine and cardiolipin. All cofactors are shown with cryo-EM density carved to within 5 Å.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F3.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#f3">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																		
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_F4.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-f4.jpg" alt="Mechanism of mitochondrial complex I." class="fig carousel-item" data-media-desc="Figure" data-media-id="figure" style="width:182px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Figure 4<span class="break">: </span><span class="figure-desc">Mechanism of mitochondrial complex I.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Structure of the core subunits of ovine complex I, coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1b</a>, with polar residues in proton channels shown as sticks, with carbon in blue, orange and green for input, connecting and output parts, respectively. Key residues, Glu (TM5), Lys (TM7), Lys/His (TM8) and Lys/Glu (TM12) from the antiporters and the corresponding residues in the E-channel (near Q site), are shown as small spheres and labelled. These residues sit on flexible loops in discontinuous transmembrane helices shown as cylinders. Polar residues linking the E-channel to the Q cavity (brown) are shown in magenta. Tyr108<sup>49-kDa</sup> and His59<sup>49-kDa</sup> are shown in cyan near the position of bound Q in bacteria. Possible proton translocation pathways are indicated by blue arrows. <b>b</b>, Graphic of the coupling mechanism. Core and some putatively regulatory supernumerary subunits are shown. Conformational changes, indicated by red arrows, propagate from the Q site/E-channel to antiporter-like subunits via the central hydrophilic axis. Shifts of helices near the cluster N2 (ref. <a href="#ref31" title="Berrisford, J. M. &amp; Sazanov, L. A. Structural basis for the mechanism of respiratory complex I. J. Biol. Chem. 284, 29773–29783 (2009)">31</a>; blue arrows) may help initiate the process. ND5 helix HL and traverse helices from four supernumerary subunits on the IMS side may serve as stators. Dashed line indicates the shift of peripheral arm in the closed conformation (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html" class="draggable supp-fig" data-related-id="sf8">Extended Data Fig. 8</a>). The NADPH-containing 39-kDa subunit and Zn-containing 13-kDa subunit are essential for activity and may serve as redox sensors. Both SDAP subunits interact with their LYR partners via flipped-out phosphopantetheine (black line). The net result of one conformational cycle, driven by NADH:ubiquinone oxidoreduction, is the translocation of four protons across the membrane (black lines indicate possible pathways).</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F4.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#f4">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf1.jpg" alt="Image processing procedures." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:164px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 1<span class="break">: </span><span class="figure-desc">Image processing procedures.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Representative micrograph of 2.6 k micrographs collected that all varied in defocus, ice thickness and particle count, with good quality particles circled. Scale bar, 100 nm. <b>b</b>, Representative 2D class averages obtained from reference-free classification. <b>c</b>, Classification and refinement procedures used in this study.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf1-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf2.jpg" alt="Image and model refinement procedures." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:187px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 2<span class="break">: </span><span class="figure-desc">Image and model refinement procedures.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Radiation-damage weighting. Relative B-factors (<i>B</i><sub>f</sub>) and intercepts (<i>C</i><sub>f</sub>) from the Relion particle polishing procedure. <b>b</b>, Left, gold-standard (two halves of data refined independently) Fourier shell correlation (FSC) curves for the maps of the entire complex complete map (resolution at FSC = 0.143 is 3.9 Å), membrane-arm-focused refinement (4.1 Å resolution) and peripheral-arm-focused refinement (3.9 Å resolution). Right, FSC curve of the combined map versus final model shows good agreement of the model with the map (FSC = 0.5 at 4.0 Å resolution). FSC curve against the entire complex complete map, which was not used in refinement, is shown as a control. <b>c</b>, Statistics of refinement.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf2-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf3.jpg" alt="Local resolution estimation and combination of maps for model building." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:144px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 3<span class="break">: </span><span class="figure-desc">Local resolution estimation and combination of maps for model building.</span>
													</span>
													<div class="description">
													<p><b>a</b>–<b>c</b>, Local resolution estimation by Resmap of the entire complex I (<b>a</b>), peripheral-arm-focused refinement map (<b>b</b>) and membrane-arm-focused refinement map (<b>c</b>). Maps are coloured according to the shown resolution scale in Å. <b>d</b>. The final map was produced by combining maps with the best local resolution features; that is, for peripheral-arm-focused refinement map (orange), for the distal part of membrane-arm-focused refinement map (green), for 42-kDa subunit map from the selected homogenous complex I class (64 k particles; blue) and the rest of the complex from the best map of the entire complex (magenta).</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf3-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf4.jpg" alt="Examples of cryo-EM density." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:171px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 4<span class="break">: </span><span class="figure-desc">Examples of cryo-EM density.</span>
													</span>
													<div class="description">
													<p><b>a</b>, <b>b</b>, Coils and α-helices from core (<b>a</b>) and supernumerary (<b>b</b>) subunits. <b>c, d,</b> Example β-sheets from core PSST subunit (<b>c</b>) and supernumerary 39-kDa subunit (<b>d</b>). Cryo-EM density is shown with the model represented as sticks and coloured by atom with carbon in grey, oxygen in red, nitrogen in blue and sulfur in yellow.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf4-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf5.jpg" alt="Identified cross-links." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:178px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 5<span class="break">: </span><span class="figure-desc">Identified cross-links.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Solvent-accessible surface (SAS) representation of cross-links. Surfaces for complex I subunits are shown transparent and coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>. Shortest SAS paths calculated using Xwalk<sup><a href="#ref54" title="Kahraman, A., Malmström, L. &amp; Aebersold, R. Xwalk: computing and visualizing distances in cross-linking experiments. Bioinformatics 27, 2163–2164 (2011)">54</a></sup> are shown for cross-links as coloured worms with inter-subunit lysine reactive cross-links in blue, inter-subunit acid reactive cross-links in red, intra-subunit lysine reactive cross-links in light blue, intra-subunit acid reactive cross-links in light red. <b>b</b>, Inter-subunit cross-link schematic. Complex I subunits are shown in a similar orientation as in <b>a</b>. Left panel with core subunits cyan, previously assigned supernumerary subunits in magenta, newly assigned or newly built regions of supernumerary subunits in green, poly-alanine regions in orange and unmodelled regions in red. Observed cross-links are indicated by dashed black lines between either blue circles (lysine reactive cross-links) or red circles (acid reactive cross-links). No cross-links were observed to the core subunits of the membrane arm and hence they were omitted for clarity. The horizontal black lines indicate the approximate boundaries of the inner mitochondrial membrane. Subunits B14.7, B15 and ASHI are shown as being behind the membrane boundaries as they are found on the opposite (far) side of the membrane arm.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf5-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF6.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf6.jpg" alt="Folds of supernumerary subunits." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:170px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 6<span class="break">: </span><span class="figure-desc">Folds of supernumerary subunits.</span>
													</span>
													<div class="description">
													<p>Subunits are shown in cartoon representation, coloured blue to red from N to C terminus. Disulfide bridges are shown as sticks with sulfur in yellow.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF6.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf6-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF7.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf7.jpg" alt="Examples of supernumerary subunits interactions." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:153px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 7<span class="break">: </span><span class="figure-desc">Examples of supernumerary subunits interactions.</span>
													</span>
													<div class="description">
													<p><b>a</b>, Side view of complex I showing surfaces for subunits B14.5a and B16.6. <b>b</b>, IMS view of complex I showing surfaces for subunits SGDH and PDSW. The point at which the two subunits are intertwined is marked with a star. <b>c</b>, View of the hydrophilic arm looking from above the membrane arm. The surface of the 18-kDa subunit that spans the hydrophilic arm is shown. <b>d</b>, Matrix view of the tip of the membrane arm with the surface of supernumerary subunit B22 shown. <b>e</b>, Close up of the centre of the membrane arm on the IMS side. This region contains many interactions between supernumerary subunits and the side chains of residues involved are shown. The region is also a hot spot for cross-links, the side chains involved are shown and cross-links are indicated with dashed lines (acid cross-links: red; basic cross-links: blue). <b>f</b>, Close up of the C-terminal helix of supernumerary subunit PDSW at the centre of the membrane arm on the IMS side. This helix extends away from complex I and is encircled by the C termini of supernumerary subunits B14.5b, ESSS and B15. The side chains of residues involved in stabilizing interactions are shown. A possible disulfide bond between PDSW (Cys154) and ESSS (Cys112) and stabilizing salt bridges are indicated by dashed lines. Subunits are coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF7.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf7-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																											
																			<li>
																																	
											<figure>												<a class="item-link" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html">
													 													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-sf8.jpg" alt="Comparison of ‘open’ and ‘closed’ 3D class structures." class="fig carousel-item" data-media-desc="Extended Data Fig." data-media-id="figure" style="width:172px;height:200px;" />
												</a>

												<figcaption>
													<span class="legend cleared">
													Extended Data Fig. 8<span class="break">: </span><span class="figure-desc">Comparison of ‘open’ and ‘closed’ 3D class structures.</span>
													</span>
													<div class="description">
													<p><b>a</b>, <b>b</b>, Side (<b>a</b>) and top (<b>b</b>) view from the matrix for the alignment of the open class structure (in cyan) and closed class structure (in grey). To generate the closed class structure, the final structure of the open class was refined in real space in Phenix (5 macro cycles with morphing at each cycle) against 4.6 Å map of the closed class (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="draggable supp-fig" data-related-id="sf1">Extended Data Fig. 1</a>). All of the α-helices were well fit into density, but owing to low resolution of the closed class no further refinement was performed and the comparison of structures involves only the relative positions of secondary structure elements. The two structures were aligned via transmembrane core subunits and are displayed as cartoon models. In the closed class the peripheral arm undergoes a hinge-like motion around the Q site towards the tip of the membrane arm, with the direction of shift indicated by the arrow in <b>b</b>. As a result, subunit B13 moves ~3 Å closer to the 42-kDa subunit, allowing for direct contacts. The shift is larger at the periphery, reaching 7 Å at the tip of the peripheral arm. Additionally, subunit ND5 and its matrix bulge move about 3 Å towards peripheral arm.</p>
													</div>

																											<div class="larger">
																															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html">Full size image</a></p>
																																														<p class="fig-link down-arrow"><a href="#sf8-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 								 							</ol>
						</div>
					</div>
				</div>
			
							<div class="tab-box" id="figure-tables">
					<h3 class="tab">Tables</h3>
					<div class="tab-content figure-browser" id="figure-browser-tables">
						<div class="tab-inner">
							<div class="item-index"><span class="item-count"></span>
																	<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html" class="view-all">View all tables</a>
															</div>
														<ol style="width: 340px">
																																		
																			<li>
																																	
																																	
											
											<figure>												<a class="item-link" href="javascript:;">
													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-st1.jpg" alt="Summary of the model" class="table carousel-item" data-media-desc="Extended Data Table" data-media-id="table" style="width:136px;height:200px;" />
												</a>
												<figcaption>
													<span class="legend cleared">
													Extended Data Table 1<span class="break">: </span><span class="figure-desc">Summary of the model</span>
													</span>
													<div class="description">
														
													</div>

																											<div class="larger">
															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST1.html">Full size image</a></p>
																															<p class="fig-link down-arrow"><a href="#st1-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 																										
																			<li>
																																	
																																	
											
											<figure>												<a class="item-link" href="javascript:;">
													
	
																												
		
		
			
	<img src="/nature/journal/v538/n7625/carousel/nature19794-st2.jpg" alt="Summary of interactions between subunits of ovine complex I" class="table carousel-item" data-media-desc="Extended Data Table" data-media-id="table" style="width:149px;height:200px;" />
												</a>
												<figcaption>
													<span class="legend cleared">
													Extended Data Table 2<span class="break">: </span><span class="figure-desc">Summary of interactions between subunits of ovine complex I</span>
													</span>
													<div class="description">
														
													</div>

																											<div class="larger">
															<p class="fig-link full-size"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST2.html">Full size image</a></p>
																															<p class="fig-link down-arrow"><a href="#st2-inarticle">View in article</a></p>
																													</div>
																									</figcaption>
											</figure>

										</li>
									 								 							</ol>
						</div>
					</div>
				</div>
			
			
			
					</div>
	</div>
	
							
										
											
							
				

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="main">

									<h1 class="section-heading toggle">Main</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7">Main<span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
					<p>The electrochemical proton gradient across the inner mitochondrial membrane required by ATP synthase is maintained by the electron transport chain proton-pumping complexes I, III and IV (refs <a href="#ref1" title="Sazanov, L. A. A giant molecular proton pump: structure and mechanism of respiratory complex I. Nat. Rev. Mol. Cell Biol. 16, 375–388 (2015)" id="ref-link-8">1</a>, <a href="#ref2" title="Moser, C. C., Farid, T. A., Chobot, S. E. &amp; Dutton, P. L. Electron tunneling chains of mitochondria. Biochim. Biophys. Acta 1757, 1096–1109 (2006)" id="ref-link-9">2</a>). Complex I is crucial for the entire process, and even mild complex I deficiencies can cause severe pathologies<sup><a href="#ref4" title="Fassone, E. &amp; Rahman, S. Complex I deficiency: clinical features, biochemistry and molecular genetics. J. Med. Genet. 49, 578–590 (2012)" id="ref-link-10">4</a></sup>. Mammalian complex I is built of 45 (44 unique) subunits. Fourteen ‘core’ subunits, conserved from bacteria, comprise the ‘minimal’ form of the enzyme<sup><a href="#ref1" title="Sazanov, L. A. A giant molecular proton pump: structure and mechanism of respiratory complex I. Nat. Rev. Mol. Cell Biol. 16, 375–388 (2015)" id="ref-link-11">1</a>, <a href="#ref5" title="Walker, J. E. The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains. Q. Rev. Biophys. 25, 253–324 (1992)" id="ref-link-12">5</a></sup>, an L-shaped structure with seven subunits in the hydrophilic peripheral arm and another seven in the membrane arm. Mammalian complex I also contains 31 ‘supernumerary’ or ‘accessory’ subunits<sup><a href="#ref5" title="Walker, J. E. The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains. Q. Rev. Biophys. 25, 253–324 (1992)" id="ref-link-13">5</a></sup>, forming a shell around the core<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-14">6</a></sup>. The role of these subunits is unclear. Complex I probably translocates four protons for every two electrons transferred from NADH to ubiquinone<sup><a href="#ref7" title="Galkin, A., Dröse, S. &amp; Brandt, U. The proton pumping stoichiometry of purified mitochondrial complex I reconstituted into proteoliposomes. Biochim. Biophys. Acta 1757, 1575–1581 (2006)" id="ref-link-15">7</a>, <a href="#ref8" title="Wikström, M., Sharma, V., Kaila, V. R., Hosler, J. P. &amp; Hummer, G. New perspectives on proton pumping in cellular respiration. Chem. Rev. 115, 2196–2221 (2015)" id="ref-link-16">8</a></sup>.</p><p>Complex I is the least characterized enzyme of the electron transport chain. The crystal structure of bacterial (<i>Thermus thermophilus</i>) complex I is the only full atomic model of the enzyme<sup><a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-17">9</a>, <a href="#ref10" title="Sazanov, L. A. &amp; Hinchliffe, P. Structure of the hydrophilic domain of respiratory complex I from Thermus thermophilus. Science 311, 1430–1436 (2006)" id="ref-link-18">10</a>, <a href="#ref11" title="Efremov, R. G. &amp; Sazanov, L. A. Structure of the membrane domain of respiratory complex I. Nature 476, 414–420 (2011)" id="ref-link-19">11</a></sup>. In a later structure of the mitochondrial enzyme from aerobic yeast <i>Yarrowia lipolytica</i>, the atomic model comprises only about 25% of the protein<sup><a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-20">12</a></sup>. Studies of bovine complex I resulted in poly-alanine models for the core and 22 supernumerary subunits<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-21">6</a>, <a href="#ref13" title="Zhu, J. et al. Structure of subcomplex Iβ of mammalian respiratory complex I leads to new supernumerary subunit assignments. Proc. Natl Acad. Sci. USA 112, 12087–12092 (2015)" id="ref-link-22">13</a></sup>. Here we present the nearly complete atomic structure of mammalian complex I, containing all subunits and all known cofactors.</p><p>We used the ovine (<i>O. aries</i>) enzyme (Methods). Classification of cryo-electron microscopy (cryo-EM) images indicated that the relative orientation between the two arms of the complex is variable, producing classes with either an ‘open’ or ‘closed’ angle between them (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="draggable supp-fig" data-related-id="sf1">Extended Data Fig. 1</a>). Particles in the ‘open’ conformation produced a higher resolution map at ~3.9 Å (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html" class="draggable supp-fig" data-related-id="sf2">Extended Data Fig. 2</a>). The resolution drops at the periphery of the molecule (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended Data Fig. 3</a>), owing to remaining differences in conformation. Therefore, we performed 3D refinements focusing on the peripheral arm and membrane arm separately, resulting in more uniformly resolved maps for the peripheral arm at 3.9 Å resolution and for the membrane arm at 4.1 Å (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended Data Fig. 3</a>). The best maps were combined for model building (<a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1a</a>, <a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended Data Figs 3d</a> and <a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html" class="draggable supp-fig" data-related-id="sf4">4</a>).</p>							
								
										
								
										
								
										
										
					



					



		 		 	 
			
												
		
		
	
	<div class="figure cleared" id="f1">
		<figure>							<figcaption><span class="legend">
											Figure 1: Structure of ovine complex I.									</span></figcaption>
			
							<div class="item-content box">
			
														
					
					  
						
						
													<a href="/nature/journal/v538/n7625/fig_tab/nature19794_F1.html" class="fig-link ">
													
						
	
																							
		
			
	<img src="/nature/journal/v538/n7625/images_article/nature19794-f1.jpg" alt="Structure of ovine complex I." class="fig" style="width:435px;height:600px;" />

						</a>									
				<div class="description"><p><b>a</b>, Cryo-EM density coloured by subunit, with core subunits in grey (left–right view). <b>b</b>, Structure depicted as a cartoon, with core subunits coloured and labelled, and supernumerary subunits in grey and transparent. Approximate lipid bilayer boundaries are indicated. <b>c</b>, Structure depicted with core subunits in grey and supernumerary subunits coloured and labelled (left–right, IMS–matrix views). Amphipathic helices at the ‘heel’ of the complex, probably attached to the lipid bilayer, are indicated as AH.</p></div>
				
								
									
						
													<ul class="resources">
																	<li class="full"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F1.html" >Full size image (1,049 KB)</a></li>
								
								
																	<li class="slides"><a href="/nature/journal/v538/n7625/slides/nature19794-pf1.ppt">Download PowerPoint slide (830 KB)</a></li>
															</ul>
															
			</div>

							
																		
															<nav>							<ul class="figure-nav">
																<li class="index"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html">Figures/tables index</a></li>
								<li class="next down-arrow"><a href="#f2" title="Arrangement of redox centres and substrate binding sites.">Next</a></li>							</ul>
						</nav>
												
		</figure>
	</div>

	

								
								
			<p>Modelling of the core subunits was facilitated by the conservation of their fold from bacteria<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-23">6</a>, <a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-24">9</a></sup>. The assignment of the 31 supernumerary subunits (~0.5 MDa) to the remaining density is challenging. To provide experimental verification for previous assignments, to locate remaining subunits and to obtain restraints on the fold of individual subunits, we performed extensive cross-linking/mass-spectrometry mapping experiments (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html" class="draggable supp-fig" data-related-id="sf5">Extended Data Fig. 5</a>, <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Tables</a>). The initial structure was improved by density-guided re-building in Rosetta<sup><a href="#ref14" title="DiMaio, F. et al. Atomic-accuracy models from 4.5-Å cryo-electron microscopy data with density-guided iterative local refinement. Nat. Methods 12, 361–365 (2015)" id="ref-link-25">14</a></sup>, resulting in final model of high quality (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html" class="draggable supp-fig" data-related-id="sf2">Extended Data Fig. 2c</a>).</p>							
								
			<p>While this manuscript was under review, a 4.2 Å resolution cryo-EM model for the bovine complex I was published<sup><a href="#ref15" title="Zhu, J., Vinothkumar, K. R. &amp; Hirst, J. Structure of mammalian respiratory complex I. Nature 536, 354–358 (2016)" id="ref-link-26">15</a></sup>. The assignments of all subunits agree with our structure, and two major conformations of the complex (somewhat different from ovine) are also observed. However, owing to lower resolution, the completeness of the atomic model is low for the supernumerary subunits (73% of residues are without side chains) and for the core 51-kDa, 24-kDa and 75-kDa subunits (extended data tables 1 and 2 in ref. <a href="#ref15" title="Zhu, J., Vinothkumar, K. R. &amp; Hirst, J. Structure of mammalian respiratory complex I. Nature 536, 354–358 (2016)" id="ref-link-27">15</a>).</p><p>In our ovine structure (<a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>), subunits were built almost entirely as atomic models with only some surface-exposed loops missing. Subunit B14.7 is disordered, so this area was modelled as poly-alanine according to its clear density in our ovine supercomplex map<sup><a href="#ref16" title="Letts, J. A., Fiedorczuk, K. &amp; Sazanov, L. A. Architecture of respiratory supercomplexes. Nature http://dx.doi.org/10.1038/nature19774 (2016)" id="ref-link-28">16</a></sup>. The model is at the atomic level for 88% of the protein (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST1.html" class="draggable supp-tab" data-related-id="st1">Extended Data Table 1</a>), presenting, to our knowledge, the most complete atomic structure of mitochondrial complex I so far.</p>							
								
			<p>The fold of core subunits is generally conserved from bacteria (<a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Discussion</a>). The Fe–S clusters are arranged in the redox chain with distances similar to bovine<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-29">6</a></sup> and <i>T. thermophilus</i><sup><a href="#ref10" title="Sazanov, L. A. &amp; Hinchliffe, P. Structure of the hydrophilic domain of respiratory complex I from Thermus thermophilus. Science 311, 1430–1436 (2006)" id="ref-link-30">10</a></sup> (<a href="/nature/journal/v538/n7625/full/nature19794.html#f2">Fig. 2a</a>). The NADH-binding site is also conserved (<a href="/nature/journal/v538/n7625/full/nature19794.html#f2">Fig. 2b</a>), preserving the entire path for electron transfer from NADH towards quinone. Key features in the membrane domain are also conserved, with four proton channels built around the central axis of polar residues propagating from the quinone-binding (Q) site into the three antiporter-like subunits.</p>							
										
					



					



		 		 	 
			
												
		
		
	
	<div class="figure cleared" id="f2">
		<figure>							<figcaption><span class="legend">
											Figure 2: Arrangement of redox centres and substrate binding sites.									</span></figcaption>
			
							<div class="item-content box">
			
														
					
					  
						
						
													<a href="/nature/journal/v538/n7625/fig_tab/nature19794_F2.html" class="fig-link ">
													
						
	
																							
		
			
	<img src="/nature/journal/v538/n7625/images_article/nature19794-f2.jpg" alt="Arrangement of redox centres and substrate binding sites." class="fig" style="width:600px;height:561px;" />

						</a>									
				<div class="description"><p><b>a</b>, Fe–S clusters are shown as spheres with centre-to-centre and edge-to-edge (in brackets) distances indicated in Å, overlaid with transparent grey depictions from <i>T. thermophilus</i>. Both traditional and structure-based (in brackets) nomenclature for clusters is shown. <b>b</b>, NADH-binding site (overlay with <i>T. thermophilus</i> structure in grey, containing NADH). Cryo-EM density for flavin mononucleotide (FMN) is shown in blue. Key residues involved in interactions with FMN and NADH are shown as sticks. <b>c</b>, Quinone-binding site with subunits coloured as <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>. Key β1–β2<sup>49-kDa</sup> loop deviates from bacterial structure (grey) and is more similar to <i>Y. lipolytica</i> (orange, PDB 4WZ7; ref. <a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-1">12</a>), clashing with the decyl-ubiquinone (DQ) head group position in <i>T. thermophilus</i> (grey; ref. <a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-2">9</a>). <b>d</b>, Environment surrounding the Q cavity (brown surface, entrance point indicated by an arrow), with some of the functionally important residues shown as sticks and labelled with non-ND1 subunit names in brackets. The quinone from the aligned <i>T. thermophilus</i> structure is shown in grey (DQ), demonstrating that the distal part of the cavity is blocked in the ovine enzyme.</p></div>
				
								
									
						
													<ul class="resources">
																	<li class="full"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F2.html" >Full size image (674 KB)</a></li>
								
								
																	<li class="slides"><a href="/nature/journal/v538/n7625/slides/nature19794-pf2.ppt">Download PowerPoint slide (678 KB)</a></li>
															</ul>
															
			</div>

																					
																		
															<nav>							<ul class="figure-nav">
								<li class="prev up-arrow"><a href="#f1" title="Structure of ovine complex I.">Previous</a></li>								<li class="index"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html">Figures/tables index</a></li>
								<li class="next down-arrow"><a href="#f3" title="Additional cofactors identified in the structure.">Next</a></li>							</ul>
						</nav>
												
		</figure>
	</div>

	

	<p>Throughout the article, we use bovine nomenclature with numbering of residues according to mature<sup><a href="#ref17" title="Carroll, J., Fearnley, I. M., Shannon, R. J., Hirst, J. &amp; Walker, J. E. Analysis of the subunit composition of complex I from bovine heart mitochondria. Mol. Cell. Proteomics 2, 117–126 (2003)" id="ref-link-31">17</a></sup> ovine sequences; see <a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST1.html" class="draggable supp-tab" data-related-id="st1">Extended Data Table 1</a> for human nomenclature. The Q site lies at the interface of the hydrophilic 49-kDa and PSST subunits, and the membrane ND1 and ND3 subunits. The unique structure of the Q site, which forms an enclosed tunnel extending from the membrane towards cluster N2 about 25 Å away, is conserved with one difference: a loop connecting two strands of the N-terminal β-sheet from the 49-kDa subunit (β1–β2<sup>49-kDa</sup> loop) extends further into the cavity, clashing with the position of the bound quinone from the bacterial structure<sup><a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-32">9</a></sup>, where it interacts with conserved His59<sup>49-kDa</sup> and Tyr108<sup>49-kDa</sup> (<a href="/nature/journal/v538/n7625/full/nature19794.html#f2">Fig. 2c, d</a>). A similar conformation was observed in the yeast enzyme, leading to the proposal that it represents the ‘deactive’ state<sup><a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-33">12</a></sup>. In the absence of substrates, mitochondrial complex I exists in the deactive state (which may prevent oxygen radical production <i>in vivo</i><sup><a href="#ref18" title="Chouchani, E. T. et al. Cardioprotection by S-nitrosation of a cysteine switch on mitochondrial complex I. Nat. Med. 19, 753–759 (2013)" id="ref-link-34">18</a></sup>), and converts into the ‘active’ state<sup><a href="#ref19" title="Vinogradov, A. D. &amp; Grivennikova, V. G. The mitochondrial complex I: progress in understanding of catalytic properties. IUBMB Life 52, 129–134 (2001)" id="ref-link-35">19</a></sup> only upon turnover. Because the β1–β2<sup>49-kDa</sup> loop in our structure will prevent quinone access closer than 20 Å to cluster N2 (blocking electron transfer), it probably also represents the deactive state. The ‘closed’ class conformation resembles one in supercomplex<sup><a href="#ref16" title="Letts, J. A., Fiedorczuk, K. &amp; Sazanov, L. A. Architecture of respiratory supercomplexes. Nature http://dx.doi.org/10.1038/nature19774 (2016)" id="ref-link-36">16</a></sup>, so may be more physiological. It remains to be established whether, as discussed previously<sup><a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-37">12</a>, <a href="#ref15" title="Zhu, J., Vinothkumar, K. R. &amp; Hirst, J. Structure of mammalian respiratory complex I. Nature 536, 354–358 (2016)" id="ref-link-38">15</a></sup>, different observed conformations are related to the catalytic cycle or indeed to active/deactive transitions, but the overall conformational flexibility of the complex is clear.</p><p>Supernumerary subunits form a shell around the core subunits<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-39">6</a>, <a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-40">12</a></sup>, especially around the membrane domain and its interface with the peripheral arm. With few exceptions, most supernumerary subunits are not globular, but form extended structures containing α-helices and coils (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF6.html" class="draggable supp-fig" data-related-id="sf6">Extended Data Fig. 6</a>), allowing for numerous interactions at interfaces with other subunits (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST2.html" class="draggable supp-tab" data-related-id="st2">Extended Data Table 2</a>). They interweave extensively with each other and the core subunits (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF7.html" class="draggable supp-fig" data-related-id="sf7">Extended Data Fig. 7</a>), making the whole mitochondrial complex assembly much more interlinked and thus more stable, with a large total buried surface area (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST2.html" class="draggable supp-tab" data-related-id="st2">Extended Data Table 2</a>). The intertwined nature of subunit structures suggests that they can be added to the complex only in a certain order, and, therefore, that the assembly of subunits must be tightly controlled<sup><a href="#ref20" title="Sánchez-Caballero, L., Guerrero-Castillo, S. &amp; Nijtmans, L. Unraveling the complexity of mitochondrial complex I assembly: A dynamic process. Biochim. Biophys. Acta 1857, 980–990 (2016)" id="ref-link-41">20</a></sup>.</p>							
								
										
								
										
								
			<p>The fold of supernumerary subunits is described in the <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Discussion</a>. In summary, those associated with the membrane arm include 12 single transmembrane helix domain<sup><a href="#ref21" title="Zickermann, V., Angerer, H., Ding, M. G., Nübel, E. &amp; Brandt, U. Small single transmembrane domain (STMD) proteins organize the hydrophobic subunits of large membrane protein complexes. FEBS Lett. 584, 2516–2525 (2010)" id="ref-link-42">21</a></sup> subunits scattered around the entire arm. Six of these surround the membrane arm tip and contribute their intertwined N-terminal domains to a large matrix ‘bulge’, the bulk of which is formed by an acyl-carrier protein (ACP)–LYR motif subunit pair (SDAP-β–B22). The large globular 42-kDa subunit from the nucleoside kinase family is attached to the matrix side of ND2 near the peripheral arm interface. On the intermembrane space (IMS) side, subunits SGDH and PDSW are ‘interlocked’ via their backbone and contain three long α-helices traversing nearly the entire membrane arm (<a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1c</a>). PDSW and the subunits with CHCH domains (PGIV, 15 kDa and B18) contain disulfide bonds that further stabilize the fold in the oxidizing environment of the IMS. PGIV clamps the ‘heel’ of the complex to the middle of the membrane arm. The disulfide-rich, interlocked helices of the IMS subunits, with their rigid and stable structure, appear to replace the hairpin/helix motif (βH) found in bacterial complex I (refs <a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-43">9</a>, <a href="#ref11" title="Efremov, R. G. &amp; Sazanov, L. A. Structure of the membrane domain of respiratory complex I. Nature 476, 414–420 (2011)" id="ref-link-44">11</a>).</p><p>Subunits associated with the peripheral arm include the NADPH-containing 39-kDa subunit, the Zn-containing 13-kDa subunit and another ACP–LYR motif pair, SDAP-α–B14, with the latter pair and B13 jointly ‘embracing’ the 42-kDa subunit. The interface between the peripheral arm and membrane domain is stabilized by the exceptionally long membrane-traversing helix of subunit B16.6, as well as by B17.2 and B14.5a, both of which contain N-terminal amphipathic α-helices bound at the membrane interface, with the rest of their polypeptides wrapping around the hydrophilic arm. Subunits PSST, TYKY and B9 also contain such amphipathic helices, all located at the heel of the complex (<a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1c</a>), probably assisting in proper peripheral arm position over the lipid bilayer.</p><p>Several cofactors present in supernumerary subunits are well resolved in the structure (<a href="/nature/journal/v538/n7625/full/nature19794.html#f3">Fig. 3</a>). The 39-kDa subunit is wedged into the side of the peripheral arm near the membrane arm interface. It contains a tightly bound non-catalytic NADPH (<a href="/nature/journal/v538/n7625/full/nature19794.html#f3">Fig. 3b</a>) that interacts with conserved Arg178<sup>PSST</sup>, providing a possible mitochondria redox state-sensitive conformational link to cluster N2. In SDAP-α, a phosphopantetheine that is covalently linked to Ser44 extends its attached acyl chain in the flipped-out<sup><a href="#ref22" title="Cronan, J. E. The chain-flipping mechanism of ACP (acyl carrier protein)-dependent enzymes appears universal. Biochem. J. 460, 157–163 (2014)" id="ref-link-45">22</a></sup> conformation into the hydrophobic crevice between the helices of the LYR subunit B14 (<a href="/nature/journal/v538/n7625/full/nature19794.html#f3">Fig. 3d</a>). A similar interaction is observed in the SDAP-β–B22 pair. These are the first structures of ACP–LYR complexes showing that their interaction depends on the extended acyl chain and revealing the role of LYR motif. Complex I is active only when fully assembled with the SDAP-α–B14 pair<sup><a href="#ref23" title="Angerer, H. Eukaryotic LYR proteins interact with mitochondrial protein complexes. Biology (Basel) 4, 133–150 (2015)" id="ref-link-46">23</a></sup>. This interaction, which depends on the acyl chain attached to the ACP, may provide a regulatory link between fatty acid synthesis and oxidative phosphorylation in mitochondria. The 13-kDa subunit contains a Zn-binding motif, coordinating a Zn<sup>2+</sup> ion in the vicinity of clusters N6a and N5. Zn-containing proteins are sensitive to oxidative stress<sup><a href="#ref24" title="Oteiza, P. I. Zinc and the modulation of redox homeostasis. Free Radic. Biol. Med. 53, 1748–1759 (2012)" id="ref-link-47">24</a></sup>, and loss of the 13-kDa subunit leads to loss of cluster N6a (ref. <a href="#ref25" title="Kmita, K. et al. Accessory NUMM (NDUFS6) subunit harbors a Zn-binding site and is essential for biogenesis of mitochondrial complex I. Proc. Natl Acad. Sci. USA 112, 5685–5690 (2015)" id="ref-link-48">25</a>) as it becomes exposed. In this way, complex I may be equipped with an oxidative-stress ‘sensor’, in addition to bound NADPH.</p>							
										
					



					



		 		 	 
			
												
		
		
	
	<div class="figure cleared" id="f3">
		<figure>							<figcaption><span class="legend">
											Figure 3: Additional cofactors identified in the structure.									</span></figcaption>
			
							<div class="item-content box">
			
														
					
					  
						
						
													<a href="/nature/journal/v538/n7625/fig_tab/nature19794_F3.html" class="fig-link ">
													
						
	
																							
		
			
	<img src="/nature/journal/v538/n7625/images_article/nature19794-f3.jpg" alt="Additional cofactors identified in the structure." class="fig" style="width:316px;height:600px;" />

						</a>									
				<div class="description"><p><b>a</b>, Overview of the model, coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1c</a>, with cofactors shown as sticks. CDL, cardiolipin; PC, phosphatidylcholine; PE, phosphatidylethanolamine; PPT, phosphopantetheine. <b>b</b>, NADPH in the 39-kDa subunit. Interacting residues are shown. <b>c</b>, Zn<sup>2+</sup> ion in the 13-kDa subunit, with coordinating residues. <b>d</b>, Phosphopantetheine in SDAP-α. <b>e</b>, Phosphopantetheine in SDAP-β. <b>f</b>. Lipids phosphatidylethanolamine, phosphatidylcholine and cardiolipin. All cofactors are shown with cryo-EM density carved to within 5 Å.</p></div>
				
								
									
						
													<ul class="resources">
																	<li class="full"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F3.html" >Full size image (1,392 KB)</a></li>
								
								
																	<li class="slides"><a href="/nature/journal/v538/n7625/slides/nature19794-pf3.ppt">Download PowerPoint slide (684 KB)</a></li>
															</ul>
															
			</div>

																					
																		
															<nav>							<ul class="figure-nav">
								<li class="prev up-arrow"><a href="#f2" title="Arrangement of redox centres and substrate binding sites.">Previous</a></li>								<li class="index"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html">Figures/tables index</a></li>
								<li class="next down-arrow"><a href="#f4" title="Mechanism of mitochondrial complex I.">Next</a></li>							</ul>
						</nav>
												
		</figure>
	</div>

	

	<p>Twelve bound lipids were identified in crevices between hydrophobic subunits. Several observed lipid molecules have four acyl chains and were therefore assigned as cardiolipins, known to be essential for activity<sup><a href="#ref26" title="Sharpley, M. S., Shannon, R. J., Draghi, F. &amp; Hirst, J. Interactions between phospholipids and NADH:ubiquinone oxidoreductase (complex I) from bovine mitochondria. Biochemistry 45, 241–248 (2006)" id="ref-link-49">26</a></sup>. Notably, a cardiolipin (CDL1; <a href="/nature/journal/v538/n7625/full/nature19794.html#f3">Fig. 3a</a>) and three other lipid molecules fill the void left by the missing (in metazoans) three ND2 N-terminal helices<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-50">6</a></sup>. This void is encircled by the two-transmembrane helix subunit B14.5b and the single transmembrane helix domain subunit KFYI, indicating that the ND2 helices may have been lost in evolution to accommodate a specific binding site for lipids. Two cardiolipins (CDL2 and CDL3) fill a large gap between the antiporter-like ND4 and ND5 subunits, preventing potential proton leaks and instability. Another cardiolipin (CDL4) stabilizes amphipathic helices at the heel of the complex. The structure thus shows the basis for the essential role of cardiolipin and other lipids.</p><p>The mechanism of coupling between electron transfer and proton translocation is still enigmatic. Conservation of key features from bacteria to mammals suggests that the basic mechanism is probably the same, with add-on ‘stabilizers’ and ‘regulators’. As we proposed previously<sup><a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-51">9</a></sup>, the central axis of polar residues in the membrane probably has a key role (<a href="/nature/journal/v538/n7625/full/nature19794.html#f4">Fig. 4a</a>). In each catalytic cycle, the negative charge stored either on Q or on nearby residues in the enclosed Q site may drive conformational changes in ND1 and the proton channel near the Q site, which would propagate via the central axis to channels in antiporter-like subunits ND2, ND4 and ND5, resulting in changes in p<i>K</i><sub>a</sub> and accessibility of key residues. The net result would be the pumping of four protons per cycle, one per each channel. The observed conformation of loops in the Q site probably reflects the deactive state. This conformation might also occur during normal function when quinol is ejected from the site into the lipid bilayer, if active/deactive transitions are related to conformations encountered during the catalytic cycle<sup><a href="#ref12" title="Zickermann, V. et al. Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I. Science 347, 44–49 (2015)" id="ref-link-52">12</a></sup>. Supernumerary subunits implicated in active/deactive transitions (39 kDa, B13 and SDAP-α–B14 pair) could also participate in catalytic conformational changes by interacting with the key TM1–TM2<sup>ND3</sup> loop flanking the Q site, and possibly through interactions with the 42-kDa subunit (<a href="/nature/journal/v538/n7625/full/nature19794.html#f4">Fig. 4b</a>). In the ‘closed’ class, B13 and SDAP-α move towards the 42-kDa subunit (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html" class="draggable supp-fig" data-related-id="sf8">Extended Data Fig. 8</a>), hinting at such a possibility. Because the 42-kDa subunit is metazoan-specific<sup><a href="#ref27" title="Brandt, U. Energy converting NADH:quinone oxidoreductase (complex I). Annu. Rev. Biochem. 75, 69–92 (2006)" id="ref-link-53">27</a></sup>, its role may be to fine tune movements during turnover. The traverse helix HL from ND5 appears to mainly have a stabilizing ‘stator’ role<sup><a href="#ref28" title="Belevich, G., Knuuti, J., Verkhovsky, M. I., Wikström, M. &amp; Verkhovskaya, M. Probing the mechanistic role of the long α-helix in subunit L of respiratory Complex I from Escherichia coli by site-directed mutagenesis. Mol. Microbiol. 82, 1086–1095 (2011)" id="ref-link-54">28</a></sup> rather than being a moving element<sup><a href="#ref29" title="Efremov, R. G., Baradaran, R. &amp; Sazanov, L. A. The architecture of respiratory complex I. Nature 465, 441–445 (2010)" id="ref-link-55">29</a></sup>. Rigid disulfide-rich supernumerary subunits traversing the IMS side of the membrane domain may represent another stator element unique to the mitochondrial enzyme (<a href="/nature/journal/v538/n7625/full/nature19794.html#f4">Fig. 4b</a>).</p>							
										
					



					



		 		 	 
			
												
		
		
	
	<div class="figure cleared" id="f4">
		<figure>							<figcaption><span class="legend">
											Figure 4: Mechanism of mitochondrial complex I.									</span></figcaption>
			
							<div class="item-content box">
			
														
					
					  
						
						
													<a href="/nature/journal/v538/n7625/fig_tab/nature19794_F4.html" class="fig-link ">
													
						
	
																							
		
			
	<img src="/nature/journal/v538/n7625/images_article/nature19794-f4.jpg" alt="Mechanism of mitochondrial complex I." class="fig" style="width:545px;height:600px;" />

						</a>									
				<div class="description"><p><b>a</b>, Structure of the core subunits of ovine complex I, coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1b</a>, with polar residues in proton channels shown as sticks, with carbon in blue, orange and green for input, connecting and output parts, respectively. Key residues, Glu (TM5), Lys (TM7), Lys/His (TM8) and Lys/Glu (TM12) from the antiporters and the corresponding residues in the E-channel (near Q site), are shown as small spheres and labelled. These residues sit on flexible loops in discontinuous transmembrane helices shown as cylinders. Polar residues linking the E-channel to the Q cavity (brown) are shown in magenta. Tyr108<sup>49-kDa</sup> and His59<sup>49-kDa</sup> are shown in cyan near the position of bound Q in bacteria. Possible proton translocation pathways are indicated by blue arrows. <b>b</b>, Graphic of the coupling mechanism. Core and some putatively regulatory supernumerary subunits are shown. Conformational changes, indicated by red arrows, propagate from the Q site/E-channel to antiporter-like subunits via the central hydrophilic axis. Shifts of helices near the cluster N2 (ref. <a href="#ref31" title="Berrisford, J. M. &amp; Sazanov, L. A. Structural basis for the mechanism of respiratory complex I. J. Biol. Chem. 284, 29773–29783 (2009)" id="ref-link-3">31</a>; blue arrows) may help initiate the process. ND5 helix HL and traverse helices from four supernumerary subunits on the IMS side may serve as stators. Dashed line indicates the shift of peripheral arm in the closed conformation (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html" class="draggable supp-fig" data-related-id="sf8">Extended Data Fig. 8</a>). The NADPH-containing 39-kDa subunit and Zn-containing 13-kDa subunit are essential for activity and may serve as redox sensors. Both SDAP subunits interact with their LYR partners via flipped-out phosphopantetheine (black line). The net result of one conformational cycle, driven by NADH:ubiquinone oxidoreduction, is the translocation of four protons across the membrane (black lines indicate possible pathways).</p></div>
				
								
									
						
													<ul class="resources">
																	<li class="full"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_F4.html" >Full size image (636 KB)</a></li>
								
								
																	<li class="slides"><a href="/nature/journal/v538/n7625/slides/nature19794-pf4.ppt">Download PowerPoint slide (656 KB)</a></li>
															</ul>
															
			</div>

																					
				
															<nav>							<ul class="figure-nav">
								<li class="prev up-arrow"><a href="#f3" title="Additional cofactors identified in the structure.">Previous</a></li>								<li class="index"><a href="/nature/journal/v538/n7625/fig_tab/nature19794_ft.html">Figures/tables index</a></li>
															</ul>
						</nav>
												
		</figure>
	</div>

	

								
								
			<p>Our structure clearly shows that supernumerary subunits stabilize the complex. Some of them, especially those containing additional cofactors (39 kDa, SDAPs, B14, B22 and 13 kDa) and phosphorylated residues (42 kDa, ESSS, MWFE, B14.5a, B14.5b and B16.6)<sup><a href="#ref30" title="Palmisano, G., Sardanelli, A. M., Signorile, A., Papa, S. &amp; Larsen, M. R. The phosphorylation pattern of bovine heart complex I subunits. Proteomics 7, 1575–1583 (2007)" id="ref-link-56">30</a></sup>, may provide regulatory links to the redox status of the cell, lipid biosynthesis and mitochondrial homeostasis. Known human pathological mutations are present in all of the core and many of the supernumerary subunits<sup><a href="#ref4" title="Fassone, E. &amp; Rahman, S. Complex I deficiency: clinical features, biochemistry and molecular genetics. J. Med. Genet. 49, 578–590 (2012)" id="ref-link-57">4</a></sup>. Our structure provides the framework for understanding the molecular basis of mutations and mechanisms of complex I function and regulation.</p>

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="methods">

									<h1 class="section-heading toggle">Methods</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7">Methods<span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
					<h2>Data reporting</h2><p>No statistical methods were used to predetermine sample size. The experiments were not randomized. The investigators were not blinded to allocation during experiments and outcome assessment.</p><h2>Protein purification and electron microscopy</h2><p>Protein was purified from <i>O. aries</i> heart mitochondria following the protocol adapted with some modifications from a previously published procedure for the bovine enzyme (ref. <a href="#ref32" title="Sazanov, L. A., Peak-Chew, S. Y., Fearnley, I. M. &amp; Walker, J. E. Resolution of the membrane domain of bovine complex I into subcomplexes: implications for the structural organization of the enzyme. Biochemistry 39, 7229–7235 (2000)" id="ref-link-58">32</a> and J.A.L. <i>et al</i>., manuscript submitted). We explored <i>O. aries</i> as a source of complex I that may be more suitable for high-resolution structural studies than the extensively studied bovine enzyme. We find that ovine enzyme appears more stable, as it is highly active after purification and retains the 42-kDa subunit, easily lost from bovine complex (J.A.L. <i>et al</i>., manuscript submitted). In terms of overall sequence similarity ovine is as good a model of the human enzyme as bovine (~84%), and all 44 different subunits of complex I were identified in the preparation by mass spectrometry (J.A.L. <i>et al</i>., manuscript submitted). In brief, fresh ovine hearts were purchased from the local abattoir and mitochondria prepared as described previously<sup><a href="#ref32" title="Sazanov, L. A., Peak-Chew, S. Y., Fearnley, I. M. &amp; Walker, J. E. Resolution of the membrane domain of bovine complex I into subcomplexes: implications for the structural organization of the enzyme. Biochemistry 39, 7229–7235 (2000)" id="ref-link-59">32</a></sup>. Mitochondrial membranes were solubilized in the branched chain detergent lauryl maltose neopentyl glycol (LMNG, 1%) and the sample applied to Q-sepharose HP anion exchange column (GE Healthcare) equilibrated with 20 mM Tris-HCl, pH 7.4, 10% (v/v) glycerol, 1 mM EDTA, 1 mM DTT and 0.1% LMNG. Protein was eluted with a NaCl gradient, peak fractions concentrated and applied to Superose 6 HiLoad 16/60 column equilibrated in 20 mM HEPES, pH 7.4, 2 mM EDTA, 1.5% (v/v) glycerol, 100 mM NaCl and 0.02% Brij-35. The peak fraction was concentrated to ~5 mg ml<sup>−1</sup> protein and ~0.2% Brij-35. Then, 2.7 μl of sample was applied to glow discharged Quantifoil R 0.6/1 copper grids and blotted for 34 s at 90% humidity in the chamber of FEI Vitrobot III. Immediately after, the sample was snap-frozen in liquid ethane. Extensive trials with different detergents, including previously used Cymal-7 (ref. <a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-60">6</a>), revealed Brij-35 as the detergent giving the most homogeneous spread of particles. Imaging was performed with a 300 kV Titan Krios electron microscope equipped with direct electron detector FEI Falcon-II (ETH Zurich, ScopeM centre) in automated data collection mode at a calibrated magnification of 1.39 Å pixel<sup>−1</sup> (×100,720) and dose of 26 e s<sup>−1</sup> Å<sup>−2</sup> with total 3-s exposure time. The data were collected as seven movie frames fractionated over the first second of exposure and an averaged image over 3 s.</p><h2>Image processing</h2><p>We collected a total of 2.6 k micrographs in two datasets, which were combined. All processing steps were done using RELION<sup><a href="#ref33" title="Scheres, S. H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. J. Struct. Biol. 180, 519–530 (2012)" id="ref-link-61">33</a></sup> unless otherwise stated. We used averaged images from high dose 3 s exposure for initial CTF estimations using CTFFIND4 (ref. <a href="#ref34" title="Rohou, A. &amp; Grigorieff, N. CTFFIND4: Fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192, 216–221 (2015)" id="ref-link-62">34</a>) and for automated particle picking in Relion, resulting in ~241 k particles. MOTIONCORR<sup><a href="#ref35" title="Li, X. et al. Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM. Nat. Methods 10, 584–590 (2013)" id="ref-link-63">35</a></sup> was used for whole-image drift correction of movie frames 1–7 (1 s) of each micrograph. Contrast transfer function (CTF) parameters of the corrected micrographs were estimated using Gctf and refined locally for each particle<sup><a href="#ref36" title="Zhang, K. Gctf: Real-time CTF determination and correction. J. Struct. Biol. 193, 1–12 (2016)" id="ref-link-64">36</a></sup>. The particles were extracted using 296<sup>2</sup> pixel box and sorted by reference-free 2D classification, resulting in ~171 k particles selected from good 2D classes. These were used for 3D classification with a regularization parameter T of 8 and a 30 Å low-pass filtered initial model from a previous low resolution model of the bovine enzyme<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-65">6</a></sup>. That resulted in ~130 k particles of good quality; however, it was clear that the relative orientation between the two arms of the complex is slightly variable, producing 3D classes with either an open or closed angle between the arms (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="draggable supp-fig" data-related-id="sf1">Extended Data Fig. 1</a>). Particles in the open conformation (~82 k particles) produced higher resolution maps and were selected for a final reconstruction. For all high-resolution refinements, particles were re-extracted from the motion corrected micrographs with a 512<sup>2</sup> pixel box to allow for high-resolution CTF correction<sup><a href="#ref37" title="Rosenthal, P. B. &amp; Henderson, R. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. J. Mol. Biol. 333, 721–745 (2003)" id="ref-link-66">37</a></sup>. After initial auto-refinement, particle-based beam-induced motion correction and radiation-damage weighing (particle polishing) was performed<sup><a href="#ref38" title="Scheres, S. H. Beam-induced motion correction for sub-megadalton cryo-EM particles. eLife 3, e03665 (2014)" id="ref-link-67">38</a></sup>. Refinement of polished particles gave a map resolved to 3.9 Å. All resolutions are based on the gold-standard (two halves of data refined independently) FSC = 0.143 criterion<sup><a href="#ref39" title="Scheres, S. H. &amp; Chen, S. Prevention of overfitting in cryo-EM structure determination. Nat. Methods 9, 853–854 (2012)" id="ref-link-68">39</a></sup>. This 3D class selection probably still allows for small variations in the conformation, therefore the local resolution varies within the map, especially at the extremities of both arms (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended Data Fig. 3</a>). At the periphery of the molecule the resolution drops not only owing to the usual decrease in the precision of particle alignments in these areas, but also due to differences in the protein conformation, greatest at the edges of the molecule. To overcome this limitation we performed 3D refinement focused on the peripheral and membrane domains separately (with the subtraction of signal from the remaining parts of the complex<sup><a href="#ref40" title="Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. Sampling the conformational space of the catalytic subunit of human γ-secretase. eLife 4, e11182 (2015)" id="ref-link-69">40</a></sup>). This resulted in a 3.9 Å map of the peripheral arm, very well resolved in all areas, including the edges of the domain. The membrane domain refined to 4.1 Å, however, the map was more uniform and so better resolved for the distal part of the domain (near subunit ND5) as compared to the density from the refinement of the entire complex (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended data Fig. 3</a>). Higher quality refinement of the peripheral arm probably stems from the fact that high electron density of eight Fe–S clusters helps in particle alignment. The least ordered part of the complex is the 42-kDa subunit, loosely attached to the membrane domain. We have performed extended 3D classification of the open class to identify the most homogeneous population, especially with respect to 42-kDa subunit. This class (64 k particles) was refined to 4.0 Å, and the resulting density was used to model the 42-kDa subunit. To assist with overall model building and refinement, several maps were carved around specific parts of the complex and combined into one map in UCSF Chimera<sup><a href="#ref41" title="Pettersen, E. F. et al. UCSF Chimera–a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004)" id="ref-link-70">41</a></sup>: peripheral arm from peripheral-arm-focused refinement, the area around ND4/5 subunits (tip of the membrane domain) from membrane-arm-focused refinement, the 42-kDa subunit density as above, and the rest of the complex from the overall 3.9 Å map for the open class (<a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1a</a> and <a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="draggable supp-fig" data-related-id="sf3">Extended Data Fig. 3</a>). The final model was refined against this combined map.</p><p>The final map is of high quality, with about three-quarters of the map at 3.9 Å resolution and the rest at 4.1 Å. Large- and medium-size side chains, as well as relatively small Val and Thr, are clearly seen in the density (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html" class="draggable supp-fig" data-related-id="sf4">Extended Data Fig. 4</a>). Carboxylates (Asp, Glu) have much lower density than other residues owing to early radiation damage, as observed previously<sup><a href="#ref42" title="Allegretti, M., Mills, D. J., McMullan, G., Kühlbrandt, W. &amp; Vonck, J. Atomic model of the F420-reducing [NiFe] hydrogenase by electron cryo-microscopy using a direct electron detector. eLife 3, e01963 (2014)" id="ref-link-71">42</a></sup>. Disulfide bridges also are subject to early damage, as in X-ray crystallography<sup><a href="#ref43" title="Meents, A., Gutmann, S., Wagner, A. &amp; Schulze-Briese, C. Origin and temperature dependence of radiation damage in biological samples at cryogenic temperatures. Proc. Natl Acad. Sci. USA 107, 1094–1099 (2010)" id="ref-link-72">43</a></sup>. Few features at the interfaces of maps used for the combined map may be better resolved in individual maps, since in overlapping regions both maps contribute. For example, the β1–β2<sup>49-kDa</sup> loop is better resolved in the peripheral-arm-focused map, which is deposited along with other constituent maps. Overall map filtered to lower resolution is very similar to the previous 5 Å resolution map for the bovine enzyme<sup><a href="#ref6" title="Vinothkumar, K. R., Zhu, J. &amp; Hirst, J. Architecture of mammalian respiratory complex I. Nature 515, 80–84 (2014)" id="ref-link-73">6</a></sup>, suggesting that the mammalian complex I structure is very well conserved. One difference is that in ovine complex the accessory four-transmembrane subunit B14.7 is disordered in the detergent used for the microscopy samples (Brij-35). It is likely to be disordered rather than detached as B14.7 was identified by mass spectrometry in the sample used for electron microscopy (data not shown). Since Brij-35 gave us the best yield of particles, we kept its use for data collection, but took advantage of the availability of cryo-EM maps of ovine respiratory supercomplexes in our laboratory. In these maps all the subunits of complex I are well ordered, and so in our final complex I model we included the poly-ALA model of B14.7 based on 5.8 Å resolution map of the ‘tight’ respirasome<sup><a href="#ref16" title="Letts, J. A., Fiedorczuk, K. &amp; Sazanov, L. A. Architecture of respiratory supercomplexes. Nature http://dx.doi.org/10.1038/nature19774 (2016)" id="ref-link-74">16</a></sup>. Loss of B14.7 also results in the disorder of the nearby C-terminal half of transverse helix HL and TM16 from ND5, as well as TM4 from ND6, which were also modelled as poly-alanine (these stretches can be recognized by B-factor set to 200) based on the tight respirasome map. The register in poly-alanine stretches is approximate. The density for the 42-kDa subunit is rather weak but this subunit clearly preserves the nucleoside kinase family fold, which allowed us to model most of it using Rosetta and visible large side-chains as a guide.</p><h2>Model building and refinement</h2><p>For the 14 core subunits the initial homology models were generated manually based on the <i>T. thermophilus</i> structure<sup><a href="#ref9" title="Baradaran, R., Berrisford, J. M., Minhas, G. S. &amp; Sazanov, L. A. Crystal structure of the entire respiratory complex I. Nature 494, 443–448 (2013)" id="ref-link-75">9</a></sup> with side-chains rebuilt to ovine sequence using SQWRL4 software<sup><a href="#ref44" title="Krivov, G. G., Shapovalov, M. V. &amp; Dunbrack, R. L. Jr. Improved prediction of protein side-chain conformations with SCWRL4. Proteins 77, 778–795 (2009)" id="ref-link-76">44</a></sup>. Homology models were generated with Phyre2 (ref. <a href="#ref45" title="Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. &amp; Sternberg, M. J. The Phyre2 web portal for protein modeling, prediction and analysis. Nat. Protocols 10, 845–858 (2015)" id="ref-link-77">45</a>) and Swiss-model<sup><a href="#ref46" title="Biasini, M. et al. SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Res. 42, W252–W258 (2014)" id="ref-link-78">46</a></sup> servers for all supernumerary subunits, although they were mostly useful only for subunits with large globular domains, such as the 42-kDa and 39-kDa subunits, as well as for those with known structure of close homologues (SDAPs and B8). Secondary structure predictions for all subunits were generated with PredictProtein<sup><a href="#ref47" title="Rost, B., Yachdav, G. &amp; Liu, J. The PredictProtein server. Nucleic Acids Res. 32, W321–W326 (2004)/" id="ref-link-79">47</a></sup>, PsiPred<sup><a href="#ref48" title="McGuffin, L. J., Bryson, K. &amp; Jones, D. T. The PSIPRED protein structure prediction server. Bioinformatics 16, 404–405 (2000)" id="ref-link-80">48</a></sup> and TMHMM<sup><a href="#ref49" title="Krogh, A., Larsson, B., von Heijne, G. &amp; Sonnhammer, E. L. Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. J. Mol. Biol. 305, 567–580 (2001)" id="ref-link-81">49</a></sup> servers, and were helpful during model building. Initial assignments of the location and the fold of supernumerary subunits were based on our cross-linking data and the secondary structure features and side-chain density observed in the cryo-EM map, with checks for consistency with the knowledge on subcomplexes and assembly intermediates in complex I. The initial models were adjusted to cryo-EM density (in cases when homology models were useful) or built manually in COOT<sup><a href="#ref50" title="Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta Crystallogr. D 60, 2126–2132 (2004)" id="ref-link-82">50</a></sup>. Lipids were tentatively assigned on the basis of appearance in the density as cardiolipins, phosphatidylcholines and phosphatidylethanolamines, known to co-purify with the complex<sup><a href="#ref26" title="Sharpley, M. S., Shannon, R. J., Draghi, F. &amp; Hirst, J. Interactions between phospholipids and NADH:ubiquinone oxidoreductase (complex I) from bovine mitochondria. Biochemistry 45, 241–248 (2006)" id="ref-link-83">26</a>, <a href="#ref51" title="Shinzawa-Itoh, K. et al. Bovine heart NADH-ubiquinone oxidoreductase contains one molecule of ubiquinone with ten isoprene units as one of the cofactors. Biochemistry 49, 487–492 (2010)" id="ref-link-84">51</a></sup>.</p><p>Initial models were re-built and refined in Rosetta release version 2016.02.58402 using protocols optimized for cryo-EM maps<sup><a href="#ref14" title="DiMaio, F. et al. Atomic-accuracy models from 4.5-Å cryo-electron microscopy data with density-guided iterative local refinement. Nat. Methods 12, 361–365 (2015)" id="ref-link-85">14</a></sup>. For each subunit, 100 different models were produced in Rosetta with optimization of density fit using elec_dens_fast function (with <i>-</i>denswt = 40, chosen from several trials), selection of the best fitting structure and structure relaxation using <i>-</i>FastRelax flag. From the produced structures several best-scoring by density fit and geometry were selected and used in COOT to guide further model building/optimization. This procedure resulted mainly in improvements to backbone geometry, especially in coils, still allowing for the good fit of side-chains into density.</p><p>After several rounds of re-building the final model was refined with the Phenix suite<sup><a href="#ref52" title="Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D 66, 213–221 (2010)" id="ref-link-86">52</a></sup> phenix.real_space_refine program for 5 macro-cycles using the electron scattering table with default and secondary structure restrains. This resulted in a high quality model in terms of geometry (Molprobity score 2.5, that is, corresponding to average structure at 2.5 Å resolution) and fit to density (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html" class="draggable supp-fig" data-related-id="sf2">Extended Data Fig. 2c</a>).</p><h2>Cross-linking</h2><p>All the cross-linking reactions were performed using purified solutions of complex I at a concentration of 1 mg ml<sup>−1</sup>. Following experimental optimisation, ten separate experiments were performed. Experiments varied in relation to the detergent added to the buffer (DDM, LMNG or LDAO/DDM), the cross-linking reagent (targeting lysine or acidic residues) and the protease used to digest the samples (trypsin or endoproteinase Glu-C) (<a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Table 1</a>).</p><p>Isotopically labelled cross-linking reagents were purchased from Creative Molecules (Canada). 4-(4,6-dimethoxy-1,3,5-triazin-2-yl)-4-methylmorpholinium chloride (DMTMM) was purchased from Sigma. Homobifunctional, isotopically-coded <i>N</i>-hydroxysuccinimide (NHS) esters disuccidinimidyl suberate (DSS H<sub>12</sub>/D<sub>12</sub>), bis-sulfodisuccinimidyl suberate (BS3 H<sub>12</sub>/D<sub>12</sub>) and disuccinimidyl adipate (DSA <sup>12</sup>C<sub>6</sub>/<sup>13</sup>C<sub>6</sub>) were used at a final concentration of 50 μM as cross-linking reagents to target lysine residues. The reactions were incubated for 45 min at 37 °C and quenched by adding NH<sub>4</sub>HCO<sub>3</sub> to a final concentration of 50 mM and incubating for further 15 min. Isotopically labelled adipic acid dihydrazide (ADH H<sub>8</sub>/D<sub>8</sub>) and suberic acid dihydrazide (SDH H<sub>12</sub>/D<sub>12</sub>) were used to target the acidic residues, using DMTMM as catalyst. The cross-linking reaction was initiated by adding ADH or SDH and DMTMM to final concentrations 5 mg ml<sup>−1</sup>, 6 mg ml<sup>−1</sup> and 12 mg ml<sup>−1</sup>, respectively. The samples were incubated at 37 °C for 60 min and the reactions stopped using gel filtration (Zeba Spin Desalting columns 7K MWCO).</p><p>The cross-linked samples were freeze-dried and then resuspended in 50 mM NH<sub>4</sub>HCO<sub>3</sub>, 8 M urea and 0.1% SDS to a final concentration of 1 mg ml<sup>−1</sup>. Size exclusion protein fractionation was performed through a Superdex 200 Increase 3.2/300 column (GE Healthcare) with 50 mM NH<sub>4</sub>HCO<sub>3</sub>, 8 M urea and 0.1% SDS as mobile phase at a flow rate of 25 μl min<sup>−1</sup>. Two-minute fractions were collected and their protein content evaluated by SDS–PAGE. Fractions of similar content were pooled into 4–5 main fractions and concentrated to 1 mg ml<sup>−1</sup> using Amicon Ultra-0.5 mL Centrifugal Filters (Millipore).</p><p>The filtered cross-linked samples were then enzymatically digested. Samples were freeze-dried and resuspended in 50 mM NH<sub>4</sub>HCO<sub>3</sub> and 8 M urea to a final protein concentration of 1 mg ml<sup>−1</sup>, reduced with 10 mM DTT and alkylated with 50 mM iodoacetamide. Following alkylation, samples were diluted with 50 mM NH<sub>4</sub>HCO<sub>3</sub> to 1 M urea before trypsin digestion (or 2 M for Glu-C digestion). Trypsin and Glu-C were added at an enzyme-to-substrate ratio of 1:20 and 1:100, respectively. Digestions were carried out overnight at 37 °C and 25 °C for trypsin and Glu-C respectively. After digestion, the samples were acidified with formic acid to a final concentration of 2% (v/v) and the peptides fractionated by peptide size exclusion chromatography, using a Superdex Peptide 3.2/300 (GE Healthcare) with 30% (v/v) acetonitrile/0.1% (v/v) TFA as mobile phase and at a flow rate of 50 μl min<sup>−1</sup>. Fractions were collected every 2 min over the elution volume 1.0 ml to 1.7 ml. Before LC–MS/MS analysis fractions were freeze dried and resuspended in 2% (v/v) acetonitrile and 2% (v/v) formic acid.</p><p>The digests were analysed by nano-scale capillary LC–MS/MS using an Ultimate U3000 HPLC (ThermoScientific Dionex) to deliver a flow of approximately 300 nl min<sup>−1</sup>. A C18 Acclaim PepMap100 5 μm, 100 μm × 20 mm nanoViper (ThermoScientific Dionex), trapped the peptides before separation on a C18 Acclaim PepMap100 3 μm, 75 μm × 250 mm nanoViper (ThermoScientific Dionex). Peptides were eluted with a gradient of acetonitrile. The analytical column outlet was directly interfaced via a nano-flow electrospray ionisation source, with a hybrid dual pressure linear ion trap mass spectrometer (Orbitrap Velos, ThermoScientific). Data-dependent analysis was carried out, using a resolution of 30,000 for the full mass spectrometry spectrum, followed by ten MS/MS spectra in the linear ion trap. Mass spectrometry spectra were collected over a <i>m</i>/<i>z</i> range of 300–2000. MS/MS scans were collected using threshold energy of 35 for collision-induced dissociation.</p><p>For data analysis, Xcalibur raw files were converted into the open mzXML format through MSConvert (Proteowizard) with a 32-bit precision. mzXML files were directly used as input for xQuest searches on a local xQuest installation<sup><a href="#ref53" title="Leitner, A., Walzthoeni, T. &amp; Aebersold, R. Lysine-specific chemical cross-linking of protein complexes and identification of cross-linking sites using LC-MS/MS and the xQuest/xProphet software pipeline. Nat. Protocols 9, 120–137 (2014)" id="ref-link-87">53</a></sup>. The selection of cross-linked precursor MS/MS data was based on the following criteria: a mass difference among the heavy and the light cross-linker of: 12.07532 Da for BS3, DSS and SDH, 6.02016 Da for DSA and 8.05016 Da for ADH; precursor charge ranging from 3+ to 8+; maximum retention time difference 2.5 min. Searches were performed against an ad hoc database containing all the sequences of ovine complex I subunits together with their reverse used as decoy database. The following parameters were set for xQuest searches: maximum number of missed cleavages (excluding the cross-linking site) 3; peptide length 4–50 amino acids; fixed modifications carbamidomethyl-Cys (mass shift 57.02146 Da); mass shift of the light cross-linker 138.06808 Da for DSS and BS3, 138.0906 Da for ADH, 110.03675 for DSA and 166.1218 for SDH; mass shift of mono-links 156.0786 and 155.0964 Da for DSS and BS3, 138.0906 Da for ADH, 127.0628 Da and 128.0468 Da for DSA, and 184.1324 Da for SDH; MS1 tolerance 10 ppm, MS2 tolerance 0.2 Da for common ions and 0.3 for cross-link ions; search in enumeration mode (exhaustive search). Search results were filtered according to the following criteria: MS1 mass tolerance window −3 to 7 ppm. Finally each MS/MS spectra was manually inspected and validated.</p><p>In total 218 unique cross-linked peptides were identified, of which 87 were between residues of different subunits (inter-subunit, <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Table 2</a>), 73 were between residues within the same subunit (intra-subunit, <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Table 3</a>), and 58 were clear false positives (<a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Table 4</a>). False positives were identified by comparison to all known biochemical and structural information on complex I and the cross-links that are considered false positives are either between residues that are too distant from each other (&#x0003E;32 Å after allowing for exposed side chain flexibility from their modelled position), located on opposite sides of the membrane or the reactive residues are buried and not solvent accessible in the intact structure. Many of the false positive cross-links are found on unstructured coils at the edges of the complex I structure indicating that they probably result from transient interactions between different complexes I during the reaction (inter-complex cross-links). True positive cross-links were more likely to be observed in more than one experiment. Some high-scoring cross-links were observed between disordered termini or loops of subunits that could not be modelled in our structure; hence the accurate determination of distance for these cross-links was not possible. Nonetheless in cases where cross-linking residues are adjacent to the modelled regions, the cross-links were considered true and are included in <a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html" class="draggable supp-fig" data-related-id="sf5">Extended Data Fig. 5</a> and <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Tables 2</a> and <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">3</a>. No cross-links were observed for any of the mitochondrially encoded core subunits, which are buried in the membrane and coated with a layer of supernumerary subunits. Good quality cross-links were observed for all supernumerary subunits expect for B14.7, KFYI and AGGG. These data in conjunction with our electron microscopy maps allowed us to unambiguously assign all supernumerary subunits. Previous assignments were confirmed and importantly, subunits that previously had no known position in the complex (10 kDa, B14.5a, MWFE, B9, MNLL, SGDH, ASHI, B17, AGGG and B12) have now been assigned and built (see <a href="/nature/journal/v538/n7625/full/nature19794.html#supplementary-information">Supplementary Discussion</a> for more details).</p>

																		
										
					
											</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="references">

									<h1 class="section-heading toggle">References</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7">References<span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
												
										
					

	
<ol class="references">
														
		<li id="ref1"><span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">A giant molecular proton pump: structure and mechanism of respiratory complex I</span>. <span class="source-title">Nat. Rev. Mol. Cell Biol.</span> <span class="volume">16</span>, <span class="start-page">375</span>–<span class="end-page">388</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXhtFWqtbfF&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=e7b55ad59d9a420d9ec8f61e8cbb0c46">CAS</a></li>
																															 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nrm3997&amp;rft_id=info:pmid/25991374&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Sazanov&amp;rft.aufirst=L. A.&amp;rft.jtitle=Nat. Rev. Mol. Cell Biol.&amp;rft.volume=16&amp;rft.spage=375&amp;rft.epage=388&amp;rft.date=2015&amp;rft.atitle=A giant molecular proton pump: structure and mechanism of respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nrm3997&amp;id=pmid:25991374&amp;genre=article&amp;aulast=Sazanov&amp;aufirst=L. A.&amp;title=Nat. Rev. Mol. Cell Biol.&amp;volume=16&amp;spage=375&amp;epage=388&amp;date=2015&amp;atitle=A giant molecular proton pump: structure and mechanism of respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25991374&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nrm3997">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref2"><span class="vcard author"><span class="fn">Moser, C. C.</span></span>, <span class="vcard author"><span class="fn">Farid, T. A.</span></span>, <span class="vcard author"><span class="fn">Chobot, S. E.</span></span> &amp; <span class="vcard author"><span class="fn">Dutton, P. L.</span></span> <span class="title">Electron tunneling chains of mitochondria</span>. <span class="source-title">Biochim. Biophys. Acta</span> <span class="volume">1757</span>, <span class="start-page">1096</span>–<span class="end-page">1109</span> (<span class="year">2006</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XhtVanurvJ&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=3d985111a3c98d567ed646ef44202831">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.bbabio.2006.04.015&amp;rft_id=info:pmid/16780790&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Moser&amp;rft.aufirst=C. C.&amp;rft.jtitle=Biochim. Biophys. Acta&amp;rft.volume=1757&amp;rft.spage=1096&amp;rft.epage=1109&amp;rft.date=2006&amp;rft.atitle=Electron tunneling chains of mitochondria&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.bbabio.2006.04.015&amp;id=pmid:16780790&amp;genre=article&amp;aulast=Moser&amp;aufirst=C. C.&amp;title=Biochim. Biophys. Acta&amp;volume=1757&amp;spage=1096&amp;epage=1109&amp;date=2006&amp;atitle=Electron tunneling chains of mitochondria&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=16780790&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.bbabio.2006.04.015">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref3"><span class="vcard author"><span class="fn">Letts, J. A.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">Gaining mass: the structure of respiratory complex I-from bacterial towards mitochondrial versions</span>. <span class="source-title">Curr. Opin. Struct. Biol.</span> <span class="volume">33</span>, <span class="start-page">135</span>–<span class="end-page">145</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.sbi.2015.08.008&amp;rft_id=info:pmid/26387075&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Letts&amp;rft.aufirst=J. A.&amp;rft.jtitle=Curr. Opin. Struct. Biol.&amp;rft.volume=33&amp;rft.spage=135&amp;rft.epage=145&amp;rft.date=2015&amp;rft.atitle=Gaining mass: the structure of respiratory complex I-from bacterial towards mitochondrial versions&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.sbi.2015.08.008&amp;id=pmid:26387075&amp;genre=article&amp;aulast=Letts&amp;aufirst=J. A.&amp;title=Curr. Opin. Struct. Biol.&amp;volume=33&amp;spage=135&amp;epage=145&amp;date=2015&amp;atitle=Gaining mass: the structure of respiratory complex I-from bacterial towards mitochondrial versions&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=26387075&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.sbi.2015.08.008">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref4"><span class="vcard author"><span class="fn">Fassone, E.</span></span> &amp; <span class="vcard author"><span class="fn">Rahman, S.</span></span> <span class="title">Complex I deficiency: clinical features, biochemistry and molecular genetics</span>. <span class="source-title">J. Med. Genet.</span> <span class="volume">49</span>, <span class="start-page">578</span>–<span class="end-page">590</span> (<span class="year">2012</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC38XhvVCitrzE&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=0d9d2751d4e6d5f86ba044747cac317e">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1136/jmedgenet-2012-101159&amp;rft_id=info:pmid/22972949&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Fassone&amp;rft.aufirst=E.&amp;rft.jtitle=J. Med. Genet.&amp;rft.volume=49&amp;rft.spage=578&amp;rft.epage=590&amp;rft.date=2012&amp;rft.atitle=Complex I deficiency: clinical features, biochemistry and molecular genetics&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1136/jmedgenet-2012-101159&amp;id=pmid:22972949&amp;genre=article&amp;aulast=Fassone&amp;aufirst=E.&amp;title=J. Med. Genet.&amp;volume=49&amp;spage=578&amp;epage=590&amp;date=2012&amp;atitle=Complex I deficiency: clinical features, biochemistry and molecular genetics&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000308805200006&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22972949&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1136/jmedgenet-2012-101159">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref5"><span class="vcard author"><span class="fn">Walker, J. E.</span></span> <span class="title">The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains</span>. <span class="source-title">Q. Rev. Biophys.</span> <span class="volume">25</span>, <span class="start-page">253</span>–<span class="end-page">324</span> (<span class="year">1992</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3sXps1KktQ%3D%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=21d8ade9dac5087feaf265ebb800b886">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1017/S003358350000425X&amp;rft_id=info:pmid/1470679&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Walker&amp;rft.aufirst=J. E.&amp;rft.jtitle=Q. Rev. Biophys.&amp;rft.volume=25&amp;rft.spage=253&amp;rft.epage=324&amp;rft.date=1992&amp;rft.atitle=The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1017/S003358350000425X&amp;id=pmid:1470679&amp;genre=article&amp;aulast=Walker&amp;aufirst=J. E.&amp;title=Q. Rev. Biophys.&amp;volume=25&amp;spage=253&amp;epage=324&amp;date=1992&amp;atitle=The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=A1992KE07200001&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=1470679&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1017/S003358350000425X">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref6"><span class="vcard author"><span class="fn">Vinothkumar, K. R.</span></span>, <span class="vcard author"><span class="fn">Zhu, J.</span></span> &amp; <span class="vcard author"><span class="fn">Hirst, J.</span></span> <span class="title">Architecture of mammalian respiratory complex I</span>. <span class="source-title">Nature</span> <span class="volume">515</span>, <span class="start-page">80</span>–<span class="end-page">84</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2cXhvVemtrvN&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=4572a5029d67bd650bcc9ce003dadfb0">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nature13686&amp;rft_id=info:pmid/25209663&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Vinothkumar&amp;rft.aufirst=K. R.&amp;rft.jtitle=Nature&amp;rft.volume=515&amp;rft.spage=80&amp;rft.epage=84&amp;rft.date=2014&amp;rft.atitle=Architecture of mammalian respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nature13686&amp;id=pmid:25209663&amp;genre=article&amp;aulast=Vinothkumar&amp;aufirst=K. R.&amp;title=Nature&amp;volume=515&amp;spage=80&amp;epage=84&amp;date=2014&amp;atitle=Architecture of mammalian respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000344187500032&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25209663&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nature13686">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref7"><span class="vcard author"><span class="fn">Galkin, A.</span></span>, <span class="vcard author"><span class="fn">Dröse, S.</span></span> &amp; <span class="vcard author"><span class="fn">Brandt, U.</span></span> <span class="title">The proton pumping stoichiometry of purified mitochondrial complex I reconstituted into proteoliposomes</span>. <span class="source-title">Biochim. Biophys. Acta</span> <span class="volume">1757</span>, <span class="start-page">1575</span>–<span class="end-page">1581</span> (<span class="year">2006</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28Xht12kt77M&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=7a08004e69be32dbbbb6f7eecd49f696">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.bbabio.2006.10.001&amp;rft_id=info:pmid/17094937&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Galkin&amp;rft.aufirst=A.&amp;rft.jtitle=Biochim. Biophys. Acta&amp;rft.volume=1757&amp;rft.spage=1575&amp;rft.epage=1581&amp;rft.date=2006&amp;rft.atitle=The proton pumping stoichiometry of purified mitochondrial complex I reconstituted into proteoliposomes&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.bbabio.2006.10.001&amp;id=pmid:17094937&amp;genre=article&amp;aulast=Galkin&amp;aufirst=A.&amp;title=Biochim. Biophys. Acta&amp;volume=1757&amp;spage=1575&amp;epage=1581&amp;date=2006&amp;atitle=The proton pumping stoichiometry of purified mitochondrial complex I reconstituted into proteoliposomes&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=17094937&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.bbabio.2006.10.001">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref8"><span class="vcard author"><span class="fn">Wikström, M.</span></span>, <span class="vcard author"><span class="fn">Sharma, V.</span></span>, <span class="vcard author"><span class="fn">Kaila, V. R.</span></span>, <span class="vcard author"><span class="fn">Hosler, J. P.</span></span> &amp; <span class="vcard author"><span class="fn">Hummer, G.</span></span> <span class="title">New perspectives on proton pumping in cellular respiration</span>. <span class="source-title">Chem. Rev.</span> <span class="volume">115</span>, <span class="start-page">2196</span>–<span class="end-page">2221</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXjtVSjs7Y%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=2b37ee3f8a4d250185b1940edd968194">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1021/cr500448t&amp;rft_id=info:pmid/25694135&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Wikstr&ouml;m&amp;rft.aufirst=M.&amp;rft.jtitle=Chem. Rev.&amp;rft.volume=115&amp;rft.spage=2196&amp;rft.epage=2221&amp;rft.date=2015&amp;rft.atitle=New perspectives on proton pumping in cellular respiration&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1021/cr500448t&amp;id=pmid:25694135&amp;genre=article&amp;aulast=Wikstr&ouml;m&amp;aufirst=M.&amp;title=Chem. Rev.&amp;volume=115&amp;spage=2196&amp;epage=2221&amp;date=2015&amp;atitle=New perspectives on proton pumping in cellular respiration&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25694135&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1021/cr500448t">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref9"><span class="vcard author"><span class="fn">Baradaran, R.</span></span>, <span class="vcard author"><span class="fn">Berrisford, J. M.</span></span>, <span class="vcard author"><span class="fn">Minhas, G. S.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">Crystal structure of the entire respiratory complex I</span>. <span class="source-title">Nature</span> <span class="volume">494</span>, <span class="start-page">443</span>–<span class="end-page">448</span> (<span class="year">2013</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3sXjtFWiu70%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=92b132e295fe543e8172290e95ec17b3">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nature11871&amp;rft_id=info:pmid/23417064&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Baradaran&amp;rft.aufirst=R.&amp;rft.jtitle=Nature&amp;rft.volume=494&amp;rft.spage=443&amp;rft.epage=448&amp;rft.date=2013&amp;rft.atitle=Crystal structure of the entire respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nature11871&amp;id=pmid:23417064&amp;genre=article&amp;aulast=Baradaran&amp;aufirst=R.&amp;title=Nature&amp;volume=494&amp;spage=443&amp;epage=448&amp;date=2013&amp;atitle=Crystal structure of the entire respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000315661500030&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=23417064&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nature11871">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref10"><span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> &amp; <span class="vcard author"><span class="fn">Hinchliffe, P.</span></span> <span class="title">Structure of the hydrophilic domain of respiratory complex I from <i>Thermus thermophilus</i></span>. <span class="source-title">Science</span> <span class="volume">311</span>, <span class="start-page">1430</span>–<span class="end-page">1436</span> (<span class="year">2006</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XitVyjsbg%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=0a94a049b21243d255c97ec5247ec154">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1126/science.1123809&amp;rft_id=info:pmid/16469879&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Sazanov&amp;rft.aufirst=L. A.&amp;rft.jtitle=Science&amp;rft.volume=311&amp;rft.spage=1430&amp;rft.epage=1436&amp;rft.date=2006&amp;rft.atitle=Structure of the hydrophilic domain of respiratory complex I from&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1126/science.1123809&amp;id=pmid:16469879&amp;genre=article&amp;aulast=Sazanov&amp;aufirst=L. A.&amp;title=Science&amp;volume=311&amp;spage=1430&amp;epage=1436&amp;date=2006&amp;atitle=Structure of the hydrophilic domain of respiratory complex I from&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000236029400040&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=16469879&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1126/science.1123809">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref11"><span class="vcard author"><span class="fn">Efremov, R. G.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">Structure of the membrane domain of respiratory complex I</span>. <span class="source-title">Nature</span> <span class="volume">476</span>, <span class="start-page">414</span>–<span class="end-page">420</span> (<span class="year">2011</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXhtVOmsL%2FJ&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=6e8581af552ee73d95ada4185a87d21c">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nature10330&amp;rft_id=info:pmid/21822288&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Efremov&amp;rft.aufirst=R. G.&amp;rft.jtitle=Nature&amp;rft.volume=476&amp;rft.spage=414&amp;rft.epage=420&amp;rft.date=2011&amp;rft.atitle=Structure of the membrane domain of respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nature10330&amp;id=pmid:21822288&amp;genre=article&amp;aulast=Efremov&amp;aufirst=R. G.&amp;title=Nature&amp;volume=476&amp;spage=414&amp;epage=420&amp;date=2011&amp;atitle=Structure of the membrane domain of respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000294209400028&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=21822288&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nature10330">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref12"><span class="vcard author"><span class="fn">Zickermann, V.</span></span> <i>et al</i>. <span class="title">Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I</span>. <span class="source-title">Science</span> <span class="volume">347</span>, <span class="start-page">44</span>–<span class="end-page">49</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXhsF2jsg%3D%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=b46de41d5ff47fc5b836f58a04202a68">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1126/science.1259859&amp;rft_id=info:pmid/25554780&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Zickermann&amp;rft.aufirst=V.&amp;rft.jtitle=Science&amp;rft.volume=347&amp;rft.spage=44&amp;rft.epage=49&amp;rft.date=2015&amp;rft.atitle=Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1126/science.1259859&amp;id=pmid:25554780&amp;genre=article&amp;aulast=Zickermann&amp;aufirst=V.&amp;title=Science&amp;volume=347&amp;spage=44&amp;epage=49&amp;date=2015&amp;atitle=Structural biology. Mechanistic insight from the crystal structure of mitochondrial complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000347102300043&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25554780&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1126/science.1259859">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref13"><span class="vcard author"><span class="fn">Zhu, J.</span></span> <i>et al</i>. <span class="title">Structure of subcomplex Iβ of mammalian respiratory complex I leads to new supernumerary subunit assignments</span>. <span class="source-title">Proc. Natl Acad. Sci. USA</span> <span class="volume">112</span>, <span class="start-page">12087</span>–<span class="end-page">12092</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1073/pnas.1510577112&amp;rft_id=info:pmid/26371297&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Zhu&amp;rft.aufirst=J.&amp;rft.jtitle=Proc. Natl Acad. Sci. USA&amp;rft.volume=112&amp;rft.spage=12087&amp;rft.epage=12092&amp;rft.date=2015&amp;rft.atitle=Structure of subcomplex I&beta; of mammalian respiratory complex I leads to new supernumerary subunit assignments&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1073/pnas.1510577112&amp;id=pmid:26371297&amp;genre=article&amp;aulast=Zhu&amp;aufirst=J.&amp;title=Proc. Natl Acad. Sci. USA&amp;volume=112&amp;spage=12087&amp;epage=12092&amp;date=2015&amp;atitle=Structure of subcomplex I&beta; of mammalian respiratory complex I leads to new supernumerary subunit assignments&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=26371297&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1073/pnas.1510577112">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref14"><span class="vcard author"><span class="fn">DiMaio, F.</span></span> <i>et al</i>. <span class="title">Atomic-accuracy models from 4.5-Å cryo-electron microscopy data with density-guided iterative local refinement</span>. <span class="source-title">Nat. Methods</span> <span class="volume">12</span>, <span class="start-page">361</span>–<span class="end-page">365</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXjtFems78%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=4bda0a413b78c71390a724e69ec17af7">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nmeth.3286&amp;rft_id=info:pmid/25707030&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=DiMaio&amp;rft.aufirst=F.&amp;rft.jtitle=Nat. Methods&amp;rft.volume=12&amp;rft.spage=361&amp;rft.epage=365&amp;rft.date=2015&amp;rft.atitle=Atomic-accuracy models from 4.5-&amp;Aring; cryo-electron microscopy data with density-guided iterative local refinement&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nmeth.3286&amp;id=pmid:25707030&amp;genre=article&amp;aulast=DiMaio&amp;aufirst=F.&amp;title=Nat. Methods&amp;volume=12&amp;spage=361&amp;epage=365&amp;date=2015&amp;atitle=Atomic-accuracy models from 4.5-&amp;Aring; cryo-electron microscopy data with density-guided iterative local refinement&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25707030&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nmeth.3286">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref15"><span class="vcard author"><span class="fn">Zhu, J.</span></span>, <span class="vcard author"><span class="fn">Vinothkumar, K. R.</span></span> &amp; <span class="vcard author"><span class="fn">Hirst, J.</span></span> <span class="title">Structure of mammalian respiratory complex I</span>. <span class="source-title">Nature</span> <span class="volume">536</span>, <span class="start-page">354</span>–<span class="end-page">358</span> (<span class="year">2016</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nature19095&amp;rft_id=info:pmid/27509854&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Zhu&amp;rft.aufirst=J.&amp;rft.jtitle=Nature&amp;rft.volume=536&amp;rft.spage=354&amp;rft.epage=358&amp;rft.date=2016&amp;rft.atitle=Structure of mammalian respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nature19095&amp;id=pmid:27509854&amp;genre=article&amp;aulast=Zhu&amp;aufirst=J.&amp;title=Nature&amp;volume=536&amp;spage=354&amp;epage=358&amp;date=2016&amp;atitle=Structure of mammalian respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=27509854&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nature19095">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref16"><span class="vcard author"><span class="fn">Letts, J. A.</span></span>, <span class="vcard author"><span class="fn">Fiedorczuk, K.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">Architecture of respiratory supercomplexes</span>. <span class="source-title">Nature</span> <a href="http://dx.doi.org/10.1038/nature19774">http://dx.doi.org/10.1038/nature19774</a> (<span class="year">2016</span>)
				</li>
		
								
														
		<li id="ref17"><span class="vcard author"><span class="fn">Carroll, J.</span></span>, <span class="vcard author"><span class="fn">Fearnley, I. M.</span></span>, <span class="vcard author"><span class="fn">Shannon, R. J.</span></span>, <span class="vcard author"><span class="fn">Hirst, J.</span></span> &amp; <span class="vcard author"><span class="fn">Walker, J. E.</span></span> <span class="title">Analysis of the subunit composition of complex I from bovine heart mitochondria</span>. <span class="volume"><!-- . --></span><span class="source-title">Mol. Cell. Proteomics</span> <span class="volume">2</span>, <span class="start-page">117</span>–<span class="end-page">126</span> (<span class="year">2003</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXivFWnt70%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=25c92e4cf9c2eb1ffa867971f580fe60">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1074/mcp.M300014-MCP200&amp;rft_id=info:pmid/12644575&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Carroll&amp;rft.aufirst=J.&amp;rft.jtitle=Mol. Cell. Proteomics&amp;rft.volume=&amp;rft.spage=117&amp;rft.epage=126&amp;rft.date=2003&amp;rft.atitle=Analysis of the subunit composition of complex I from bovine heart mitochondria&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1074/mcp.M300014-MCP200&amp;id=pmid:12644575&amp;genre=article&amp;aulast=Carroll&amp;aufirst=J.&amp;title=Mol. Cell. Proteomics&amp;volume=&amp;spage=117&amp;epage=126&amp;date=2003&amp;atitle=Analysis of the subunit composition of complex I from bovine heart mitochondria&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000182128300006&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=12644575&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1074/mcp.M300014-MCP200">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref18"><span class="vcard author"><span class="fn">Chouchani, E. T.</span></span> <i>et al</i>. <span class="title">Cardioprotection by <i>S</i>-nitrosation of a cysteine switch on mitochondrial complex I. Nat</span>. <span class="source-title">Med</span>. <span class="volume">19</span>, <span class="start-page">753</span>–<span class="end-page">759</span> (<span class="year">2013</span>)
				</li>
		
								
														
		<li id="ref19"><span class="vcard author"><span class="fn">Vinogradov, A. D.</span></span> &amp; <span class="vcard author"><span class="fn">Grivennikova, V. G.</span></span> <span class="title">The mitochondrial complex I: progress in understanding of catalytic properties</span>. <span class="source-title">IUBMB Life</span> <span class="volume">52</span>, <span class="start-page">129</span>–<span class="end-page">134</span> (<span class="year">2001</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1080/15216540152845920&amp;rft_id=info:pmid/11798024&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Vinogradov&amp;rft.aufirst=A. D.&amp;rft.jtitle=IUBMB Life&amp;rft.volume=52&amp;rft.spage=129&amp;rft.epage=134&amp;rft.date=2001&amp;rft.atitle=The mitochondrial complex I: progress in understanding of catalytic properties&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1080/15216540152845920&amp;id=pmid:11798024&amp;genre=article&amp;aulast=Vinogradov&amp;aufirst=A. D.&amp;title=IUBMB Life&amp;volume=52&amp;spage=129&amp;epage=134&amp;date=2001&amp;atitle=The mitochondrial complex I: progress in understanding of catalytic properties&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=11798024&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1080/15216540152845920">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref20"><span class="vcard author"><span class="fn">Sánchez-Caballero, L.</span></span>, <span class="vcard author"><span class="fn">Guerrero-Castillo, S.</span></span> &amp; <span class="vcard author"><span class="fn">Nijtmans, L.</span></span> <span class="title">Unraveling the complexity of mitochondrial complex I assembly: A dynamic process</span>. <span class="source-title">Biochim. Biophys. Acta</span> <span class="volume">1857</span>, <span class="start-page">980</span>–<span class="end-page">990</span> (<span class="year">2016</span>)
					<ul class="cleared">
																															 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.bbabio.2016.03.031&amp;rft_id=info:pmid/{pubmed}&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=S&aacute;nchez-Caballero&amp;rft.aufirst=L.&amp;rft.jtitle=Biochim. Biophys. Acta&amp;rft.volume=1857&amp;rft.spage=980&amp;rft.epage=990&amp;rft.date=2016&amp;rft.atitle=Unraveling the complexity of mitochondrial complex I assembly: A dynamic process&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.bbabio.2016.03.031&amp;id=pmid:{pubmed}&amp;genre=article&amp;aulast=S&aacute;nchez-Caballero&amp;aufirst=L.&amp;title=Biochim. Biophys. Acta&amp;volume=1857&amp;spage=980&amp;epage=990&amp;date=2016&amp;atitle=Unraveling the complexity of mitochondrial complex I assembly: A dynamic process&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
											<a href="http://dx.doi.org/10.1016/j.bbabio.2016.03.031">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref21"><span class="vcard author"><span class="fn">Zickermann, V.</span></span>, <span class="vcard author"><span class="fn">Angerer, H.</span></span>, <span class="vcard author"><span class="fn">Ding, M. G.</span></span>, <span class="vcard author"><span class="fn">Nübel, E.</span></span> &amp; <span class="vcard author"><span class="fn">Brandt, U.</span></span> <span class="title">Small single transmembrane domain (STMD) proteins organize the hydrophobic subunits of large membrane protein complexes</span>. <span class="source-title">FEBS Lett.</span> <span class="volume">584</span>, <span class="start-page">2516</span>–<span class="end-page">2525</span> (<span class="year">2010</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3cXmslGisro%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=b5dbd1e5e6b229d488debdb3b8728123">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.febslet.2010.04.021&amp;rft_id=info:pmid/20398659&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Zickermann&amp;rft.aufirst=V.&amp;rft.jtitle=FEBS Lett.&amp;rft.volume=584&amp;rft.spage=2516&amp;rft.epage=2525&amp;rft.date=2010&amp;rft.atitle=Small single transmembrane domain (STMD) proteins organize the hydrophobic subunits of large membrane protein complexes&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.febslet.2010.04.021&amp;id=pmid:20398659&amp;genre=article&amp;aulast=Zickermann&amp;aufirst=V.&amp;title=FEBS Lett.&amp;volume=584&amp;spage=2516&amp;epage=2525&amp;date=2010&amp;atitle=Small single transmembrane domain (STMD) proteins organize the hydrophobic subunits of large membrane protein complexes&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000278031800005&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=20398659&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.febslet.2010.04.021">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref22"><span class="vcard author"><span class="fn">Cronan, J. E.</span></span> <span class="title">The chain-flipping mechanism of ACP (acyl carrier protein)-dependent enzymes appears universal</span>. <span class="source-title">Biochem. J.</span> <span class="volume">460</span>, <span class="start-page">157</span>–<span class="end-page">163</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1042/BJ20140239&amp;rft_id=info:pmid/24825445&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Cronan&amp;rft.aufirst=J. E.&amp;rft.jtitle=Biochem. J.&amp;rft.volume=460&amp;rft.spage=157&amp;rft.epage=163&amp;rft.date=2014&amp;rft.atitle=The chain-flipping mechanism of ACP (acyl carrier protein)-dependent enzymes appears universal&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1042/BJ20140239&amp;id=pmid:24825445&amp;genre=article&amp;aulast=Cronan&amp;aufirst=J. E.&amp;title=Biochem. J.&amp;volume=460&amp;spage=157&amp;epage=163&amp;date=2014&amp;atitle=The chain-flipping mechanism of ACP (acyl carrier protein)-dependent enzymes appears universal&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=24825445&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1042/BJ20140239">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref23"><span class="vcard author"><span class="fn">Angerer, H.</span></span> <span class="title">Eukaryotic LYR proteins interact with mitochondrial protein complexes</span>. <span class="source-title">Biology (Basel)</span> <span class="volume">4</span>, <span class="start-page">133</span>–<span class="end-page">150</span> (<span class="year">2015</span>)
					<ul class="cleared">
																								<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25686363&amp;amp;dopt=Abstract">PubMed</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref24"><span class="vcard author"><span class="fn">Oteiza, P. I.</span></span> <span class="title">Zinc and the modulation of redox homeostasis</span>. <span class="source-title">Free Radic. Biol. Med.</span> <span class="volume">53</span>, <span class="start-page">1748</span>–<span class="end-page">1759</span> (<span class="year">2012</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.freeradbiomed.2012.08.568&amp;rft_id=info:pmid/22960578&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Oteiza&amp;rft.aufirst=P. I.&amp;rft.jtitle=Free Radic. Biol. Med.&amp;rft.volume=53&amp;rft.spage=1748&amp;rft.epage=1759&amp;rft.date=2012&amp;rft.atitle=Zinc and the modulation of redox homeostasis&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.freeradbiomed.2012.08.568&amp;id=pmid:22960578&amp;genre=article&amp;aulast=Oteiza&amp;aufirst=P. I.&amp;title=Free Radic. Biol. Med.&amp;volume=53&amp;spage=1748&amp;epage=1759&amp;date=2012&amp;atitle=Zinc and the modulation of redox homeostasis&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22960578&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.freeradbiomed.2012.08.568">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref25"><span class="vcard author"><span class="fn">Kmita, K.</span></span> <i>et al</i>. <span class="title">Accessory NUMM (NDUFS6) subunit harbors a Zn-binding site and is essential for biogenesis of mitochondrial complex I</span>. <span class="source-title">Proc. Natl Acad. Sci. USA</span> <span class="volume">112</span>, <span class="start-page">5685</span>–<span class="end-page">5690</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1073/pnas.1424353112&amp;rft_id=info:pmid/25902503&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Kmita&amp;rft.aufirst=K.&amp;rft.jtitle=Proc. Natl Acad. Sci. USA&amp;rft.volume=112&amp;rft.spage=5685&amp;rft.epage=5690&amp;rft.date=2015&amp;rft.atitle=Accessory NUMM (NDUFS6) subunit harbors a Zn-binding site and is essential for biogenesis of mitochondrial complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1073/pnas.1424353112&amp;id=pmid:25902503&amp;genre=article&amp;aulast=Kmita&amp;aufirst=K.&amp;title=Proc. Natl Acad. Sci. USA&amp;volume=112&amp;spage=5685&amp;epage=5690&amp;date=2015&amp;atitle=Accessory NUMM (NDUFS6) subunit harbors a Zn-binding site and is essential for biogenesis of mitochondrial complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25902503&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1073/pnas.1424353112">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref26"><span class="vcard author"><span class="fn">Sharpley, M. S.</span></span>, <span class="vcard author"><span class="fn">Shannon, R. J.</span></span>, <span class="vcard author"><span class="fn">Draghi, F.</span></span> &amp; <span class="vcard author"><span class="fn">Hirst, J.</span></span> <span class="title">Interactions between phospholipids and NADH:ubiquinone oxidoreductase (complex I) from bovine mitochondria</span>. <span class="source-title">Biochemistry</span> <span class="volume">45</span>, <span class="start-page">241</span>–<span class="end-page">248</span> (<span class="year">2006</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXht12jsb%2FF&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=518c4b87136f2fee5ebd229352dbd341">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1021/bi051809x&amp;rft_id=info:pmid/16388600&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Sharpley&amp;rft.aufirst=M. S.&amp;rft.jtitle=Biochemistry&amp;rft.volume=45&amp;rft.spage=241&amp;rft.epage=248&amp;rft.date=2006&amp;rft.atitle=Interactions between phospholipids and NADH:ubiquinone oxidoreductase (complex I) from bovine mitochondria&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1021/bi051809x&amp;id=pmid:16388600&amp;genre=article&amp;aulast=Sharpley&amp;aufirst=M. S.&amp;title=Biochemistry&amp;volume=45&amp;spage=241&amp;epage=248&amp;date=2006&amp;atitle=Interactions between phospholipids and NADH:ubiquinone oxidoreductase (complex I) from bovine mitochondria&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000234694800026&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=16388600&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1021/bi051809x">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref27"><span class="vcard author"><span class="fn">Brandt, U.</span></span> <span class="title">Energy converting NADH:quinone oxidoreductase (complex I)</span>. <span class="source-title">Annu. Rev. Biochem.</span> <span class="volume">75</span>, <span class="start-page">69</span>–<span class="end-page">92</span> (<span class="year">2006</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XosVKhsr0%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=5b2410721738e82ce3ca98b7cc13f402">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1146/annurev.biochem.75.103004.142539&amp;rft_id=info:pmid/16756485&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Brandt&amp;rft.aufirst=U.&amp;rft.jtitle=Annu. Rev. Biochem.&amp;rft.volume=75&amp;rft.spage=69&amp;rft.epage=92&amp;rft.date=2006&amp;rft.atitle=Energy converting NADH:quinone oxidoreductase (complex I)&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1146/annurev.biochem.75.103004.142539&amp;id=pmid:16756485&amp;genre=article&amp;aulast=Brandt&amp;aufirst=U.&amp;title=Annu. Rev. Biochem.&amp;volume=75&amp;spage=69&amp;epage=92&amp;date=2006&amp;atitle=Energy converting NADH:quinone oxidoreductase (complex I)&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000239807600004&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=16756485&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1146/annurev.biochem.75.103004.142539">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref28"><span class="vcard author"><span class="fn">Belevich, G.</span></span>, <span class="vcard author"><span class="fn">Knuuti, J.</span></span>, <span class="vcard author"><span class="fn">Verkhovsky, M. I.</span></span>, <span class="vcard author"><span class="fn">Wikström, M.</span></span> &amp; <span class="vcard author"><span class="fn">Verkhovskaya, M.</span></span> <span class="title">Probing the mechanistic role of the long α-helix in subunit L of respiratory Complex I from <i>Escherichia coli</i> by site-directed mutagenesis</span>. <span class="source-title">Mol. Microbiol.</span> <span class="volume">82</span>, <span class="start-page">1086</span>–<span class="end-page">1095</span> (<span class="year">2011</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXhs1antrjO&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=3f42bdc908c5858b0fa432c468008802">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1111/j.1365-2958.2011.07883.x&amp;rft_id=info:pmid/22060017&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Belevich&amp;rft.aufirst=G.&amp;rft.jtitle=Mol. Microbiol.&amp;rft.volume=82&amp;rft.spage=1086&amp;rft.epage=1095&amp;rft.date=2011&amp;rft.atitle=Probing the mechanistic role of the long &alpha;-helix in subunit L of respiratory Complex I from&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1111/j.1365-2958.2011.07883.x&amp;id=pmid:22060017&amp;genre=article&amp;aulast=Belevich&amp;aufirst=G.&amp;title=Mol. Microbiol.&amp;volume=82&amp;spage=1086&amp;epage=1095&amp;date=2011&amp;atitle=Probing the mechanistic role of the long &alpha;-helix in subunit L of respiratory Complex I from&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000298087200005&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22060017&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1111/j.1365-2958.2011.07883.x">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref29"><span class="vcard author"><span class="fn">Efremov, R. G.</span></span>, <span class="vcard author"><span class="fn">Baradaran, R.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">The architecture of respiratory complex I</span>. <span class="source-title">Nature</span> <span class="volume">465</span>, <span class="start-page">441</span>–<span class="end-page">445</span> (<span class="year">2010</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3cXmsF2itbc%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=4b206b437ec450637200f2ba838e1d06">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nature09066&amp;rft_id=info:pmid/20505720&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Efremov&amp;rft.aufirst=R. G.&amp;rft.jtitle=Nature&amp;rft.volume=465&amp;rft.spage=441&amp;rft.epage=445&amp;rft.date=2010&amp;rft.atitle=The architecture of respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nature09066&amp;id=pmid:20505720&amp;genre=article&amp;aulast=Efremov&amp;aufirst=R. G.&amp;title=Nature&amp;volume=465&amp;spage=441&amp;epage=445&amp;date=2010&amp;atitle=The architecture of respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000278043700026&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=20505720&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nature09066">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref30"><span class="vcard author"><span class="fn">Palmisano, G.</span></span>, <span class="vcard author"><span class="fn">Sardanelli, A. M.</span></span>, <span class="vcard author"><span class="fn">Signorile, A.</span></span>, <span class="vcard author"><span class="fn">Papa, S.</span></span> &amp; <span class="vcard author"><span class="fn">Larsen, M. R.</span></span> <span class="title">The phosphorylation pattern of bovine heart complex I subunits</span>. <span class="source-title">Proteomics</span> <span class="volume">7</span>, <span class="start-page">1575</span>–<span class="end-page">1583</span> (<span class="year">2007</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1002/pmic.200600801&amp;rft_id=info:pmid/17443843&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Palmisano&amp;rft.aufirst=G.&amp;rft.jtitle=Proteomics&amp;rft.volume=7&amp;rft.spage=1575&amp;rft.epage=1583&amp;rft.date=2007&amp;rft.atitle=The phosphorylation pattern of bovine heart complex I subunits&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1002/pmic.200600801&amp;id=pmid:17443843&amp;genre=article&amp;aulast=Palmisano&amp;aufirst=G.&amp;title=Proteomics&amp;volume=7&amp;spage=1575&amp;epage=1583&amp;date=2007&amp;atitle=The phosphorylation pattern of bovine heart complex I subunits&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=17443843&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1002/pmic.200600801">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref31"><span class="vcard author"><span class="fn">Berrisford, J. M.</span></span> &amp; <span class="vcard author"><span class="fn">Sazanov, L. A.</span></span> <span class="title">Structural basis for the mechanism of respiratory complex I</span>. <span class="source-title">J. Biol. Chem.</span> <span class="volume">284</span>, <span class="start-page">29773</span>–<span class="end-page">29783</span> (<span class="year">2009</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1MXht1KqurjK&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=df07ff58cb2c26392a88fe13c57cc957">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1074/jbc.M109.032144&amp;rft_id=info:pmid/19635800&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Berrisford&amp;rft.aufirst=J. M.&amp;rft.jtitle=J. Biol. Chem.&amp;rft.volume=284&amp;rft.spage=29773&amp;rft.epage=29783&amp;rft.date=2009&amp;rft.atitle=Structural basis for the mechanism of respiratory complex I&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1074/jbc.M109.032144&amp;id=pmid:19635800&amp;genre=article&amp;aulast=Berrisford&amp;aufirst=J. M.&amp;title=J. Biol. Chem.&amp;volume=284&amp;spage=29773&amp;epage=29783&amp;date=2009&amp;atitle=Structural basis for the mechanism of respiratory complex I&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000270896800058&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=19635800&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1074/jbc.M109.032144">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref32"><span class="vcard author"><span class="fn">Sazanov, L. A.</span></span>, <span class="vcard author"><span class="fn">Peak-Chew, S. Y.</span></span>, <span class="vcard author"><span class="fn">Fearnley, I. M.</span></span> &amp; <span class="vcard author"><span class="fn">Walker, J. E.</span></span> <span class="title">Resolution of the membrane domain of bovine complex I into subcomplexes: implications for the structural organization of the enzyme</span>. <span class="source-title">Biochemistry</span> <span class="volume">39</span>, <span class="start-page">7229</span>–<span class="end-page">7235</span> (<span class="year">2000</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXjsVaqsbk%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=46ede7dd5aae5e5269dc6825e02a8385">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1021/bi000335t&amp;rft_id=info:pmid/10852722&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Sazanov&amp;rft.aufirst=L. A.&amp;rft.jtitle=Biochemistry&amp;rft.volume=39&amp;rft.spage=7229&amp;rft.epage=7235&amp;rft.date=2000&amp;rft.atitle=Resolution of the membrane domain of bovine complex I into subcomplexes: implications for the structural organization of the enzyme&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1021/bi000335t&amp;id=pmid:10852722&amp;genre=article&amp;aulast=Sazanov&amp;aufirst=L. A.&amp;title=Biochemistry&amp;volume=39&amp;spage=7229&amp;epage=7235&amp;date=2000&amp;atitle=Resolution of the membrane domain of bovine complex I into subcomplexes: implications for the structural organization of the enzyme&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000087778300023&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=10852722&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1021/bi000335t">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref33"><span class="vcard author"><span class="fn">Scheres, S. H.</span></span> <span class="title">RELION: implementation of a Bayesian approach to cryo-EM structure determination</span>. <span class="source-title">J. Struct. Biol.</span> <span class="volume">180</span>, <span class="start-page">519</span>–<span class="end-page">530</span> (<span class="year">2012</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC38Xhs12jsLvO&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=eb59bb151c6bf18831e248f460ea7c73">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.jsb.2012.09.006&amp;rft_id=info:pmid/23000701&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Scheres&amp;rft.aufirst=S. H.&amp;rft.jtitle=J. Struct. Biol.&amp;rft.volume=180&amp;rft.spage=519&amp;rft.epage=530&amp;rft.date=2012&amp;rft.atitle=RELION: implementation of a Bayesian approach to cryo-EM structure determination&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.jsb.2012.09.006&amp;id=pmid:23000701&amp;genre=article&amp;aulast=Scheres&amp;aufirst=S. H.&amp;title=J. Struct. Biol.&amp;volume=180&amp;spage=519&amp;epage=530&amp;date=2012&amp;atitle=RELION: implementation of a Bayesian approach to cryo-EM structure determination&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000311471000014&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=23000701&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.jsb.2012.09.006">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref34"><span class="vcard author"><span class="fn">Rohou, A.</span></span> &amp; <span class="vcard author"><span class="fn">Grigorieff, N.</span></span> <span class="title">CTFFIND4: Fast and accurate defocus estimation from electron micrographs</span>. <span class="source-title">J. Struct. Biol.</span> <span class="volume">192</span>, <span class="start-page">216</span>–<span class="end-page">221</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.jsb.2015.08.008&amp;rft_id=info:pmid/26278980&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Rohou&amp;rft.aufirst=A.&amp;rft.jtitle=J. Struct. Biol.&amp;rft.volume=192&amp;rft.spage=216&amp;rft.epage=221&amp;rft.date=2015&amp;rft.atitle=CTFFIND4: Fast and accurate defocus estimation from electron micrographs&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.jsb.2015.08.008&amp;id=pmid:26278980&amp;genre=article&amp;aulast=Rohou&amp;aufirst=A.&amp;title=J. Struct. Biol.&amp;volume=192&amp;spage=216&amp;epage=221&amp;date=2015&amp;atitle=CTFFIND4: Fast and accurate defocus estimation from electron micrographs&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=26278980&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.jsb.2015.08.008">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref35"><span class="vcard author"><span class="fn">Li, X.</span></span> <i>et al</i>. <span class="title">Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM</span>. <span class="source-title">Nat. Methods</span> <span class="volume">10</span>, <span class="start-page">584</span>–<span class="end-page">590</span> (<span class="year">2013</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3sXntVakur0%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=4d7c3c996bf615eef1d5738c0cef997c">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nmeth.2472&amp;rft_id=info:pmid/23644547&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Li&amp;rft.aufirst=X.&amp;rft.jtitle=Nat. Methods&amp;rft.volume=10&amp;rft.spage=584&amp;rft.epage=590&amp;rft.date=2013&amp;rft.atitle=Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nmeth.2472&amp;id=pmid:23644547&amp;genre=article&amp;aulast=Li&amp;aufirst=X.&amp;title=Nat. Methods&amp;volume=10&amp;spage=584&amp;epage=590&amp;date=2013&amp;atitle=Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000319668700031&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=23644547&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="/doifinder/10.1038/nmeth.2472">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref36"><span class="vcard author"><span class="fn">Zhang, K.</span></span> <span class="title">Gctf: Real-time CTF determination and correction</span>. <span class="source-title">J. Struct. Biol.</span> <span class="volume">193</span>, <span class="start-page">1</span>–<span class="end-page">12</span> (<span class="year">2016</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXhvVOqsbzF&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=1093de8709d5d95088d18c8036fdc76f">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.jsb.2015.11.003&amp;rft_id=info:pmid/26592709&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Zhang&amp;rft.aufirst=K.&amp;rft.jtitle=J. Struct. Biol.&amp;rft.volume=193&amp;rft.spage=1&amp;rft.epage=12&amp;rft.date=2016&amp;rft.atitle=Gctf: Real-time CTF determination and correction&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.jsb.2015.11.003&amp;id=pmid:26592709&amp;genre=article&amp;aulast=Zhang&amp;aufirst=K.&amp;title=J. Struct. Biol.&amp;volume=193&amp;spage=1&amp;epage=12&amp;date=2016&amp;atitle=Gctf: Real-time CTF determination and correction&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=26592709&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.jsb.2015.11.003">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref37"><span class="vcard author"><span class="fn">Rosenthal, P. B.</span></span> &amp; <span class="vcard author"><span class="fn">Henderson, R.</span></span> <span class="title">Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy</span>. <span class="source-title">J. Mol. Biol.</span> <span class="volume">333</span>, <span class="start-page">721</span>–<span class="end-page">745</span> (<span class="year">2003</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXot1Gkt7k%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=d127df186a3888a14fa3f03e5c4ab63f">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.jmb.2003.07.013&amp;rft_id=info:pmid/14568533&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Rosenthal&amp;rft.aufirst=P. B.&amp;rft.jtitle=J. Mol. Biol.&amp;rft.volume=333&amp;rft.spage=721&amp;rft.epage=745&amp;rft.date=2003&amp;rft.atitle=Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.jmb.2003.07.013&amp;id=pmid:14568533&amp;genre=article&amp;aulast=Rosenthal&amp;aufirst=P. B.&amp;title=J. Mol. Biol.&amp;volume=333&amp;spage=721&amp;epage=745&amp;date=2003&amp;atitle=Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000186185300005&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=14568533&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1016/j.jmb.2003.07.013">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref38"><span class="vcard author"><span class="fn">Scheres, S. H.</span></span> <span class="title">Beam-induced motion correction for sub-megadalton cryo-EM particles</span>. <span class="source-title">eLife</span> <span class="volume">3</span>, <span class="start-page">e03665</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.7554/eLife.03665&amp;rft_id=info:pmid/25122622&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Scheres&amp;rft.aufirst=S. H.&amp;rft.jtitle=eLife&amp;rft.volume=3&amp;rft.spage=e03665&amp;rft.epage=&amp;rft.date=2014&amp;rft.atitle=Beam-induced motion correction for sub-megadalton cryo-EM particles&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.7554/eLife.03665&amp;id=pmid:25122622&amp;genre=article&amp;aulast=Scheres&amp;aufirst=S. H.&amp;title=eLife&amp;volume=3&amp;spage=e03665&amp;epage=&amp;date=2014&amp;atitle=Beam-induced motion correction for sub-megadalton cryo-EM particles&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25122622&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.7554/eLife.03665">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref39"><span class="vcard author"><span class="fn">Scheres, S. H.</span></span> &amp; <span class="vcard author"><span class="fn">Chen, S.</span></span> <span class="title">Prevention of overfitting in cryo-EM structure determination</span>. <span class="source-title">Nat. Methods</span> <span class="volume">9</span>, <span class="start-page">853</span>–<span class="end-page">854</span> (<span class="year">2012</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC38XhtFWmsLfO&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=d18cb3bbd51b4cf8ae72d136f2b25be0">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nmeth.2115&amp;rft_id=info:pmid/22842542&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Scheres&amp;rft.aufirst=S. H.&amp;rft.jtitle=Nat. Methods&amp;rft.volume=9&amp;rft.spage=853&amp;rft.epage=854&amp;rft.date=2012&amp;rft.atitle=Prevention of overfitting in cryo-EM structure determination&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nmeth.2115&amp;id=pmid:22842542&amp;genre=article&amp;aulast=Scheres&amp;aufirst=S. H.&amp;title=Nat. Methods&amp;volume=9&amp;spage=853&amp;epage=854&amp;date=2012&amp;atitle=Prevention of overfitting in cryo-EM structure determination&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000308497800004&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22842542&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1038/nmeth.2115">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref40"><span class="vcard author"><span class="fn">Bai, X. C.</span></span>, <span class="vcard author"><span class="fn">Rajendra, E.</span></span>, <span class="vcard author"><span class="fn">Yang, G.</span></span>, <span class="vcard author"><span class="fn">Shi, Y.</span></span> &amp; <span class="vcard author"><span class="fn">Scheres, S. H.</span></span> <span class="title">Sampling the conformational space of the catalytic subunit of human γ-secretase</span>. <span class="source-title">eLife</span> <span class="volume">4</span>, <span class="start-page">e11182</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.7554/eLife.11182&amp;rft_id=info:pmid/26623517&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Bai&amp;rft.aufirst=X. C.&amp;rft.jtitle=eLife&amp;rft.volume=4&amp;rft.spage=e11182&amp;rft.epage=&amp;rft.date=2015&amp;rft.atitle=Sampling the conformational space of the catalytic subunit of human &gamma;-secretase&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.7554/eLife.11182&amp;id=pmid:26623517&amp;genre=article&amp;aulast=Bai&amp;aufirst=X. C.&amp;title=eLife&amp;volume=4&amp;spage=e11182&amp;epage=&amp;date=2015&amp;atitle=Sampling the conformational space of the catalytic subunit of human &gamma;-secretase&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=26623517&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.7554/eLife.11182">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref41"><span class="vcard author"><span class="fn">Pettersen, E. F.</span></span> <i>et al</i>. <span class="title">UCSF Chimera–a visualization system for exploratory research and analysis</span>. <span class="source-title">J. Comput. Chem.</span> <span class="volume">25</span>, <span class="start-page">1605</span>–<span class="end-page">1612</span> (<span class="year">2004</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXmvVOhsbs%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=9547f53ee64ee465077db4be0b890eab">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1002/jcc.20084&amp;rft_id=info:pmid/15264254&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Pettersen&amp;rft.aufirst=E. F.&amp;rft.jtitle=J. Comput. Chem.&amp;rft.volume=25&amp;rft.spage=1605&amp;rft.epage=1612&amp;rft.date=2004&amp;rft.atitle=UCSF Chimera&ndash;a visualization system for exploratory research and analysis&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1002/jcc.20084&amp;id=pmid:15264254&amp;genre=article&amp;aulast=Pettersen&amp;aufirst=E. F.&amp;title=J. Comput. Chem.&amp;volume=25&amp;spage=1605&amp;epage=1612&amp;date=2004&amp;atitle=UCSF Chimera&ndash;a visualization system for exploratory research and analysis&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000223379100005&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=15264254&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1002/jcc.20084">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref42"><span class="vcard author"><span class="fn">Allegretti, M.</span></span>, <span class="vcard author"><span class="fn">Mills, D. J.</span></span>, <span class="vcard author"><span class="fn">McMullan, G.</span></span>, <span class="vcard author"><span class="fn">Kühlbrandt, W.</span></span> &amp; <span class="vcard author"><span class="fn">Vonck, J.</span></span> <span class="title">Atomic model of the F<sub>420</sub>-reducing [NiFe] hydrogenase by electron cryo-microscopy using a direct electron detector</span>.<span class="source-title"> eLife</span> <span class="volume">3</span>, <span class="start-page">e01963</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2cXovVyksb0%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=bcaabbb8c2b0adab2a311dc839932863">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.7554/eLife.01963&amp;rft_id=info:pmid/24569482&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Allegretti&amp;rft.aufirst=M.&amp;rft.jtitle=eLife&amp;rft.volume=3&amp;rft.spage=e01963&amp;rft.epage=&amp;rft.date=2014&amp;rft.atitle=Atomic model of the F&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.7554/eLife.01963&amp;id=pmid:24569482&amp;genre=article&amp;aulast=Allegretti&amp;aufirst=M.&amp;title=eLife&amp;volume=3&amp;spage=e01963&amp;epage=&amp;date=2014&amp;atitle=Atomic model of the F&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=24569482&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.7554/eLife.01963">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref43"><span class="vcard author"><span class="fn">Meents, A.</span></span>, <span class="vcard author"><span class="fn">Gutmann, S.</span></span>, <span class="vcard author"><span class="fn">Wagner, A.</span></span> &amp; <span class="vcard author"><span class="fn">Schulze-Briese, C.</span></span> <span class="title">Origin and temperature dependence of radiation damage in biological samples at cryogenic temperatures</span>. <span class="source-title">Proc. Natl Acad. Sci. USA</span> <span class="volume">107</span>, <span class="start-page">1094</span>–<span class="end-page">1099</span> (<span class="year">2010</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1073/pnas.0905481107&amp;rft_id=info:pmid/20080548&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Meents&amp;rft.aufirst=A.&amp;rft.jtitle=Proc. Natl Acad. Sci. USA&amp;rft.volume=107&amp;rft.spage=1094&amp;rft.epage=1099&amp;rft.date=2010&amp;rft.atitle=Origin and temperature dependence of radiation damage in biological samples at cryogenic temperatures&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1073/pnas.0905481107&amp;id=pmid:20080548&amp;genre=article&amp;aulast=Meents&amp;aufirst=A.&amp;title=Proc. Natl Acad. Sci. USA&amp;volume=107&amp;spage=1094&amp;epage=1099&amp;date=2010&amp;atitle=Origin and temperature dependence of radiation damage in biological samples at cryogenic temperatures&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=20080548&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1073/pnas.0905481107">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref44"><span class="vcard author"><span class="fn">Krivov, G. G.</span></span>, <span class="vcard author"><span class="fn">Shapovalov, M. V.</span></span> &amp; <span class="vcard author"><span class="fn">Dunbrack, R. L. Jr.</span></span> <span class="title">Improved prediction of protein side-chain conformations with SCWRL4</span>. <span class="source-title">Proteins</span> <span class="volume">77</span>, <span class="start-page">778</span>–<span class="end-page">795</span> (<span class="year">2009</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1MXhtlSru7nN&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=6436972f8e3210d16079729a08479e14">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1002/prot.22488&amp;rft_id=info:pmid/19603484&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Krivov&amp;rft.aufirst=G. G.&amp;rft.jtitle=Proteins&amp;rft.volume=77&amp;rft.spage=778&amp;rft.epage=795&amp;rft.date=2009&amp;rft.atitle=Improved prediction of protein side-chain conformations with SCWRL4&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1002/prot.22488&amp;id=pmid:19603484&amp;genre=article&amp;aulast=Krivov&amp;aufirst=G. G.&amp;title=Proteins&amp;volume=77&amp;spage=778&amp;epage=795&amp;date=2009&amp;atitle=Improved prediction of protein side-chain conformations with SCWRL4&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000271602000003&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=19603484&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1002/prot.22488">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref45"><span class="vcard author"><span class="fn">Kelley, L. A.</span></span>, <span class="vcard author"><span class="fn">Mezulis, S.</span></span>, <span class="vcard author"><span class="fn">Yates, C. M.</span></span>, <span class="vcard author"><span class="fn">Wass, M. N.</span></span> &amp; <span class="vcard author"><span class="fn">Sternberg, M. J.</span></span> <span class="title">The Phyre2 web portal for protein modeling, prediction and analysis</span>. <span class="source-title">Nat. Protocols</span> <span class="volume">10</span>, <span class="start-page">845</span>–<span class="end-page">858</span> (<span class="year">2015</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2MXnvFSqurs%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=9dcf8811b6e7047976776df7e58812d3">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nprot.2015.053&amp;rft_id=info:pmid/25950237&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Kelley&amp;rft.aufirst=L. A.&amp;rft.jtitle=Nat. Protocols&amp;rft.volume=10&amp;rft.spage=845&amp;rft.epage=858&amp;rft.date=2015&amp;rft.atitle=The Phyre2 web portal for protein modeling, prediction and analysis&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nprot.2015.053&amp;id=pmid:25950237&amp;genre=article&amp;aulast=Kelley&amp;aufirst=L. A.&amp;title=Nat. Protocols&amp;volume=10&amp;spage=845&amp;epage=858&amp;date=2015&amp;atitle=The Phyre2 web portal for protein modeling, prediction and analysis&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=25950237&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1038/nprot.2015.053">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref46"><span class="vcard author"><span class="fn">Biasini, M.</span></span> <i>et al</i>. <span class="title">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</span>. <span class="source-title">Nucleic Acids Res</span>. <span class="volume">42</span>, <span class="start-page">W252</span>–<span class="end-page">W258</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC2cXhtFCqs73I&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=28dc8368b19cb96022fd8a5258628f8d">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gku340&amp;rft_id=info:pmid/24782522&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Biasini&amp;rft.aufirst=M.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=42&amp;rft.spage=W252&amp;rft.epage=W258&amp;rft.date=2014&amp;rft.atitle=SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gku340&amp;id=pmid:24782522&amp;genre=article&amp;aulast=Biasini&amp;aufirst=M.&amp;title=Nucleic Acids Res&amp;volume=42&amp;spage=W252&amp;epage=W258&amp;date=2014&amp;atitle=SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000339715000042&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=24782522&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1093/nar/gku340">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref47"><span class="vcard author"><span class="fn">Rost, B.</span></span>, <span class="vcard author"><span class="fn">Yachdav, G.</span></span> &amp; <span class="vcard author"><span class="fn">Liu, J.</span></span> <span class="title">The PredictProtein server</span>. <span class="source-title">Nucleic Acids Res</span>. <span class="volume">32</span>, <span class="start-page">W321</span>–<span class="end-page">W326</span> (<span class="year">2004</span>)/
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXlvFKntLc%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=e728b2efeacf333d07b41d0223c0e98b">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkh377&amp;rft_id=info:pmid/15215403&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Rost&amp;rft.aufirst=B.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=32&amp;rft.spage=W321&amp;rft.epage=W326&amp;rft.date=2004&amp;rft.atitle=The PredictProtein server&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkh377&amp;id=pmid:15215403&amp;genre=article&amp;aulast=Rost&amp;aufirst=B.&amp;title=Nucleic Acids Res&amp;volume=32&amp;spage=W321&amp;epage=W326&amp;date=2004&amp;atitle=The PredictProtein server&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000222273100065&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=15215403&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1093/nar/gkh377">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref48"><span class="vcard author"><span class="fn">McGuffin, L. J.</span></span>, <span class="vcard author"><span class="fn">Bryson, K.</span></span> &amp; <span class="vcard author"><span class="fn">Jones, D. T.</span></span> <span class="title">The PSIPRED protein structure prediction server</span>. <span class="source-title">Bioinformatics</span> <span class="volume">16</span>, <span class="start-page">404</span>–<span class="end-page">405</span> (<span class="year">2000</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXltFOksrw%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=2e6efa573a0d716202b4d11a95ca8b11">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/bioinformatics/16.4.404&amp;rft_id=info:pmid/10869041&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=McGuffin&amp;rft.aufirst=L. J.&amp;rft.jtitle=Bioinformatics&amp;rft.volume=16&amp;rft.spage=404&amp;rft.epage=405&amp;rft.date=2000&amp;rft.atitle=The PSIPRED protein structure prediction server&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/bioinformatics/16.4.404&amp;id=pmid:10869041&amp;genre=article&amp;aulast=McGuffin&amp;aufirst=L. J.&amp;title=Bioinformatics&amp;volume=16&amp;spage=404&amp;epage=405&amp;date=2000&amp;atitle=The PSIPRED protein structure prediction server&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000087933900014&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=10869041&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1093/bioinformatics/16.4.404">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref49"><span class="vcard author"><span class="fn">Krogh, A.</span></span>, <span class="vcard author"><span class="fn">Larsson, B.</span></span>, <span class="vcard author"><span class="fn">von Heijne, G.</span></span> &amp; <span class="vcard author"><span class="fn">Sonnhammer, E. L.</span></span> <span class="title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes</span>. <span class="source-title">J. Mol. Biol.</span> <span class="volume">305</span>, <span class="start-page">567</span>–<span class="end-page">580</span> (<span class="year">2001</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXisFCguw%3D%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=c004ea0f635d33f07a92fbb1adfefbdb">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1006/jmbi.2000.4315&amp;rft_id=info:pmid/11152613&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Krogh&amp;rft.aufirst=A.&amp;rft.jtitle=J. Mol. Biol.&amp;rft.volume=305&amp;rft.spage=567&amp;rft.epage=580&amp;rft.date=2001&amp;rft.atitle=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1006/jmbi.2000.4315&amp;id=pmid:11152613&amp;genre=article&amp;aulast=Krogh&amp;aufirst=A.&amp;title=J. Mol. Biol.&amp;volume=305&amp;spage=567&amp;epage=580&amp;date=2001&amp;atitle=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000167760800017&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=11152613&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1006/jmbi.2000.4315">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref50"><span class="vcard author"><span class="fn">Emsley, P.</span></span> &amp; <span class="vcard author"><span class="fn">Cowtan, K.</span></span> <span class="title">Coot: model-building tools for molecular graphics</span>. <span class="source-title">Acta Crystallogr. D</span> <span class="volume">60</span>, <span class="start-page">2126</span>–<span class="end-page">2132</span> (<span class="year">2004</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXhtVars73P&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=aa2c3dc083dd2c6c3d0e3ed027b7497e">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1107/S0907444904019158&amp;rft_id=info:pmid/15572765&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Emsley&amp;rft.aufirst=P.&amp;rft.jtitle=Acta Crystallogr. D&amp;rft.volume=60&amp;rft.spage=2126&amp;rft.epage=2132&amp;rft.date=2004&amp;rft.atitle=Coot: model-building tools for molecular graphics&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1107/S0907444904019158&amp;id=pmid:15572765&amp;genre=article&amp;aulast=Emsley&amp;aufirst=P.&amp;title=Acta Crystallogr. D&amp;volume=60&amp;spage=2126&amp;epage=2132&amp;date=2004&amp;atitle=Coot: model-building tools for molecular graphics&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000225360500002&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=15572765&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1107/S0907444904019158">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref51"><span class="vcard author"><span class="fn">Shinzawa-Itoh, K.</span></span> <i>et al</i>. <span class="title">Bovine heart NADH-ubiquinone oxidoreductase contains one molecule of ubiquinone with ten isoprene units as one of the cofactors</span>. <span class="source-title">Biochemistry</span> <span class="volume">49</span>, <span class="start-page">487</span>–<span class="end-page">492</span> (<span class="year">2010</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1021/bi9016318&amp;rft_id=info:pmid/19961238&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Shinzawa-Itoh&amp;rft.aufirst=K.&amp;rft.jtitle=Biochemistry&amp;rft.volume=49&amp;rft.spage=487&amp;rft.epage=492&amp;rft.date=2010&amp;rft.atitle=Bovine heart NADH-ubiquinone oxidoreductase contains one molecule of ubiquinone with ten isoprene units as one of the cofactors&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1021/bi9016318&amp;id=pmid:19961238&amp;genre=article&amp;aulast=Shinzawa-Itoh&amp;aufirst=K.&amp;title=Biochemistry&amp;volume=49&amp;spage=487&amp;epage=492&amp;date=2010&amp;atitle=Bovine heart NADH-ubiquinone oxidoreductase contains one molecule of ubiquinone with ten isoprene units as one of the cofactors&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=19961238&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1021/bi9016318">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref52"><span class="vcard author"><span class="fn">Adams, P. D.</span></span> <i>et al</i>. <span class="title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <span class="source-title">Acta Crystallogr. D</span> <span class="volume">66</span>, <span class="start-page">213</span>–<span class="end-page">221</span> (<span class="year">2010</span>)
					<ul class="cleared">
																																				<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3cXhs1Sisbc%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=87294aac3e6e41335f48debeff99b800">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1107/S0907444909052925&amp;rft_id=info:pmid/20124702&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Adams&amp;rft.aufirst=P. D.&amp;rft.jtitle=Acta Crystallogr. D&amp;rft.volume=66&amp;rft.spage=213&amp;rft.epage=221&amp;rft.date=2010&amp;rft.atitle=PHENIX: a comprehensive Python-based system for macromolecular structure solution&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1107/S0907444909052925&amp;id=pmid:20124702&amp;genre=article&amp;aulast=Adams&amp;aufirst=P. D.&amp;title=Acta Crystallogr. D&amp;volume=66&amp;spage=213&amp;epage=221&amp;date=2010&amp;atitle=PHENIX: a comprehensive Python-based system for macromolecular structure solution&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000273820800013&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=20124702&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1107/S0907444909052925">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref53"><span class="vcard author"><span class="fn">Leitner, A.</span></span>, <span class="vcard author"><span class="fn">Walzthoeni, T.</span></span> &amp; <span class="vcard author"><span class="fn">Aebersold, R.</span></span> <span class="title">Lysine-specific chemical cross-linking of protein complexes and identification of cross-linking sites using LC-MS/MS and the xQuest/xProphet software pipeline</span>. <span class="source-title">Nat. Protocols</span> <span class="volume">9</span>, <span class="start-page">120</span>–<span class="end-page">137</span> (<span class="year">2014</span>)
					<ul class="cleared">
																																		 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nprot.2013.168&amp;rft_id=info:pmid/24356771&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Leitner&amp;rft.aufirst=A.&amp;rft.jtitle=Nat. Protocols&amp;rft.volume=9&amp;rft.spage=120&amp;rft.epage=137&amp;rft.date=2014&amp;rft.atitle=Lysine-specific chemical cross-linking of protein complexes and identification of cross-linking sites using LC-MS/MS and the xQuest/xProphet software pipeline&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nprot.2013.168&amp;id=pmid:24356771&amp;genre=article&amp;aulast=Leitner&amp;aufirst=A.&amp;title=Nat. Protocols&amp;volume=9&amp;spage=120&amp;epage=137&amp;date=2014&amp;atitle=Lysine-specific chemical cross-linking of protein complexes and identification of cross-linking sites using LC-MS/MS and the xQuest/xProphet software pipeline&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=24356771&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1038/nprot.2013.168">Article</a></li>
										</ul>
			
				</li>
		
								
														
		<li id="ref54"><span class="vcard author"><span class="fn">Kahraman, A.</span></span>, <span class="vcard author"><span class="fn">Malmström, L.</span></span> &amp; <span class="vcard author"><span class="fn">Aebersold, R.</span></span> <span class="title">Xwalk: computing and visualizing distances in cross-linking experiments</span>. <span class="source-title">Bioinformatics</span> <span class="volume">27</span>, <span class="start-page">2163</span>–<span class="end-page">2164</span> (<span class="year">2011</span>)
					<ul class="cleared">
																																	<li>
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXptFCqsbs%3D&amp;pissn=0028-0836&amp;pyear=2016&amp;md5=052a4feb087445dec232d8ee2b442d31">CAS</a></li>
																				 							<li><a href="http://rp8jq9jy4s.search.serialssolutions.com/?&amp;url_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/bioinformatics/btr348&amp;rft_id=info:pmid/21666267&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Kahraman&amp;rft.aufirst=A.&amp;rft.jtitle=Bioinformatics&amp;rft.volume=27&amp;rft.spage=2163&amp;rft.epage=2164&amp;rft.date=2011&amp;rft.atitle=Xwalk: computing and visualizing distances in cross-linking experiments&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/bioinformatics/btr348&amp;id=pmid:21666267&amp;genre=article&amp;aulast=Kahraman&amp;aufirst=A.&amp;title=Bioinformatics&amp;volume=27&amp;spage=2163&amp;epage=2164&amp;date=2011&amp;atitle=Xwalk: computing and visualizing distances in cross-linking experiments&amp;sid=nature:Nature">OpenURL</a></li>
																					<li>
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=21666267&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>
											<a href="http://dx.doi.org/10.1093/bioinformatics/btr348">Article</a></li>
										</ul>
			
				</li>
		
								
	</ol>
	<p class="download-refs"><a href="/nature/journal/v538/n7625/ris/nature19794refs.ris">Download references</a></p>
	

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="acknowledgments">

									<h1 class="section-heading toggle">Acknowledgements</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7">Acknowledgements<span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
					<p>We thank the ETH Zurich ScopeM Center for access to Titan Krios EM. Data processing was performed at the IST high-performance computer cluster. K.F. is partially funded by the Medical Research Council UK PhD fellowship. J.A.L. holds a long-term fellowship from FEBS. This project has received funding from the European Union’s 2020 research and innovation programme under grant agreement No 701309.</p>

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="author-information">

									<h1 class="section-heading toggle">Author information</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7">Author information<span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
												
										
					


																															

													
	<div id="author-affiliations">
		
		<h2 id="affil-auth">Affiliations</h2>
		<ol class="affiliations" data-test="af-section">
																<li id="a1">
						<h3>Institute of Science and Technology Austria, Klosterneuburg 3400, Austria</h3>
						 
							
								
																																																		
								
																																																		
								
																																																		
								
																																																		
							
															<ul class="affiliation-authors">
																			<li>Karol Fiedorczuk,</li>
																			<li>James A. Letts,</li>
																			<li>Karol Kaszuba &amp;</li>
																			<li>Leonid A. Sazanov</li>
																	</ul>
																		</li>
																				<li id="a2">
						<h3>MRC Mitochondrial Biology Unit, Cambridge CB2 0XY, UK</h3>
						 
							
								
																																																		
							
															<ul class="affiliation-authors">
																			<li>Karol Fiedorczuk</li>
																	</ul>
																		</li>
																				<li id="a3">
						<h3>MRC Laboratory of Molecular Biology, Cambridge CB2 OQH, UK</h3>
						 
							
								
																																																		
								
																																																		
							
															<ul class="affiliation-authors">
																			<li>Gianluca Degliesposti &amp;</li>
																			<li>Mark Skehel</li>
																	</ul>
																		</li>
																				
						
							
																																		</ol>
	</div>



	<div id="author-contributions">
		<h2 id="contrib-auth">Contributions</h2>
		<p>K.F. purified complex I for grid preparation, prepared cryo-EM grids, acquired and processed EM data, and co-built the models; J.A.L. purified complex I for cross-linking experiments, analysed cross-linking data and co-built the models; G.D. performed cross-linking/mass-spectrometry experiments, K.K. performed model re-building in Rosetta and sequence alignments; G.D. and M.S. analysed cross-linking data; L.A.S. designed and supervised the project, processed and analysed data and wrote the manuscript, with contributions from all authors.</p>
	</div>

 
	<div id="competing-financial-interests">
		<h2 id="cfi">Competing financial interests</h2>

								
												<p class="conflict">The authors declare no competing financial interests.</p>
													</div>

	<div id="corresponding-authors">
		<h2 id="corres-auth">Corresponding author</h2>
		<p class="correspondence">Correspondence to: </p>
		<ul class="correspondence">
					<li id="c1"><a href="/nature/foxtrot/svc/authoremailform?doi=10.1038/nature19794&amp;file=/nature/journal/v538/n7625/full/nature19794.html&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I&amp;author=Leonid+A.+Sazanov" class="contact">Leonid A. Sazanov</a></li>
							</ul>
	</div>

	<p>The electron microscopy maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes <a href="http://www.ebi.ac.uk/pdbe/entry/EMD-4084">EMD-4084</a>, <a href="http://www.ebi.ac.uk/pdbe/entry/EMD-4090">EMD-4090</a>, <a href="http://www.ebi.ac.uk/pdbe/entry/EMD-4091">EMD-4091</a> and <a href="http://www.ebi.ac.uk/pdbe/entry/EMD-4093">EMD-4093</a> (combined map). The model has been deposited in the Protein Data Bank (PDB) under accession code <a href="http://www.pdb.org/pdb/search/structidSearch.do?structureId=5LNK">5LNK</a>.</p><p><b>Reviewer Information</b></p><p><i>Nature</i> thanks P. Ädelroth, M. Ryan and the other anonymous reviewer(s) for their contribution to the peer review of this work.</p>


	<div id="author-extra-details" >
		<h2>Author details</h2>
		<ul><li id="auth-1" class="author-details cleared no-bio"><h3>Karol Fiedorczuk</h3><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Karol+Fiedorczuk&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karol+Fiedorczuk%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Karol+Fiedorczuk&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-2" class="author-details cleared no-bio"><h3>James A. Letts</h3><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=James+A.+Letts&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=James+A.+Letts%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=James+A.+Letts&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-3" class="author-details cleared no-bio"><h3>Gianluca Degliesposti</h3><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Gianluca+Degliesposti&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gianluca+Degliesposti%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Gianluca+Degliesposti&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-4" class="author-details cleared no-bio"><h3>Karol Kaszuba</h3><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Karol+Kaszuba&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karol+Kaszuba%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Karol+Kaszuba&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-5" class="author-details cleared no-bio"><h3>Mark Skehel</h3><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Mark+Skehel&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mark+Skehel%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Mark+Skehel&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-6" class="author-details cleared no-bio"><h3>Leonid A. Sazanov</h3><p class="email"><a href="/nature/foxtrot/svc/authoremailform?doi=10.1038/nature19794&amp;file=/nature/journal/v538/n7625/full/nature19794.html&amp;title=Atomic+structure+of+the+entire+mammalian+mitochondrial+complex+I&amp;author=Leonid+A.+Sazanov" class="contact">Contact Leonid A. Sazanov</a></p><div class="author-search"><h4>Search for this author in:</h4><ul class="inline-list"><li><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Leonid+A.+Sazanov&amp;sp-p-2=all">NPG journals</a><span class="bullet">&bull;</span> </li> <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Leonid+A.+Sazanov%5Bauthor%5D">PubMed</a><span class="bullet">&bull;</span> </li> <li class="last"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Leonid+A.+Sazanov&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li></ul>
	</div>
	

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="extended-data">

									<h1 class="section-heading toggle">Extended data figures and tables</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7">Extended data figures and tables<span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7"><a href="#supplementary-information">Supplementary information</a></li>
						</ul></nav>

					
					
												
										
					

<div class="box supp-info">
	
	
				
									<h2 class="icon-figures">Extended Data Figures</h2>
				<ol>
									<li id="sf1-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="supp-section-link supp-fig" data-related-id="sf1">
																Extended Data Figure 1: Image processing procedures.
								(489 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, Representative micrograph of 2.6 k micrographs collected that all varied in defocus, ice thickness and particle count, with good quality particles circled. Scale bar, 100 nm. <b>b</b>, Representative 2D class averages obtained from reference-free classification. <b>c</b>, Classification and refinement procedures used in this study.</p></dd>					</dl></li>
									<li id="sf2-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html" class="supp-section-link supp-fig" data-related-id="sf2">
																Extended Data Figure 2: Image and model refinement procedures.
								(210 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, Radiation-damage weighting. Relative B-factors (<i>B</i><sub>f</sub>) and intercepts (<i>C</i><sub>f</sub>) from the Relion particle polishing procedure. <b>b</b>, Left, gold-standard (two halves of data refined independently) Fourier shell correlation (FSC) curves for the maps of the entire complex complete map (resolution at FSC = 0.143 is 3.9 Å), membrane-arm-focused refinement (4.1 Å resolution) and peripheral-arm-focused refinement (3.9 Å resolution). Right, FSC curve of the combined map versus final model shows good agreement of the model with the map (FSC = 0.5 at 4.0 Å resolution). FSC curve against the entire complex complete map, which was not used in refinement, is shown as a control. <b>c</b>, Statistics of refinement.</p></dd>					</dl></li>
									<li id="sf3-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html" class="supp-section-link supp-fig" data-related-id="sf3">
																Extended Data Figure 3: Local resolution estimation and combination of maps for model building.
								(1,148 KB)
							</a>
						</dt>
						<dd><p><b>a</b>–<b>c</b>, Local resolution estimation by Resmap of the entire complex I (<b>a</b>), peripheral-arm-focused refinement map (<b>b</b>) and membrane-arm-focused refinement map (<b>c</b>). Maps are coloured according to the shown resolution scale in Å. <b>d</b>. The final map was produced by combining maps with the best local resolution features; that is, for peripheral-arm-focused refinement map (orange), for the distal part of membrane-arm-focused refinement map (green), for 42-kDa subunit map from the selected homogenous complex I class (64 k particles; blue) and the rest of the complex from the best map of the entire complex (magenta).</p></dd>					</dl></li>
									<li id="sf4-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html" class="supp-section-link supp-fig" data-related-id="sf4">
																Extended Data Figure 4: Examples of cryo-EM density.
								(556 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, <b>b</b>, Coils and α-helices from core (<b>a</b>) and supernumerary (<b>b</b>) subunits. <b>c, d,</b> Example β-sheets from core PSST subunit (<b>c</b>) and supernumerary 39-kDa subunit (<b>d</b>). Cryo-EM density is shown with the model represented as sticks and coloured by atom with carbon in grey, oxygen in red, nitrogen in blue and sulfur in yellow.</p></dd>					</dl></li>
									<li id="sf5-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html" class="supp-section-link supp-fig" data-related-id="sf5">
																Extended Data Figure 5: Identified cross-links.
								(630 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, Solvent-accessible surface (SAS) representation of cross-links. Surfaces for complex I subunits are shown transparent and coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>. Shortest SAS paths calculated using Xwalk<sup><a href="#ref54" title="Kahraman, A., Malmström, L. &amp; Aebersold, R. Xwalk: computing and visualizing distances in cross-linking experiments. Bioinformatics 27, 2163–2164 (2011)" id="ref-link-4">54</a></sup> are shown for cross-links as coloured worms with inter-subunit lysine reactive cross-links in blue, inter-subunit acid reactive cross-links in red, intra-subunit lysine reactive cross-links in light blue, intra-subunit acid reactive cross-links in light red. <b>b</b>, Inter-subunit cross-link schematic. Complex I subunits are shown in a similar orientation as in <b>a</b>. Left panel with core subunits cyan, previously assigned supernumerary subunits in magenta, newly assigned or newly built regions of supernumerary subunits in green, poly-alanine regions in orange and unmodelled regions in red. Observed cross-links are indicated by dashed black lines between either blue circles (lysine reactive cross-links) or red circles (acid reactive cross-links). No cross-links were observed to the core subunits of the membrane arm and hence they were omitted for clarity. The horizontal black lines indicate the approximate boundaries of the inner mitochondrial membrane. Subunits B14.7, B15 and ASHI are shown as being behind the membrane boundaries as they are found on the opposite (far) side of the membrane arm.</p></dd>					</dl></li>
									<li id="sf6-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF6.html" class="supp-section-link supp-fig" data-related-id="sf6">
																Extended Data Figure 6: Folds of supernumerary subunits.
								(557 KB)
							</a>
						</dt>
						<dd><p>Subunits are shown in cartoon representation, coloured blue to red from N to C terminus. Disulfide bridges are shown as sticks with sulfur in yellow.</p></dd>					</dl></li>
									<li id="sf7-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF7.html" class="supp-section-link supp-fig" data-related-id="sf7">
																Extended Data Figure 7: Examples of supernumerary subunits interactions.
								(1,221 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, Side view of complex I showing surfaces for subunits B14.5a and B16.6. <b>b</b>, IMS view of complex I showing surfaces for subunits SGDH and PDSW. The point at which the two subunits are intertwined is marked with a star. <b>c</b>, View of the hydrophilic arm looking from above the membrane arm. The surface of the 18-kDa subunit that spans the hydrophilic arm is shown. <b>d</b>, Matrix view of the tip of the membrane arm with the surface of supernumerary subunit B22 shown. <b>e</b>, Close up of the centre of the membrane arm on the IMS side. This region contains many interactions between supernumerary subunits and the side chains of residues involved are shown. The region is also a hot spot for cross-links, the side chains involved are shown and cross-links are indicated with dashed lines (acid cross-links: red; basic cross-links: blue). <b>f</b>, Close up of the C-terminal helix of supernumerary subunit PDSW at the centre of the membrane arm on the IMS side. This helix extends away from complex I and is encircled by the C termini of supernumerary subunits B14.5b, ESSS and B15. The side chains of residues involved in stabilizing interactions are shown. A possible disulfide bond between PDSW (Cys154) and ESSS (Cys112) and stabilizing salt bridges are indicated by dashed lines. Subunits are coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>.</p></dd>					</dl></li>
									<li id="sf8-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html" class="supp-section-link supp-fig" data-related-id="sf8">
																Extended Data Figure 8: Comparison of ‘open’ and ‘closed’ 3D class structures.
								(905 KB)
							</a>
						</dt>
						<dd><p><b>a</b>, <b>b</b>, Side (<b>a</b>) and top (<b>b</b>) view from the matrix for the alignment of the open class structure (in cyan) and closed class structure (in grey). To generate the closed class structure, the final structure of the open class was refined in real space in Phenix (5 macro cycles with morphing at each cycle) against 4.6 Å map of the closed class (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="draggable supp-fig" data-related-id="sf1">Extended Data Fig. 1</a>). All of the α-helices were well fit into density, but owing to low resolution of the closed class no further refinement was performed and the comparison of structures involves only the relative positions of secondary structure elements. The two structures were aligned via transmembrane core subunits and are displayed as cartoon models. In the closed class the peripheral arm undergoes a hinge-like motion around the Q site towards the tip of the membrane arm, with the direction of shift indicated by the arrow in <b>b</b>. As a result, subunit B13 moves ~3 Å closer to the 42-kDa subunit, allowing for direct contacts. The shift is larger at the periphery, reaching 7 Å at the tip of the peripheral arm. Additionally, subunit ND5 and its matrix bulge move about 3 Å towards peripheral arm.</p></dd>					</dl></li>
								</ol>
						
				
			
				
									<h2 class="icon-tables">Extended Data Tables</h2>
				<ol>
									<li id="st1-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST1.html" class="supp-section-link supp-tab" data-related-id="st1">
																Extended Data Table 1: Summary of the model
								(539 KB)
							</a>
						</dt>
											</dl></li>
									<li id="st2-inarticle"><dl>
						<dt>
							<a href="/nature/journal/v538/n7625/fig_tab/nature19794_ST2.html" class="supp-section-link supp-tab" data-related-id="st2">
																Extended Data Table 2: Summary of interactions between subunits of ovine complex I
								(634 KB)
							</a>
						</dt>
											</dl></li>
								</ol>
						
</div>	

									</div>

																										</div></section>

		

			
																								
									
									<section><div class="section   expanded" id="supplementary-information">

									<h1 class="section-heading toggle">Supplementary information</h1>
								<div class="content ">
					
																	<nav><ul class="section-nav inline-list cleared">

							
																																																																																																																																																																																																																																																														
																																																																																																																																																																																																																																																																																																																																																																																																										<li class="#2 and 1 of 7"><a href="#main">Main</a><span class="bullet">&bull;</span> </li> <li class="#3 and 2 of 7"><a href="#methods">Methods</a><span class="bullet">&bull;</span> </li> <li class="#4 and 3 of 7"><a href="#references">References</a><span class="bullet">&bull;</span> </li> <li class="#5 and 4 of 7"><a href="#acknowledgments">Acknowledgements</a><span class="bullet">&bull;</span> </li> <li class="#6 and 5 of 7"><a href="#author-information">Author information</a><span class="bullet">&bull;</span> </li> <li class="#7 and 6 of 7"><a href="#extended-data">Extended data figures and tables</a><span class="bullet">&bull;</span> </li> <li class="#8 and 7 of 7">Supplementary information</li>
						</ul></nav>

					
					
														
										
							
										

<div class="box supp-info">
	
																
					
		<h2 class="pdf">PDF files</h2>
		<ol>
					<li><dl><dt><a href="/nature/journal/v538/n7625/extref/nature19794-s1.pdf">Supplementary Information (677 KB)</a></dt><dd><p>This file contains Supplementary Discussion and Supplementary Tables 1-4.</p></dd></dl></li>
				</ol>
	</div>	

									</div>

																									<footer><div class="article-footer">
							<p><a href="/nature/journal/v538/n7625/aappdf/nature19794.pdf">An earlier version of this paper was published online on 05 September 2016</a>.</p>
						</div></footer>

												</div></section>

		
		
		
									
										
					<section>
	<h1 class="hidden">Additional data</h1>
	<div class="additional-data hidden">
		
		

		
		
						
				
											
												
				<ul id="supplementary-figures-list">
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 1: Image processing procedures.
							</div>

							<div id="sf1" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf1.jpg" style="width:328px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, Representative micrograph of 2.6 k micrographs collected that all varied in defocus, ice thickness and particle count, with good quality particles circled. Scale bar, 100 nm. <b>b</b>, Representative 2D class averages obtained from reference-free classification. <b>c</b>, Classification and refinement procedures used in this study.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 2: Image and model refinement procedures.
							</div>

							<div id="sf2" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf2.jpg" style="width:375px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, Radiation-damage weighting. Relative B-factors (<i>B</i><sub>f</sub>) and intercepts (<i>C</i><sub>f</sub>) from the Relion particle polishing procedure. <b>b</b>, Left, gold-standard (two halves of data refined independently) Fourier shell correlation (FSC) curves for the maps of the entire complex complete map (resolution at FSC = 0.143 is 3.9 Å), membrane-arm-focused refinement (4.1 Å resolution) and peripheral-arm-focused refinement (3.9 Å resolution). Right, FSC curve of the combined map versus final model shows good agreement of the model with the map (FSC = 0.5 at 4.0 Å resolution). FSC curve against the entire complex complete map, which was not used in refinement, is shown as a control. <b>c</b>, Statistics of refinement.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF2.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 3: Local resolution estimation and combination of maps for model building.
							</div>

							<div id="sf3" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf3.jpg" style="width:289px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>–<b>c</b>, Local resolution estimation by Resmap of the entire complex I (<b>a</b>), peripheral-arm-focused refinement map (<b>b</b>) and membrane-arm-focused refinement map (<b>c</b>). Maps are coloured according to the shown resolution scale in Å. <b>d</b>. The final map was produced by combining maps with the best local resolution features; that is, for peripheral-arm-focused refinement map (orange), for the distal part of membrane-arm-focused refinement map (green), for 42-kDa subunit map from the selected homogenous complex I class (64 k particles; blue) and the rest of the complex from the best map of the entire complex (magenta).</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF3.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 4: Examples of cryo-EM density.
							</div>

							<div id="sf4" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf4.jpg" style="width:343px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, <b>b</b>, Coils and α-helices from core (<b>a</b>) and supernumerary (<b>b</b>) subunits. <b>c, d,</b> Example β-sheets from core PSST subunit (<b>c</b>) and supernumerary 39-kDa subunit (<b>d</b>). Cryo-EM density is shown with the model represented as sticks and coloured by atom with carbon in grey, oxygen in red, nitrogen in blue and sulfur in yellow.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF4.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 5: Identified cross-links.
							</div>

							<div id="sf5" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf5.jpg" style="width:356px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, Solvent-accessible surface (SAS) representation of cross-links. Surfaces for complex I subunits are shown transparent and coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>. Shortest SAS paths calculated using Xwalk<sup><a href="#ref54" title="Kahraman, A., Malmström, L. &amp; Aebersold, R. Xwalk: computing and visualizing distances in cross-linking experiments. Bioinformatics 27, 2163–2164 (2011)" id="ref-link-4">54</a></sup> are shown for cross-links as coloured worms with inter-subunit lysine reactive cross-links in blue, inter-subunit acid reactive cross-links in red, intra-subunit lysine reactive cross-links in light blue, intra-subunit acid reactive cross-links in light red. <b>b</b>, Inter-subunit cross-link schematic. Complex I subunits are shown in a similar orientation as in <b>a</b>. Left panel with core subunits cyan, previously assigned supernumerary subunits in magenta, newly assigned or newly built regions of supernumerary subunits in green, poly-alanine regions in orange and unmodelled regions in red. Observed cross-links are indicated by dashed black lines between either blue circles (lysine reactive cross-links) or red circles (acid reactive cross-links). No cross-links were observed to the core subunits of the membrane arm and hence they were omitted for clarity. The horizontal black lines indicate the approximate boundaries of the inner mitochondrial membrane. Subunits B14.7, B15 and ASHI are shown as being behind the membrane boundaries as they are found on the opposite (far) side of the membrane arm.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF5.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 6: Folds of supernumerary subunits.
							</div>

							<div id="sf6" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf6.jpg" style="width:341px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p>Subunits are shown in cartoon representation, coloured blue to red from N to C terminus. Disulfide bridges are shown as sticks with sulfur in yellow.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF6.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 7: Examples of supernumerary subunits interactions.
							</div>

							<div id="sf7" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf7.jpg" style="width:306px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, Side view of complex I showing surfaces for subunits B14.5a and B16.6. <b>b</b>, IMS view of complex I showing surfaces for subunits SGDH and PDSW. The point at which the two subunits are intertwined is marked with a star. <b>c</b>, View of the hydrophilic arm looking from above the membrane arm. The surface of the 18-kDa subunit that spans the hydrophilic arm is shown. <b>d</b>, Matrix view of the tip of the membrane arm with the surface of supernumerary subunit B22 shown. <b>e</b>, Close up of the centre of the membrane arm on the IMS side. This region contains many interactions between supernumerary subunits and the side chains of residues involved are shown. The region is also a hot spot for cross-links, the side chains involved are shown and cross-links are indicated with dashed lines (acid cross-links: red; basic cross-links: blue). <b>f</b>, Close up of the C-terminal helix of supernumerary subunit PDSW at the centre of the membrane arm on the IMS side. This helix extends away from complex I and is encircled by the C termini of supernumerary subunits B14.5b, ESSS and B15. The side chains of residues involved in stabilizing interactions are shown. A possible disulfide bond between PDSW (Cys154) and ESSS (Cys112) and stabilizing salt bridges are indicated by dashed lines. Subunits are coloured as in <a href="/nature/journal/v538/n7625/full/nature19794.html#f1">Fig. 1</a>.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF7.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Figure 8: Comparison of ‘open’ and ‘closed’ 3D class structures.
							</div>

							<div id="sf8" class="supp-item supp-figures-item no-padding cleared">
								<div class="supp-figures-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-sf8.jpg" style="width:345px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																				<div class="caption"><p><b>a</b>, <b>b</b>, Side (<b>a</b>) and top (<b>b</b>) view from the matrix for the alignment of the open class structure (in cyan) and closed class structure (in grey). To generate the closed class structure, the final structure of the open class was refined in real space in Phenix (5 macro cycles with morphing at each cycle) against 4.6 Å map of the closed class (<a href="/nature/journal/v538/n7625/fig_tab/nature19794_SF1.html" class="draggable supp-fig" data-related-id="sf1">Extended Data Fig. 1</a>). All of the α-helices were well fit into density, but owing to low resolution of the closed class no further refinement was performed and the comparison of structures involves only the relative positions of secondary structure elements. The two structures were aligned via transmembrane core subunits and are displayed as cartoon models. In the closed class the peripheral arm undergoes a hinge-like motion around the Q site towards the tip of the membrane arm, with the direction of shift indicated by the arrow in <b>b</b>. As a result, subunit B13 moves ~3 Å closer to the 42-kDa subunit, allowing for direct contacts. The shift is larger at the periphery, reaching 7 Å at the tip of the peripheral arm. Additionally, subunit ND5 and its matrix bulge move about 3 Å towards peripheral arm.</p></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_SF8.html">Full size image</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
									</ul>
					
					
											
												
				<ul id="supplementary-tables-list">
																																					
						<li>
							<div class="supp-title">
																Extended Data Table 1: Summary of the model
							</div>

							<div id="st1" class="supp-item supp-tables-item no-padding cleared">
								<div class="supp-tables-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-st1.jpg" style="width:272px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																					
																				<div class="caption"></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_ST1.html">Full size table</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
																																					
						<li>
							<div class="supp-title">
																Extended Data Table 2: Summary of interactions between subunits of ovine complex I
							</div>

							<div id="st2" class="supp-item supp-tables-item no-padding cleared">
								<div class="supp-tables-wrapper supp-wrapper">
																			<img src="/nature/journal/v538/n7625/images_supplementary/nature19794-st2.jpg" style="width:300px;height:400px;" alt="" />
																												<div class="zoom-info" style="display:none;">Hover over figure to zoom</div>
																	</div>

								<div class="meta">
									<div class="caption-wrapper">
																					<div id="st2-fn1" class="footnote"><p>Analysis was performed using the PISA server (<a href="http://www.ebi.ac.uk/pdbe/pisa/">http://www.ebi.ac.uk/pdbe/pisa/</a>). Δ<i>G</i><sub>int</sub> indicates the solvation free energy gain upon formation of the assembly. <i>N</i><sub>HB</sub>, number of hydrogen bonds at the interface; N<sub>SB</sub>, number of salt bridges at the interface.</p></div>
																				<div class="caption"></div>
									</div>
								</div>
							</div>

							<div class="source-data">
															</div>

							<div class="larger">
								<p class="full-size"><a class="view-full-size" href="/nature/journal/v538/n7625/fig_tab/nature19794_ST2.html">Full size table</a></p>
							</div>

															<div class="control-zoom">
									<p class="zoom-toggle">
										<a class="zoom-off" href="#">Enable zoom</a>
									</p>
								</div>
													</li>
									</ul>
					
			</div>
</section>
	
																								
	</article>
</div>
<div id="extranav" role="complementary" class="col">
												
										
							
								



						
	
		

										
										
					

						

	


				
	<div class="box module" id="summary-box">
					<div id="edsumm" class="summary-item editors-summary" >
				<h2 class="sidebar-heading">Editor's summary
											<a href="http://arabicedition.nature.com/research" class="lang-arabic"><span class="hidden">in</span> &#1575;&#1604;&#1593;&#1585;&#1576;&#1610;&#1577;</a>
									</h2>
									<div class="full-snippet"><p>The first and largest enzyme in the respiratory chain in mammalian mitochondria is complex I, which transfers electrons from NADH to ubiquinone coupled to proton translocation across the membrane. Its huge size and complexity have meant that detailed structural information has been hard to come by. Here Leonid Sazanov and colleagues present the nearly complete atomic structure of ovine mitochondrial complex I, solved at a resolution of 3.9 &#x000C5;. The structure, revealing 45 different protein subunits combined in a in highly interlinked assembly, provides a basis for understanding the molecular basis of mutations and mechanisms of complex I function and regulation.</p></div>
							</div>
		
		
		
		
		
		
		
			</div>
												
										
					


												
										
					
				
		
				
							
												
									
																																							
																																													
															
			
						
																				
														
																																																																																																	
																																																																																																													
													
																												
										
									
			
		
							
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
										
		
									
		
	
	
		
										
					
		
										
					
		
										
					
		
										
					
		
																																																																										
					
		
										
			
			
											
				
													<div id="div-gpt-ad-1" class="ad adunit-sidebar-1 div-gpt-ad mixed" data-gpt-unitpath="/285/nature.com/article" data-gpt-sizes="300x250,300x100" data-gpt-targeting="pos=right;artid=nature19794;type=lt;issue=7625;subjmeta=631,57,1464,45,535,80,642,333,1465;abr=!webtv">
													<noscript>
						
						
																		
						
						<a href="http://pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=300x250|300x100&t=pos%3Dright%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810"><img src="http://pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=300x250|300x100&t=pos%3Dright%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810" alt="Advertisement" /></a>

													</noscript>
											</div>
																					
										
					







			<div id="special-feature-box" class="module box no-padding">
			<div class="box-padding">
				<h2 class="sidebar-heading">Editors' pick</h2>

				
				
								<ul class="feature-list feature-a">
						<li  >
															<div class="illustration">
								<a href="http://www.nature.com/news/the-polling-crisis-how-to-tell-what-people-really-think-1.20815">									<img src="/nature/images/selected-feature_2016-10-20.jpg" alt="Editors' pick image" />
								</a>																	<p class="credit">Image credit: Sébastien Thibault</p>								</div>
													</li>
						<li class="right-arrow last">
							<a href="http://www.nature.com/news/the-polling-crisis-how-to-tell-what-people-really-think-1.20815">The polling crisis: How can we tell what people really think?</a>						</li>
				</ul>
			</div>

		</div>
													
										
					


	
											
	
								



	
								
	<div class="module">
		<div id="nature-jobs-events-box" class="tab-group random box no-padding" role="tablist" >
							<div data-info-type="nature-science-jobs" id="nature-science-jobs" role="presentation"  class="tab-box first  nature-science-jobs"  >
					<h3 data-info-type="nature-science-jobs" id="nature-science-jobs-head" role="tab" class="tab sidebar-heading"  aria-controls="nature-science-jobs-list" >Science jobs</h3>

					<div class="tab-content" id="nature-science-jobs-list" role="tabpanel" >
						<div class="tab-inner"  aria-labelledby="nature-science-jobs-head" >
							<h4>NatureJobs.com</h4>

							
							<ul class="jobs-list">
																	<li>
										<h3><a href="http://www.nature.com/naturejobs/science/jobs/573949-faculty-position-for-research-director-of-the-animal-bsl3-facility-at-duke-nus">Faculty Position for Research Director of the Animal BSL3 Facility at Duke-NUS</a></h3>
										<ul>
											<li>DUKE-NUS Medical school</li>																					</ul>
									</li>
																	<li>
										<h3><a href="http://www.nature.com/naturejobs/science/jobs/583445-chief-of-newborn-medicine">Chief of Newborn Medicine</a></h3>
										<ul>
											<li>Mass General for Children at North Shore Medical Center (NSMC)</li>																					</ul>
									</li>
																	<li>
										<h3><a href="http://www.nature.com/naturejobs/science/jobs/574477-ph-d-program-and-integrated-ms-ph-d-program-of-ibs-school-ust">Ph.D. Program and Integrated MS-Ph.D. Program of IBS School-UST</a></h3>
										<ul>
											<li>Institute for Basic Science</li>																					</ul>
									</li>
															</ul>
						</div>

						<div class="box-footer">
							<ul class="action-links cleared">
								<li class="post first"><a href="/naturejobs/science/jobs/new">Post a job</a></li>
								<li class="more last"><a href="/naturejobs/science">More science jobs</a></li>
							</ul>
						</div>
					</div>
				</div>
			
											<div data-info-type="nature-science-events" id="nature-science-events" role="presentation" class="tab-box active nature-science-events">
					<h3 data-info-type="nature-science-events" id="nature-science-events-head" class="tab sidebar-heading" role="tab" aria-controls="nature-science-events-list" >Science events</h3>

					<div class="tab-content " id="nature-science-events-list" role="tabpanel" >
						<div class="tab-inner" aria-labelledby="nature-science-events-head" >
															<h4>NatureEvents Directory</h4>
																<ul class="events-list">
																			<li>
											<h3 class="title"><a href="http://www.nature.com/natureevents/science/events/49173-Agricultural_Genomics_From_Variation_Improved_Production">Agricultural Genomics - From Variation  Improved Production</a></h3>
											<p><strong><time datetime="2016-11-06">06 November 2016</time>
 &mdash; <time datetime="2016-11-08">08 November 2016</time>
</strong></p>
											<p>United Kingdom</p>
										</li>
																			<li>
											<h3 class="title"><a href="http://www.nature.com/natureevents/science/events/47293-2016_World_Life_Science_Conference">2016 World Life Science Conference</a></h3>
											<p><strong><time datetime="2016-11-01">01 November 2016</time>
 &mdash; <time datetime="2016-11-03">03 November 2016</time>
</strong></p>
											<p>Beijing, China</p>
										</li>
																			<li>
											<h3 class="title"><a href="http://www.nature.com/natureevents/science/events/47623-3rd_International_Conference_on_Self_Assembly_and_Molecular_Electronics_November_16_18_2016_Aalborg_Denmark">3rd International Conference on Self Assembly and Molecular Electronics November 16-18, 2016, Aalborg, Denmark</a></h3>
											<p><strong><time datetime="2016-11-16">16 November 2016</time>
 &mdash; <time datetime="2016-11-18">18 November 2016</time>
</strong></p>
											<p>Skjernvej 4A, Aalborg, Denmark</p>
										</li>
																	</ul>
													</div>

						<div class="box-footer">
							<ul class="action-links cleared">
								<li class="post first"><a href="/natureevents/science/events/new">Post a free event</a></li>
								<li class="more last"><a href="/natureevents/science">More science events</a></li>
							</ul>
						</div>
					</div>
				</div>
					</div>
	</div>
												
										
					

				







	

	<div class="module">
		<div class="tab-group box no-padding" id="related-top-content">

			
																			
																				
											<div class="first active tab-box" id="related-content-articles">
							<h3 class="tab">Discover more</h3>
														<div class="tab-content">
								<div class="box-padding">
									<ol>
																																																																																										<li>
												<article>
													<h1><a href="http://dx.doi.org/10.1038/nature19095">Structure of mammalian respiratory complex I</a></h1>
													<p><span class="journal-title">Nature</span> <time datetime="2016-08-10"><span class="time">
10 Aug 2016</span></time>
</p>
												</article>
											</li>
																																																																																										<li>
												<article>
													<h1><a href="http://dx.doi.org/10.1038/nature13686">Architecture of mammalian respiratory complex I</a></h1>
													<p><span class="journal-title">Nature</span> <time datetime="2014-09-07"><span class="time">
07 Sep 2014</span></time>
</p>
												</article>
											</li>
																																																																																										<li>
												<article>
													<h1><a href="http://dx.doi.org/10.1038/nrm3997">A giant molecular proton pump: structure and mechanism of respiratory complex I</a></h1>
													<p><span class="journal-title">Nature Reviews Molecular Cell Biology</span> <time datetime="2015-05-20"><span class="time">
20 May 2015</span></time>
</p>
												</article>
											</li>
																			</ol>
								</div>
								
							</div>
						</div>
												
							 					
											<div class="tab-box" id="top-content-most-read">
							<h3 class="tab">Most read</h3>
							
							<div class="tab-content">
								<div class="box-padding">
									<ol class="article-list">

																												<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/nature19323">The antibody aducanumab reduces A&beta; plaques in Alzheimer&rsquo;s disease</a></h1>
													<p class="publication"><span class="journal-title">Nature</span> <time datetime="2016-08-31" ><span class="time">31 August 2016</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/nature19328">HER2 expression identifies dynamic functional states within circulating breast cancer cells</a></h1>
													<p class="publication"><span class="journal-title">Nature</span> <time datetime="2016-08-24" ><span class="time">24 August 2016</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/nature19321">Structure of the voltage-gated calcium channel Ca<sub>v</sub>1.1 at 3.6&thinsp;&Aring; resolution</a></h1>
													<p class="publication"><span class="journal-title">Nature</span> <time datetime="2016-08-31" ><span class="time">31 August 2016</span></time></p>
												</div>
											</article>
										</li>
									
									</ol>
								</div>
								
							</div>
						</div>	
														</div>

	</div>
	
												
										
									
		
				
							
												
									
																																							
																																													
															
			
						
																				
														
																																																																																																	
																																																																																																													
													
																												
										
									
			
		
							
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
										
		
									
		
	
	
		
										
					
		
										
					
		
										
					
		
										
					
		
																																																																										
					
		
										
			
			
											
				
													<div id="div-gpt-ad-2" class="ad adunit-sidebar-2 div-gpt-ad mixed" data-gpt-unitpath="/285/nature.com/article" data-gpt-sizes="300x250,160x600" data-gpt-targeting="pos=right;artid=nature19794;type=lt;issue=7625;subjmeta=631,57,1464,45,535,80,642,333,1465;abr=!webtv">
													<noscript>
						
						
												
						
						<a href="http://pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=300x250|160x600&t=pos%3Dright%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810"><img src="http://pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=300x250|160x600&t=pos%3Dright%26artid%3Dnature19794%26type%3Dlt%26issue%3D7625%26subjmeta%3D631%2C57%2C1464%2C45%2C535%2C80%2C642%2C333%2C1465%26abr%3D%21webtv&c=102291130538810" alt="Advertisement" /></a>

													</noscript>
											</div>
												</div>
	
			</div>

			<div id="constrain-footer" class="cleared">
				<footer>											
										
					<div id="footer" class="constrain">
		<div class="inner">
		
		
					<dl class="clear ">
				<dt>Nature</dt>
				<dd class="issn"><abbr title="International Standard Serial Number">ISSN</abbr><span class="hidden">:</span> 0028-0836</dd>				<dd class="eissn"><abbr title="Electronic International Standard Serial Number">EISSN</abbr><span class="hidden">:</span> 1476-4687</dd>																			</dl>
		
		<div class="footer-main">

			<ul class="footer-links first">
							<li>
											<a href="/npg">About us</a></li>
							<li>
											<a href="/npg_/contact">Contact us</a></li>
							<li>
											<a href="/info/accessibility_statement.html">Accessibility statement</a></li>
							<li>
											<a href="/help">Help</a></li>
						</ul>

			<ul class="footer-links">
							<li>
											<a href="/info/privacy.html">Privacy policy</a></li>
							<li>
											<a href="/info/cookies.html">Use of cookies</a></li>
							<li>
											<a href="/info/legal_notice.html">Legal notice</a></li>
							<li>
											<a href="/info/tandc.html">Terms</a></li>
						</ul>

			<ul class="footer-links">
							<li>
											<a href="/naturejobs">Nature<span class="hidden"> </span>jobs</a></li>
							<li>
											<a href="http://www.natureasia.com">Nature Asia</a></li>
							<li>
											<a href="/scitable">Nature Education</a></li>
							<li>
											<a href="/webfeeds">RSS web feeds</a></li>
						</ul>

						<ul class="footer-links last">
							<li>
											<a href="/nature/about/index.html">About <strong>Nature</strong></a></li>
							<li>
											<a href="/nature/about/contact/index.html">Contact <strong>Nature</strong></a></li>
							<li>
											<a href="/nature/about/editors/index.html">About the Editors</a></li>
							<li>
											<a href="/nature/awards/index.html"><strong>Nature</strong> awards</a></li>
						</ul>
						
			<div id="mobile-css-test"></div>

										
										
					





			<form method="get" action="/search/executeSearch" id="footer-search" class="search-form" role="search">
			<input type="hidden" name="sp-c" value="25"/>
			<input type="hidden" name="sp-s" value=""/>
															 									 						<input type="hidden" name="sp-q-1" value="NATURE,NEWS"/>						 			 <fieldset class="search-and-go"><label for="footer-keyword"><span>Search:</span></label><input type="search" id="footer-keyword" name="sp-q" value="" class="txt" maxlength="200"/><button type="submit" id="footer-search-submit">Go</button></fieldset></form>
		
			<p class="copyright">
				<img width="140" height="14" alt="Springer Nature" class="logo-image" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIwAAAAOCAQAAABXsTxmAAAFO0lEQVR4Ac2WA4DuOBDH82zbtm1bM2n35dm2bdvm2bZt27Z9t8Z3nWmzabd9e0ZnMfF/fkmmFfSowtZ42AabrPFWWfG/emRbXIErYOmG3G4Zl+AKa/y/tjh8DjHPknCeAyqvKjmytCoji5teUMQqpcqoMnPzmboNuVVhVditUcVGlRhVwg1gWh7t67F2G6uPrB7LQaVYDmrVBkWoRhan9ZzfXFm0LXZ1yTHeTPEQw2fY5Q01c5Bq8o2RrtFFWUlOnqs4+eeKZUJ+/1iVl7q9xYvHu3hkWyHkTA0Kf8AbrcbC68WWAR/ixtFFef8srtnP/pfsr2cJkny7kysHL8+c7U0YLcTI0uRrwweFsEplllPxJZjBAANg8GW3Dn70g4H3vNZPuTSaS8YWCYEvkRdXnxUmkE/bBW+HY4Gl/rHOqbSrsZs4spwQWEWOFUKD0YbfqkoGjGcPEO0IMF+pAn4wsRz4vDcikf9uYgzRYIztD4JhqYNZ/ncGjF3ZjIir+afAmFjCYKzGLO+zCfm1GA0Gt9rVcJ8+By4YrGK3wfc9oSEwbDP8YHAoj/rAbhPLocrAfKySeWLulc3JVG0NBl+2K1iDMZn8kaUDYH5xfh9hMF9rMBoEvuMFIlQlGASD8FUesRQGqRpBMJBowIRjkcvIk3NdVc6VU7loFxx7QfbNAma5s1h59q7QYOjuwU6u23gOMG9tyOkDcwX3HW6gawxwbdYafI7neZAFDvCD4e3h+eALH5iT3HMC/z2r54KHzTXO9sQEY9EnZpBPKChM9471Q1AvBIaXDYOBGWEw+AaPs3xgHuVRJcNg8E3cztbOWaWkD8xDvIfNAmDa4QfO/5ud1k99YF6nbYAimEb//xwY3MXzT9dg8GpXlfcSkN05JGr42WqQCeaQ7AXXegeVwbDEKZSkMQ0bRoA5H+9z/j5qwMCHTjmFBd+ALzn2ZDD5MmCN6g27jVzJYX5F8g0Y2UrOoVRpNcaPNBirrN4yfIbDLP9HwQRjCeYY/b6iCzUDvubKGzwwxp4itsHki6uEiAIju3PrLj8YyCD++Br3jdcY4HO8lW1gKB2nW32ECIJRBeArp+UieE+DkXHuRXK8vTxO/UEwwVj0iXnaVTUtjzBPXCPu+AmDYYEQj6/CGv5K0JOl8o7GLDsajCfsRy0On9e7qWrgTQyGT0xUjqGzBRn0LaVqBMFAayFgDa/+gwaDh7jljNxM89L5jgIDL7LaBpnJNzWWIzoWuSycY0bY7bmpOS/wkj/HmEfnGAu57bUNOaPByL5cp0/Mxdx7ohCeH5998sXt3HZBEIzdRohRJfBn8jUYeIFLxl6IBHM7n6vuQkwryNDfNpsciCWcfFUx/Mlp/BJe4YExeSB7MPpSYL8oMOw9acBY3bjvD7AIRuAzLhhZnNu/hgfI8LQBw14qnRxVJgiGFewxYGRxemHgN3gNWwqdb1UsDAZ3uJsNa+A2nZMYTCgWWMTey54qS8AgSDLc8dXRRX8LjNyrl4gGA4M0GK6/JrCvv3CPB3wrPu8Ho99jMDWQY9ryFlbCZA3GGszeeUKYrcCBITB0zj72rf6JKh8EY2Kx6vLF8kzOEXTncaE8i3fihXKmm3TsNrAJNmGXAJj5VEdpNK6p3Cw389ukAdW53xy43BlheUJXcN+qXnCT4Up8Dp7F63GeXcG9Orgajzp2EPaQBFWA55npYqXZ+TuXlHSiFruyd+XGU4lX7sYKO3orSK7v6q5m1macheUyuAzvw0tg6dhCXv8FoVho9s6wB47hEXkAd9ntfwVHU0mvVTQJMQAAAABJRU5ErkJggg==" /> 
				<small>&#169; 2016 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.</small>
				<small>partner of <a href="/info/partners.html">AGORA, HINARI, OARE, INASP, ORCID, CrossRef, COUNTER and COPE</a></small>
			</p>
		</div>
	</div>
</div>

					</footer>

			</div>

										
												
									
																																							
																					
															
			
						
																				
										
																																																																																			
																																																																																																													
													
																												
										
									
			
		
							
		
													
		
										
		
													
		
										
		
													
		
										
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
		
		
													
		
										
		
									
		
	
	
		
										
					
		
										
					
		
										
					
		
																																																																										
					
		
										
			
			
											
				
																			<div id="div-gpt-ad-4" class="ad adunit-out-of-page div-gpt-ad out-of-page" data-gpt-unitpath="/285/nature.com/article" data-gpt-targeting="artid=nature19794;type=lt;issue=7625;subjmeta=631,57,1464,45,535,80,642,333,1465;abr=!webtv"></div>
																				</div>

									
										
					


			
																																																		
	
				<script src="/view/scripts/jquery/jquery-1.7.2.min.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.baseline.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.hitch.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.linkify.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.colorfade.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.scrollintoview.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.ellipsis.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.swapclasses.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.preview.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.grow.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.collision.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/plugins/jquery.initplaceholders.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.configuration.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.css.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.stringutils.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.cookie.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.broadcaster.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.pagemanager.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.popup.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.truncator.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.bitstate.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.slideshow.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.authorlists.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.tabgroup.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.popup-this.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.zoomer.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.simplecarousel.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.sidebar.js" type="text/javascript"></script>
																																												<!--[if gt IE 8]><!-->													
	
				<script src="/view/scripts/global.mobile.js" type="text/javascript"></script>
				<!--<![endif]-->																																																					
	
				<script src="http://content.readcube.com/ping?doi=10.1038/nature19794&amp;format=js&amp;last_modified=2016-10-19" type="text/javascript"></script>
																																																									
	
				<script src="http://admin.brightcove.com/js/BrightcoveExperiences.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.draggablepopup.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.message.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.twitterfeed.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.typehelper.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/nature/scripts/site.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/global.deepdyve.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/article/global.article.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/article/global.highlighter.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/article/global.figures.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/article/global.articlesetup.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/article/global.init.js" type="text/javascript"></script>
																																																									
	
				<script src="/view/scripts/icb/global.icb.js" type="text/javascript"></script>
																								
										
					

<noscript>
	<div><img alt="" id="DCSIMG" width="1" height="1" src="http://statse.webtrendslive.com/dcs0zztfg00000s969s37qoal_2f6z/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=10.2.55" /></div>
</noscript>

<!-- end webtrends Version: 8.6.0 -->
																																																																																																																																																																			<script type="text/javascript"><!--//--><![CDATA[//><!-- 
			com.nature.Configuration.add({'siteName': 'nature', 'isLoggedIn': 'yes', 'classicArticle': 'true', 'mobileSettings': 'false', 'webtrendsEnabled': 'true', 'webtrendsTagsToRemove': '[mod_name]', 'brightcovePlayerId': '2035075680001', 'brightcovePlayerKey': 'AQ~~,AAAByWTdmvk~,YEX2I6TuT0mVx18B2wHeXXiHeH3folKC', 'brightcovePlayerToken': 'Dh8K29uMMw3OTeOPdu-45QHidPod5hRfAcDouTmGs1ZExlUytHBKBA..', 'articleLevel': '1', 'articleDoi': '10.1038/nature19794', 'nextDoi': '10.1038/nature19785', 'prevDoi': '10.1038/nature19836', 'typeText': 'Letter', 'mopRegister': '/register/nature', 'mopSubscribe': 'https://secure.nature.com/subscribe/nature', 'mopLogin': '/foxtrot/svc/login', 'mopTitle': 'nature.com'});
			//--><!]]></script>
							
	</body>
</html>